<?xml version="1.0" encoding="utf-8"?>
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2020-05-10 Sun 01:10 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Other Papers</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Abhinav Sharma" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="https://fniessen.github.io/org-html-themes/styles/readtheorg/css/htmlize.css"/>
<link rel="stylesheet" type="text/css" href="https://fniessen.github.io/org-html-themes/styles/readtheorg/css/readtheorg.css"/>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script type="text/javascript" src="https://fniessen.github.io/org-html-themes/styles/lib/js/jquery.stickytableheaders.min.js"></script>
<script type="text/javascript" src="https://fniessen.github.io/org-html-themes/styles/readtheorg/js/readtheorg.js"></script>
<style> p{margin-bottom:5px;}</style>
<style> li{margin-bottom:10px}</style
<script type="text/javascript">
// @license magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&amp;dn=gpl-3.0.txt GPL-v3-or-Later
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.cacheClassElem = elem.className;
         elem.cacheClassTarget = target.className;
         target.className = "code-highlighted";
         elem.className   = "code-highlighted";
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(elem.cacheClassElem)
         elem.className = elem.cacheClassElem;
       if(elem.cacheClassTarget)
         target.className = elem.cacheClassTarget;
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="content">
<h1 class="title">Other Papers</h1>
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#org57b38f3">1. <span class="todo TODO">TODO</span> Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.</a></li>
<li><a href="#org23b41c5">2. <span class="todo TODO">TODO</span> Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.</a></li>
<li><a href="#org988d028">3. <span class="todo TODO">TODO</span> Machine learning and tubercular drug target recognition.</a></li>
<li><a href="#orgcbaad56">4. <span class="todo WAIT">WAIT</span> Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem</a>
<ul>
<li><a href="#org58ecfe1">4.1. Notes</a></li>
<li><a href="#orgff3322c">4.2. FileName</a></li>
<li><a href="#org91e2aee">4.3. Abstract</a></li>
<li><a href="#org62fa5ca">4.4. FutureWork</a></li>
</ul>
</li>
<li><a href="#org7e83980">5. <span class="todo ___">[ ]</span> Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm</a>
<ul>
<li><a href="#org1be269f">5.1. Notes</a></li>
<li><a href="#org7b3a768">5.2. FileName</a></li>
<li><a href="#org1786337">5.3. Abstract</a></li>
<li><a href="#org5ad2bd5">5.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgdd31c86">6. <span class="todo ___">[-]</span> Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper</a>
<ul>
<li><a href="#orgada71d5">6.1. Notes</a></li>
<li><a href="#orgeb9a2ba">6.2. FileName</a></li>
<li><a href="#org8877e70">6.3. Abstract</a></li>
<li><a href="#org85aa865">6.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgbdebada">7. <span class="done _X_">[X]</span> Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities</a>
<ul>
<li><a href="#org92291d3">7.1. Notes</a></li>
<li><a href="#org0c14674">7.2. FileName</a></li>
<li><a href="#org4c1eb0a">7.3. Abstract</a></li>
<li><a href="#org89610da">7.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgbb028f6">8. <span class="done DONE">DONE</span> Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data</a>
<ul>
<li><a href="#orgf9ac363">8.1. Notes</a></li>
<li><a href="#org23c093c">8.2. FileName</a></li>
<li><a href="#org832f007">8.3. Abstract</a></li>
<li><a href="#org3c2a443">8.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org52f1ec4">9. <span class="done DONE">DONE</span> Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction</a>
<ul>
<li><a href="#orgb517e0b">9.1. Notes</a></li>
<li><a href="#org7788556">9.2. FileName</a></li>
<li><a href="#orgce98bae">9.3. Abstract</a></li>
<li><a href="#org35fc982">9.4. Conclusion</a></li>
<li><a href="#org28c1651">9.5. Supplementary</a></li>
</ul>
</li>
<li><a href="#orgd1db7f7">10. <span class="todo ___">[-]</span> Ching2018 - Opportunities and obstacles for deep learning in biology and medicine</a>
<ul>
<li><a href="#org27736b4">10.1. Notes</a></li>
<li><a href="#orgab82770">10.2. FileName</a></li>
<li><a href="#orgeae9cc2">10.3. Abstract</a></li>
<li><a href="#org9788566">10.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org6cbfe18">11. <span class="todo WAIT">WAIT</span> Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data</a>
<ul>
<li><a href="#org2b56346">11.1. Notes</a></li>
<li><a href="#org27d6392">11.2. FileName</a></li>
<li><a href="#org856c6ab">11.3. Abstract</a></li>
<li><a href="#org31498b4">11.4. Conclusion</a></li>
<li><a href="#org4acfce8">11.5. Supplementary</a>
<ul>
<li><a href="#org23f4eb8">11.5.1. Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#orgd1b6c19">12. <span class="todo ___">[?]</span> Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org431a354">12.1. Notes</a></li>
<li><a href="#orgc6417ce">12.2. FileName</a></li>
<li><a href="#orgd9ccb82">12.3. Abstract</a></li>
<li><a href="#org0062525">12.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgf9684f6">13. <span class="todo STRT">STRT</span> Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports</a>
<ul>
<li><a href="#orga737675">13.1. Notes</a></li>
<li><a href="#org6ec5335">13.2. FileName</a></li>
<li><a href="#org39f1227">13.3. Abstract</a></li>
<li><a href="#orgd9a67c6">13.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org4293c3b">14. <span class="todo ___">[-]</span> Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence</a>
<ul>
<li><a href="#orgc6e6bab">14.1. Notes</a></li>
<li><a href="#org922047b">14.2. FileName</a></li>
<li><a href="#org3ab3f5b">14.3. Abstract</a></li>
<li><a href="#org0f65e4f">14.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org6cc35d9">15. <span class="done _X_">[X]</span> Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models</a>
<ul>
<li><a href="#orgc34f6de">15.1. Notes</a></li>
<li><a href="#org46e54cb">15.2. FileName</a></li>
<li><a href="#org69459be">15.3. Abstract</a></li>
<li><a href="#org1427865">15.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgeec8a60">16. <span class="done _X_">[X]</span> Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation</a>
<ul>
<li><a href="#orgc086304">16.1. Notes</a></li>
<li><a href="#org5023c9c">16.2. FileName</a></li>
<li><a href="#org75e3d14">16.3. Abstract</a></li>
<li><a href="#orga7b7477">16.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org0fb76b4">17. <span class="done _X_">[X]</span> Ekins2014<sub>AreBiggerDataSets.pdf</sub></a>
<ul>
<li><a href="#org051540e">17.1. Notes</a></li>
<li><a href="#orgd01e027">17.2. Title</a></li>
<li><a href="#orgf84f2d8">17.3. Abstract</a></li>
<li><a href="#org07c827c">17.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org914fdf0">18. <span class="done _X_">[X]</span> Ekins2014<sub>CombiningComputationalMethods.pdf</sub></a>
<ul>
<li><a href="#org0762453">18.1. Notes</a></li>
<li><a href="#orgc060b52">18.2. Title</a></li>
<li><a href="#org4c07fa7">18.3. Abstract</a></li>
<li><a href="#org6ca8159">18.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org03d038c">19. <span class="done _X_">[X]</span> Ekins2016<sub>MachineLearningModel.pdf</sub></a>
<ul>
<li><a href="#org6e8da0a">19.1. Notes</a></li>
<li><a href="#org2df7156">19.2. Title</a></li>
<li><a href="#orgc6b6a47">19.3. Abstract</a></li>
<li><a href="#orgc708f08">19.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org31dc89c">20. <span class="done _X_">[X]</span> Ekins2017<sub>MachineLearning.pdf</sub></a>
<ul>
<li><a href="#org77271bc">20.1. Notes</a></li>
<li><a href="#org49a22de">20.2. Title</a></li>
<li><a href="#org9219c8a">20.3. Abstract</a></li>
<li><a href="#org6d5d88c">20.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org0b14e0a">21. <span class="todo ___">[-]</span> Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis</a>
<ul>
<li><a href="#orgf45bb2e">21.1. Notes</a></li>
<li><a href="#orgbf3c921">21.2. FileName</a></li>
<li><a href="#org15feb0d">21.3. Abstract</a></li>
<li><a href="#orgec53136">21.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org95a0312">22. <span class="todo ___">[-]</span> Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay</a>
<ul>
<li><a href="#org01eb91e">22.1. Notes</a></li>
<li><a href="#orgd52acda">22.2. FileName</a></li>
<li><a href="#org5dc068e">22.3. Abstract</a></li>
<li><a href="#org841c69b">22.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgb86575d">23. <span class="todo ___">[-]</span> Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data</a>
<ul>
<li><a href="#orgd388bd6">23.1. Notes</a></li>
<li><a href="#org8e6b88d">23.2. FileName</a></li>
<li><a href="#org1f6dc26">23.3. Abstract</a></li>
<li><a href="#org7f814aa">23.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgcc12666">24. <span class="done _X_">[X]</span> Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models</a>
<ul>
<li><a href="#org49f1e94">24.1. Notes</a></li>
<li><a href="#org88b6ba4">24.2. FileName</a></li>
<li><a href="#org111d250">24.3. Abstract</a></li>
<li><a href="#org911b57e">24.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org42f08c9">25. <span class="todo WAIT">WAIT</span> Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#orge988dcc">25.1. Notes</a></li>
<li><a href="#orgef78b44">25.2. FileName</a></li>
<li><a href="#org3b7214d">25.3. Abstract</a></li>
<li><a href="#orgcf3b80d">25.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgf8112f4">26. <span class="todo WAIT">WAIT</span> Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance</a>
<ul>
<li><a href="#orgf7aa7be">26.1. Notes</a></li>
<li><a href="#orgd3857ee">26.2. FileName</a></li>
<li><a href="#org37851bc">26.3. Abstract</a></li>
<li><a href="#orgcb5405a">26.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org55dfe62">27. <span class="todo ___">[-]</span> Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes</a>
<ul>
<li><a href="#org95ca24c">27.1. Notes</a></li>
<li><a href="#org7bcf5f5">27.2. FileName</a></li>
<li><a href="#org4ba203e">27.3. Abstract</a></li>
<li><a href="#org9180cf6">27.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org41a77f7">28. <span class="todo ___">[-]</span> Killian2019 - Learning to Prescribe Interventions for Tuberculosis Patients Using Digital Adherence Data</a>
<ul>
<li><a href="#org62f9cb1">28.1. Notes</a></li>
<li><a href="#org2d8d3a1">28.2. FileName</a></li>
<li><a href="#orgf7b752c">28.3. Abstract</a></li>
<li><a href="#orge1c8c9a">28.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org799097a">29. <span class="todo WAIT">WAIT</span> Kouchaki2019 - Application of machine learning techniques to tuberculosis drug resistance analysis</a>
<ul>
<li><a href="#org106060c">29.1. Notes</a></li>
<li><a href="#org94270d2">29.2. FileName</a></li>
<li><a href="#orga069eaa">29.3. Abstract</a></li>
<li><a href="#orge7c29f7">29.4. Conclusion</a></li>
<li><a href="#org5eb2280">29.5. Supplementary</a>
<ul>
<li><a href="#org7c6a8fc">29.5.1. Kouchaki2019<sub>ApplicationOfML</sub><sub>Supplementary.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#orga83158f">30. <span class="done DONE">DONE</span> Kouchaki2020 - Multi-Label Random Forest Model for Tuberculosis Drug Resistance Classification and Mutation Ranking</a>
<ul>
<li><a href="#org9be3362">30.1. Notes</a></li>
<li><a href="#org7f3c9bb">30.2. FileName</a></li>
<li><a href="#org0ba967d">30.3. Abstract</a></li>
<li><a href="#orgae759f4">30.4. Conclusion</a></li>
<li><a href="#org04124da">30.5. Supplementary</a>
<ul>
<li><a href="#org1008759">30.5.1. Kouchaki2020<sub>MultiLabel.zip</sub></a></li>
<li><a href="#orga9b14aa">30.5.2. Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub></a></li>
<li><a href="#orgf4cd826">30.5.3. Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org3123fae">31. <span class="todo ___">[-]</span> Kovalishyn2019 - Hybrid Design of Isonicotinic Acid Hydrazide Derivatives: Machine Learning Studies, Synthesis and Biological Evaluation of their Anti- tuberculosis Activity</a>
<ul>
<li><a href="#orgdc0d077">31.1. Notes</a></li>
<li><a href="#org36706ac">31.2. FileName</a></li>
<li><a href="#org8d36144">31.3. Abstract</a></li>
<li><a href="#org7a94d65">31.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org901ec89">32. <span class="todo ___">[-]</span> Kumari2019 - A genetic programming-based approach to identify potential inhibitors of serine protease of Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org537b1b7">32.1. Notes</a></li>
<li><a href="#org3bffb78">32.2. FileName</a></li>
<li><a href="#org5dafb05">32.3. Abstract</a></li>
<li><a href="#org513c1c3">32.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgf3a0f5c">33. <span class="todo ___">[-]</span> Lai2019 - Comparison of the Predictive Outcomes for Anti-Tuberculosis Drug-Induced Hepatotoxicity by Different Machine Learning Techniques</a>
<ul>
<li><a href="#org775523b">33.1. Notes</a></li>
<li><a href="#org48be139">33.2. FileName</a></li>
<li><a href="#org8788a2a">33.3. Abstract</a></li>
<li><a href="#orgc2da543">33.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orge57d55e">34. <span class="todo ___">[-]</span> Lane2018 - Comparing and Validating Machine Learning Models for Mycobacterium tuberculosis Drug Discovery</a>
<ul>
<li><a href="#org9a3c98a">34.1. Notes</a></li>
<li><a href="#org021950e">34.2. FileName</a></li>
<li><a href="#orge96fb12">34.3. Abstract</a></li>
<li><a href="#orgefed891">34.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgfe6b046">35. <span class="done _X_">[X]</span> Lin2001 - Conserved codon composition of ribosomal protein coding genes in Escherichia coli, Mycobacterium tuberculosis and Saccharomyces cerevisiae: lessons from supervised machine learning in functional genomics</a>
<ul>
<li><a href="#orgc2a806b">35.1. Notes</a></li>
<li><a href="#orgee3a42b">35.2. FileName</a></li>
<li><a href="#org0e1e902">35.3. Abstract</a></li>
<li><a href="#org20e73e0">35.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org3320a29">36. <span class="todo ___">[?]</span> LokeshKumar2019 - Prediction of Multi Drug Resistant Tuberculosis using Machine Learning Techniques</a>
<ul>
<li><a href="#org7253372">36.1. Notes</a></li>
<li><a href="#org10e1318">36.2. FileName</a></li>
<li><a href="#org59c4bd6">36.3. Abstract</a></li>
<li><a href="#org71e9d41">36.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd2df039">37. <span class="done _X_">[X]</span> Mahe2019 - A large scale evaluation of TBProfiler and Mykrobe for antibiotic resistance prediction in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org4d414dc">37.1. Notes</a></li>
<li><a href="#org7a953f6">37.2. FileName</a></li>
<li><a href="#org3ed93c3">37.3. Abstract</a></li>
<li><a href="#org5e89d65">37.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org6663d3e">38. <span class="todo ___">[-]</span> Marino2016 - Computational and Empirical Studies Predict Mycobacterium tuberculosis-Specific T Cells as a Biomarker for Infection Outcome</a>
<ul>
<li><a href="#org0e24d78">38.1. Notes</a></li>
<li><a href="#orgcd08c7b">38.2. FileName</a></li>
<li><a href="#org064cd4e">38.3. Abstract</a></li>
<li><a href="#org01fd79a">38.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orge8d8ccf">39. <span class="todo ___">[-]</span> Marques2019 - Pulmonary nontuberculous mycobacterial infections: presumptive diagnosis based on the international microbiological criteria adopted in the state of São Paulo, Brazil, 2011-2014</a>
<ul>
<li><a href="#orgd69853c">39.1. Notes</a></li>
<li><a href="#org1f2600f">39.2. FileName</a></li>
<li><a href="#org07117df">39.3. Abstract</a></li>
<li><a href="#org344e749">39.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org74089f1">40. <span class="todo ___">[-]</span> Meldenez2015 - On Combining Multiple-Instance Learning and Active Learning for Computer-Aided Detection of Tuberculosis</a>
<ul>
<li><a href="#orgd8008be">40.1. Notes</a></li>
<li><a href="#org948de83">40.2. FileName</a></li>
<li><a href="#orgf7329e5">40.3. Abstract</a></li>
<li><a href="#org22427ec">40.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgf77043f">41. <span class="todo ___">[-]</span> Mikusova2016 - Learning from the past for TB drug discovery in the future</a>
<ul>
<li><a href="#orge333a3c">41.1. Notes</a></li>
<li><a href="#org40a47f9">41.2. FileName</a></li>
<li><a href="#org8a1b246">41.3. Abstract</a></li>
<li><a href="#orga5f8d17">41.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgb1e74c9">42. <span class="todo ___">[-]</span> Ngo2019 - Genomic prediction of tuberculosis drug- resistance: benchmarking existing databases and prediction algorithms</a>
<ul>
<li><a href="#orgb78612c">42.1. Notes</a></li>
<li><a href="#orgb4b8475">42.2. FileName</a></li>
<li><a href="#org5ec115e">42.3. Abstract</a></li>
<li><a href="#org1d32a25">42.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd294a50">43. <span class="todo ___">[-]</span> Periwal2011 - Predictive models for anti-tubercular molecules using machine learning on high-throughput biological screening datasets</a>
<ul>
<li><a href="#org622b584">43.1. Notes</a></li>
<li><a href="#orgb2a1106">43.2. FileName</a></li>
<li><a href="#orgee70c69">43.3. Abstract</a></li>
<li><a href="#org786a841">43.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgeaf56b4">44. <span class="todo ___">[ ]</span> Ratnatunga2020 - The Rise of Non-Tuberculosis Mycobacterial Lung Disease</a>
<ul>
<li><a href="#org44d632f">44.1. Notes</a></li>
<li><a href="#org7e0c5ba">44.2. FileName</a></li>
<li><a href="#orge9372ae">44.3. Abstract</a></li>
<li><a href="#orgd8c691d">44.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orga687d85">45. <span class="todo WAIT">WAIT</span> Romero2020 - Decision tree machine learning applied to bovine tuberculosis risk factors to aid disease control decision making</a>
<ul>
<li><a href="#org81825b2">45.1. Notes</a></li>
<li><a href="#orgf4a62e1">45.2. FileName</a></li>
<li><a href="#orgc58b5a1">45.3. Abstract</a></li>
<li><a href="#org32c76ad">45.4. Conclusion</a></li>
<li><a href="#org20cdb11">45.5. Supplementary</a>
<ul>
<li><a href="#org93362d7">45.5.1. Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org9f971cb">46. <span class="done DONE">DONE</span> Sauer2018 - Feature selection and prediction of treatment failure in tuberculosis</a>
<ul>
<li><a href="#orgdd9ad5d">46.1. Notes</a></li>
<li><a href="#org5cdbcc3">46.2. FileName</a></li>
<li><a href="#org53a2297">46.3. Abstract</a></li>
<li><a href="#orga2b155c">46.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org062b295">47. <span class="todo ___">[-]</span> Schiff2018 - Clinical relevance of non-tuberculous mycobacteria isolated from respiratory specimens: seven year experience in a UK hospital</a>
<ul>
<li><a href="#org16aa2f8">47.1. Notes</a></li>
<li><a href="#orge8b7ca0">47.2. FileName</a></li>
<li><a href="#org29ec187">47.3. Abstract</a></li>
<li><a href="#org17178fd">47.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org2649eee">48. <span class="todo WAIT">WAIT</span> Sergeev2016 - Genome-wide Analysis of MDR and XDR Tuberculosis from Belarus: Machine-learning Approach</a>
<ul>
<li><a href="#orgce785ce">48.1. Notes</a></li>
<li><a href="#org6202544">48.2. FileName</a></li>
<li><a href="#org8855cd8">48.3. Abstract</a></li>
<li><a href="#org1a3f55d">48.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org5508a5a">49. <span class="todo ___">[-]</span> Vizcaino2010 - Computational Prediction and Experimental Assessment of Secreted/Surface Proteins from Mycobacterium tuberculosis H37Rv</a>
<ul>
<li><a href="#org3898256">49.1. Notes</a></li>
<li><a href="#orgb77fa24">49.2. FileName</a></li>
<li><a href="#org81f28a2">49.3. Abstract</a></li>
<li><a href="#org227a48e">49.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org709bfe6">50. <span class="todo ___">[-]</span> Waman2019 - Mycobacterial genomics and structural bioinformatics: opportunities and challenges in drug discovery</a>
<ul>
<li><a href="#org3694c31">50.1. Notes</a></li>
<li><a href="#orga2ce9f6">50.2. FileName</a></li>
<li><a href="#org00b7acf">50.3. Abstract</a></li>
<li><a href="#org757c3c5">50.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org73e95c0">51. <span class="todo ___">[?]</span> Wang2020 - A deep learning-based method for drug-target interaction prediction based on long short-term memory neural network</a>
<ul>
<li><a href="#orgda3edf0">51.1. Notes</a></li>
<li><a href="#org63e13ba">51.2. FileName</a></li>
<li><a href="#orgff382d7">51.3. Abstract</a></li>
<li><a href="#org2703fae">51.4. Conclusion</a></li>
<li><a href="#orgf3774ef">51.5. Supplementary</a></li>
</ul>
</li>
<li><a href="#orge662795">52. <span class="todo ___">[?]</span> Weiner2017 - High-throughput and computational approaches for diagnostic and prognostic host tuberculosis biomarkers</a>
<ul>
<li><a href="#org656acf5">52.1. Notes</a></li>
<li><a href="#orgf2ebcd2">52.2. FileName</a></li>
<li><a href="#org98880f6">52.3. Abstract</a></li>
<li><a href="#org833402f">52.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgefd6324">53. <span class="todo WAIT">WAIT</span> Wilder2019 - Improving tuberculosis treatment by integrating optimization and learning</a>
<ul>
<li><a href="#org6973ad5">53.1. FileName</a></li>
<li><a href="#org8eadce3">53.2. Notes</a></li>
<li><a href="#org9f7b559">53.3. Abstract</a></li>
<li><a href="#org7902c74">53.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org1556e3b">54. <span class="done DONE">DONE</span> Yang2017 - Machine learning for classifying tuberculosis drug-resistance from DNA sequencing data</a>
<ul>
<li><a href="#orgefbcefe">54.1. Notes</a></li>
<li><a href="#org2fd7808">54.2. FileName</a></li>
<li><a href="#org0fc008a">54.3. Abstract</a></li>
<li><a href="#org6087ded">54.4. Conclusion</a></li>
<li><a href="#org511e792">54.5. Supplementary</a></li>
</ul>
</li>
<li><a href="#orgd140b58">55. <span class="done DONE">DONE</span> Yang2019 - DeepAMR for predicting co-occurrent resistance of Mycobacterium tuberculosis</a>
<ul>
<li><a href="#orgd323000">55.1. Notes</a></li>
<li><a href="#org2c3345b">55.2. FileName</a></li>
<li><a href="#org734aa38">55.3. Abstract</a></li>
<li><a href="#orgdd3dd6e">55.4. Conclusion</a></li>
<li><a href="#org28f98bb">55.5. Supplementary</a></li>
</ul>
</li>
<li><a href="#orga442335">56. <span class="todo ___">[?]</span> Zhang2020 - An integrated machine learning approach for predicting DosR-regulated genes in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org0adf093">56.1. Notes</a></li>
<li><a href="#orgff85635">56.2. FileName</a></li>
<li><a href="#orgde621f2">56.3. Abstract</a></li>
<li><a href="#org22e7dc7">56.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org174cdbc">57. <span class="todo ___">[?]</span> Gabrilian2019 - TB DEPOT (Data Exploration Portal): A multi- domain tuberculosis data analysis resource</a>
<ul>
<li><a href="#orgbe83ed9">57.1. Notes</a></li>
<li><a href="#org93b29fa">57.2. FileName</a></li>
<li><a href="#orge791a09">57.3. Abstract</a></li>
<li><a href="#org61252e0">57.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org5e0241d">58. <span class="done _X_">[X]</span> Garnier2019 - Automatic diagnostics of tuberculosis using convolutional neural networks analysis of MODS digital images</a>
<ul>
<li><a href="#org5003aad">58.1. Notes</a></li>
<li><a href="#org6ceb1d6">58.2. FileName</a></li>
<li><a href="#org7aedc2a">58.3. Abstract</a></li>
<li><a href="#org50f88c8">58.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org7ace1ea">59. <span class="todo ___">[-]</span> Muzondiwa2020 - Resistance Sniffer: An online tool for prediction of drug resistance patterns of Mycobacterium tuberculosis isolates using next generation sequencing data</a>
<ul>
<li><a href="#org0858bb1">59.1. Notes</a></li>
<li><a href="#orgc9e78f0">59.2. FileName</a></li>
<li><a href="#org62bef29">59.3. Abstract</a></li>
<li><a href="#org1284d92">59.4. Conclusion</a></li>
</ul>
</li>
</ul>
</div>
</div>

<div id="outline-container-org57b38f3" class="outline-2">
<h2 id="org57b38f3"><span class="section-number-2">1</span> <span class="todo TODO">TODO</span> Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.</h2>
<div class="outline-text-2" id="text-1">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30084331">https://www.ncbi.nlm.nih.gov/pubmed/30084331</a>
</p>
</div>
</div>
<div id="outline-container-org23b41c5" class="outline-2">
<h2 id="org23b41c5"><span class="section-number-2">2</span> <span class="todo TODO">TODO</span> Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.</h2>
<div class="outline-text-2" id="text-2">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30973110">https://www.ncbi.nlm.nih.gov/pubmed/30973110</a>
</p>
</div>
</div>
<div id="outline-container-org988d028" class="outline-2">
<h2 id="org988d028"><span class="section-number-2">3</span> <span class="todo TODO">TODO</span> Machine learning and tubercular drug target recognition.</h2>
<div class="outline-text-2" id="text-3">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/24245763">https://www.ncbi.nlm.nih.gov/pubmed/24245763</a>
</p>
</div>
</div>





<div id="outline-container-orgcbaad56" class="outline-2">
<h2 id="orgcbaad56"><span class="section-number-2">4</span> <span class="todo WAIT">WAIT</span> Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem</h2>
<div class="outline-text-2" id="text-4">
</div>
<div id="outline-container-org58ecfe1" class="outline-3">
<h3 id="org58ecfe1"><span class="section-number-3">4.1</span> Notes</h3>
<div class="outline-text-3" id="text-4-1">
<ul class="org-ul">
<li>Survey paper</li>
<li>Could be used build up the background for the need for research in NTM</li>
</ul>
</div>
</div>
<div id="outline-container-orgff3322c" class="outline-3">
<h3 id="orgff3322c"><span class="section-number-3">4.2</span> FileName</h3>
<div class="outline-text-3" id="text-4-2">
<ul class="org-ul">
<li>Ahmed2020<sub>NonTuberculosisMycoANeglectedAndEmergingProblem.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org91e2aee" class="outline-3">
<h3 id="org91e2aee"><span class="section-number-3">4.3</span> Abstract</h3>
<div class="outline-text-3" id="text-4-3">
<ul class="org-ul">
<li>The incidence of NTM infections is rising owing to a growing population of
immunocompromised and vulnerable individuals, complex medical and surgical
procedures, as well as increased awareness and diagnostic capabilities.</li>
</ul>
</div>
</div>
<div id="outline-container-org62fa5ca" class="outline-3">
<h3 id="org62fa5ca"><span class="section-number-3">4.4</span> FutureWork</h3>
<div class="outline-text-3" id="text-4-4">
<ul class="org-ul">
<li>Simpler and more rapid tests would be ideal in making the diagnosis of NTM
disease more quickly. However, the development of such tests is complicated
by the large number of NTM species that can cause disease and by the criteria
for disease, which require more than just the presence of the microorganism.
As a result, NTM disease will require multiple diagnostic parameters, leading
to delays and higher costs. More specific biomarkers to monitor the progress
of treatment and test cure are lacking.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org7e83980" class="outline-2">
<h2 id="org7e83980"><span class="section-number-2">5</span> <span class="todo ___">[ ]</span> Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm</h2>
<div class="outline-text-2" id="text-5">
</div>
<div id="outline-container-org1be269f" class="outline-3">
<h3 id="org1be269f"><span class="section-number-3">5.1</span> Notes</h3>
<div class="outline-text-3" id="text-5-1">
<ul class="org-ul">
<li>Can adopt a similar approach towards the classification of NTM lineages
EMILYN: Taxonomy does not make sense for NTM. Even a DB doesn&rsquo;t exist
specifically for Resistance and Prediction but the raw genomes are available
in NCBI</li>

<li>Can alter the algorithm to XGBoost or Neural Networks to improve accuracy
EMILYN: This has been done already</li>
</ul>
</div>
</div>
<div id="outline-container-org7b3a768" class="outline-3">
<h3 id="org7b3a768"><span class="section-number-3">5.2</span> FileName</h3>
<div class="outline-text-3" id="text-5-2">
<ul class="org-ul">
<li>Aze2015<sub>GenomicsAndMachine.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org1786337" class="outline-3">
<h3 id="org1786337"><span class="section-number-3">5.3</span> Abstract</h3>
<div class="outline-text-3" id="text-5-3">
<ul class="org-ul">
<li><p>
Mycobacterium tuberculosis complex taxonomy has rapidly evolved in the last 20
years through intensive clinical isolation, advances in sequencing and in the
description of fast-evolving loci (CRISPR and MIRU-VNTR)
</p>

<p>
EMILYN: This paper has been a bit outdated now since someone reclassified these
again and found MTB is a complex and combines ( Tuberculosis, Africanum and
Bovis )
</p></li>

<li>The objectives of this study were 1) to explicit the consensus that exists
between the alternative taxonomies, and 2) to provide an on-line tool to ease
classification of new isolates. Genotyping (24-VNTR, 43-spacers spoli-
gotypes, IS6110-RFLP) was undertaken for 3,454 clinical isolates from the
Netherlands (2004-2008). The resulting database was enlarged with African
isolates to include most human tuberculosis diversity. Assignations were
obtained using TB-Lineage, MIRU-VNTRPlus, SITVITWEB and an algorithm from
Borile et al</li>

<li>Original and consensus assignations of the all isolates from the database were
subsequently implemented into an ensemble learning approach based on Machine
Learning tool Weka to derive a classification scheme.</li>

<li><p>
When applied to independent datasets, it was able to suggest new sublineages
such as pseudo-Beijing. This Lineage Prediction tool, efficient on 15-MIRU,
24-VNTR and spoligotype data is available on the web interface “TBminer.”
</p>

<p>
EMILYN: No one really uses this TBMiner actually due to the interface not
being good and is inconsistent. Maybe we can work on this later to improve the
online service.
</p></li>

<li>We successfully used Machine Learning on a large dataset to set up and make
available the first consensual taxonomy for human Mycobacterium tuberculosis
complex</li>
</ul>
</div>
</div>

<div id="outline-container-org5ad2bd5" class="outline-3">
<h3 id="org5ad2bd5"><span class="section-number-3">5.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-5-4">
<ul class="org-ul">
<li>We developed an approach making use of standard typing data for human M.</li>
</ul>
<p>
tuberculosis isolates to infer a consensual taxonomy concordant with most
up-to-date data on Whole Genome Sequencing diversity.
</p>

<ul class="org-ul">
<li>We hope that the same approach can benefit other human pathogens having
alternative taxonomies according to Serotypes, CRISPRtypes, MLST-types such as
Salmo- nella, Listeria, Brucella, and more broadly to other organisms such as
bacterial plant- pathogens.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgdd31c86" class="outline-2">
<h2 id="orgdd31c86"><span class="section-number-2">6</span> <span class="todo ___">[-]</span> Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper</h2>
<div class="outline-text-2" id="text-6">
</div>
<div id="outline-container-orgada71d5" class="outline-3">
<h3 id="orgada71d5"><span class="section-number-3">6.1</span> Notes</h3>
<div class="outline-text-3" id="text-6-1">
<ul class="org-ul">
<li>Not directly related to (N)TB</li>
<li>Survey paper</li>
<li>Analyzes various ML techniques</li>
<li>Mentions various drug-target-interaction databases</li>
</ul>
</div>
</div>
<div id="outline-container-orgeb9a2ba" class="outline-3">
<h3 id="orgeb9a2ba"><span class="section-number-3">6.2</span> FileName</h3>
<div class="outline-text-3" id="text-6-2">
<ul class="org-ul">
<li>Bagherian2019<sub>MachineLearning.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8877e70" class="outline-3">
<h3 id="org8877e70"><span class="section-number-3">6.3</span> Abstract</h3>
<div class="outline-text-3" id="text-6-3">
<ul class="org-ul">
<li>There is a need to develop novel and efficient prediction approaches in order
to avoid costly and laborious yet not-always-deterministic experiments to
determine drug–target interactions (DTIs) by experiments alone</li>
<li>In this article, we describe the data required for the task of DTI prediction
followed by a comprehensive catalog consisting of machine learning methods and
databases, which have been proposed and utilized to predict DTIs</li>
<li>The advantages and disadvantages of each set of methods are also briefly
discussed</li>
</ul>
</div>
</div>
<div id="outline-container-org85aa865" class="outline-3">
<h3 id="org85aa865"><span class="section-number-3">6.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-6-4">
<ul class="org-ul">
<li>Almost all the methods used in DTI prediction, particularly similarity-based
methods, heavily rely on assertions concerning similar drugs and similar
targets, the type of database used for the prediction plays a significant
role.</li>
<li>Future predictions should rely on more comprehensive inter- nal databases,
which would require a significant effort to map and curate data across the
sources that utilize different ways to define, name and identify the drugs and
targets.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgbdebada" class="outline-2">
<h2 id="orgbdebada"><span class="section-number-2">7</span> <span class="done _X_">[X]</span> Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities</h2>
<div class="outline-text-2" id="text-7">
</div>
<div id="outline-container-org92291d3" class="outline-3">
<h3 id="org92291d3"><span class="section-number-3">7.1</span> Notes</h3>
<div class="outline-text-3" id="text-7-1">
<ul class="org-ul">
<li>Image processing algorithms, CNN</li>
<li>They proposed an online database of X-ray images - by now this might have been developed</li>
</ul>
</div>
</div>
<div id="outline-container-org0c14674" class="outline-3">
<h3 id="org0c14674"><span class="section-number-3">7.2</span> FileName</h3>
<div class="outline-text-3" id="text-7-2">
<ul class="org-ul">
<li>Cao2016<sub>ImprovingDiagnosticsUsingDL.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org4c1eb0a" class="outline-3">
<h3 id="org4c1eb0a"><span class="section-number-3">7.3</span> Abstract</h3>
<div class="outline-text-3" id="text-7-3">
<ul class="org-ul">
<li>The combination of machine learning and mobile computing techniques offers a
unique opportunity to accelerate the TB diagnosis among these communities</li>

<li>We first introduce major technique barriers and proposed system architecture</li>

<li>The first activity aims to develop large-scale, real-world and well-annotated
X-ray image database dedicated for automated TB screening.</li>

<li>The second research activity focus on developing effective and efficient
computational models (in particularly, deep convolutional neural networks
(CNN)-based models) to classify the image into different category of TB
manifestations</li>

<li>Our future work includes: (1) to further improve the performance of the
algorithms; and (2) to deploy our system in the city of Carabayllo in Perú, a
densely occupied urban community and high-burden TB</li>
</ul>
</div>
</div>

<div id="outline-container-org89610da" class="outline-3">
<h3 id="org89610da"><span class="section-number-3">7.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-7-4">
<ul class="org-ul">
<li>The first progress is to build a large-scale, real-world, and well-annotated chest X- ray image
database dedicated for TB screening.</li>

<li>The second progress is to develop effective and efficient computational model for TB manifestation
categorization.</li>

<li>We will continue to develop the large scale, real-world X-ray TB database, as
well as continuing to improve the performance of the computing algorithms</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgbb028f6" class="outline-2">
<h2 id="orgbb028f6"><span class="section-number-2">8</span> <span class="done DONE">DONE</span> Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data</h2>
<div class="outline-text-2" id="text-8">
<p>
EMILYN: Please confirm if similar initial data could be found for NTM then we
can work on NTM projects. Otherwise we can work on same data with different algorithm
</p>
</div>
<div id="outline-container-orgf9ac363" class="outline-3">
<h3 id="orgf9ac363"><span class="section-number-3">8.1</span> Notes</h3>
<div class="outline-text-3" id="text-8-1">
<ul class="org-ul">
<li>A good in-depth ML based paper</li>
<li>They&rsquo;ve shared high-level implementation details briefly</li>
<li>Hyperparameters are also in the supplementary material</li>
<li>I was able to find the code on Github
<a href="https://github.com/biosharp-dotnet/wdnn">https://github.com/biosharp-dotnet/wdnn</a></li>
<li>Could apply the same techniques to NTM</li>
<li>Could change the algorithm and apply on the same data</li>
<li>Could compare multiple algorithms in context of NTM
EMILYN: We could make these three projects from this paper itself.</li>
<li>The author built upon the previous paper by improving the accuracy of the ML
model - we could do the same with NTM</li>
</ul>
</div>
</div>

<div id="outline-container-org23c093c" class="outline-3">
<h3 id="org23c093c"><span class="section-number-3">8.2</span> FileName</h3>
<div class="outline-text-3" id="text-8-2">
<ul class="org-ul">
<li>Chen2018<sub>DeepLearningPredictsTuberculosis.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org832f007" class="outline-3">
<h3 id="org832f007"><span class="section-number-3">8.3</span> Abstract</h3>
<div class="outline-text-3" id="text-8-3">
<ul class="org-ul">
<li>Using targeted or whole genome sequencing and conventional drug resistance
phenotyping data from 3,601 Mycobacterium tuberculosis strains, 1,228 of which
were multidrug resistant, we implemented the first multitask deep learning
framework to predict phenotypic drug resistance to 10 anti-tubercular drugs</li>

<li><p>
The proposed wide and deep neural network (WDNN) achieved improved predictive
performance compared to regularized logistic regression and random forest
</p>

<p>
EMILYN: We can package this tool for bioconda useful
</p></li>
</ul>
</div>
</div>
<div id="outline-container-org3c2a443" class="outline-3">
<h3 id="org3c2a443"><span class="section-number-3">8.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-8-4">
<ul class="org-ul">
<li>We demonstrate that L2 regression and MD-WDNN trained on a larger diverse
dataset using a method of aggregating rare variants outperforms our previously
reported random forest model. (Chen, 2018)</li>

<li>A few prior studies have utilized algorithmic or machine learning methods
using genomic data to account for the complex relationship between genotype
and drug resistance in MTB</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org52f1ec4" class="outline-2">
<h2 id="org52f1ec4"><span class="section-number-2">9</span> <span class="done DONE">DONE</span> Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction</h2>
<div class="outline-text-2" id="text-9">
<p>
EMILYN: Check if similar data exists for NTM or if we can download the same data.
</p>
</div>
<div id="outline-container-orgb517e0b" class="outline-3">
<h3 id="orgb517e0b"><span class="section-number-3">9.1</span> Notes</h3>
<div class="outline-text-3" id="text-9-1">
<ul class="org-ul">
<li>A good in-depth ML based paper</li>
<li>Could apply the same techniques to NTM</li>
<li>Could change the algorithm and apply on the same data</li>
<li>This paper builds upon the work done by same author Chen2018</li>
<li>I was able to find the code on Github
<a href="https://github.com/biosharp-dotnet/wdnn">https://github.com/biosharp-dotnet/wdnn</a></li>
</ul>
</div>
</div>
<div id="outline-container-org7788556" class="outline-3">
<h3 id="org7788556"><span class="section-number-3">9.2</span> FileName</h3>
<div class="outline-text-3" id="text-9-2">
<ul class="org-ul">
<li>Chen2019<sub>BeyondMultiDrugResistance.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgce98bae" class="outline-3">
<h3 id="orgce98bae"><span class="section-number-3">9.3</span> Abstract</h3>
<div class="outline-text-3" id="text-9-3">
<ul class="org-ul">
<li>Whole genome sequencing of clinical Mycobacterium tuberculosis isolates
promises to circumvent the long wait times and limited scope of conventional
phenotypic antimicrobial susceptibility, but gaps remain for predicting
phenotype accurately from genotypic data especially for certain drugs.</li>

<li>We collected targeted or whole genome sequencing and conventional drug
resistance phenotyping data from 3601 Mycobacterium tuberculosis strains
enriched for resistance to first- and second-line drugs, with 1228 multidrug
resistant strains</li>

<li>We investigated the utility of (1) rare variants and variants known to be
determinants of resistance for at least one drug and (2) machine and
statistical learning architectures in predicting phenotypic drug resistance to
10 anti-tuberculosis drugs</li>
</ul>
</div>
</div>

<div id="outline-container-org35fc982" class="outline-3">
<h3 id="org35fc982"><span class="section-number-3">9.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-9-4">
<ul class="org-ul">
<li>Our approach allowed us to gain new insights into the relative im- portance of
specific genetic variants for resistance prediction</li>

<li>In summary, we present an exploration of machine learning and traditional
statistical models to identify the resistance of MTB isolates to 10
anti-tuberculosis drugs from whole genome sequencing data.</li>

<li>Our models trained on rare and common genetic variants achieved
state-of-the-art performance on a large, aggregated TB dataset, with
prediction times of less than a tenth of a second, demonstrating the efficacy
of our model as a diagnostic tool for MTB drug resistance.</li>

<li>The MD-WDNN represented the first multidrug model to our knowledge that
incorporated a high number of genotypic predictors known to be important to
determining resistance for one or more included drugs</li>
</ul>
</div>
</div>

<div id="outline-container-org28c1651" class="outline-3">
<h3 id="org28c1651"><span class="section-number-3">9.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-9-5">
<ul class="org-ul">
<li>Chen2019<sub>BeyondMultiDrugResistance</sub><sub>Supplement</sub><sub>4.xlsx</sub></li>
<li>Chen2019<sub>BeyondMultiDrugResistance</sub><sub>Supplement</sub><sub>3.xlsx</sub></li>
<li>Chen2019<sub>BeyondMultiDrugResistance</sub><sub>Supplement</sub><sub>2.xlsx</sub></li>
<li>Chen2019<sub>BeyondMultiDrugResistance</sub><sub>Supplement</sub><sub>1.docx</sub></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-orgd1db7f7" class="outline-2">
<h2 id="orgd1db7f7"><span class="section-number-2">10</span> <span class="todo ___">[-]</span> Ching2018 - Opportunities and obstacles for deep learning in biology and medicine</h2>
<div class="outline-text-2" id="text-10">
</div>
<div id="outline-container-org27736b4" class="outline-3">
<h3 id="org27736b4"><span class="section-number-3">10.1</span> Notes</h3>
<div class="outline-text-3" id="text-10-1">
<ul class="org-ul">
<li>Found the code <a href="https://github.com/greenelab/deep-review">https://github.com/greenelab/deep-review</a></li>
<li>Can use this paper to build up the background for using DL in MTB and NTM analysis</li>
</ul>
</div>
</div>
<div id="outline-container-orgab82770" class="outline-3">
<h3 id="orgab82770"><span class="section-number-3">10.2</span> FileName</h3>
<div class="outline-text-3" id="text-10-2">
<ul class="org-ul">
<li>Ching2018<sub>OpportunitiesAndObstaclesForDLInBiology.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgeae9cc2" class="outline-3">
<h3 id="orgeae9cc2"><span class="section-number-3">10.3</span> Abstract</h3>
<div class="outline-text-3" id="text-10-3">
<ul class="org-ul">
<li>Following from an extensive literature review, we find that deep learning has
yet to revolutionize biomedicine or definitively resolve any of the most
pressing challenges in the field, but promising advances have been made on the
prior state of the art</li>
<li>The limited amount of labelled data for training presents problems in some
domains, as do legal and privacy constraints on work with sensitive health
records.</li>
</ul>
</div>
</div>

<div id="outline-container-org9788566" class="outline-3">
<h3 id="org9788566"><span class="section-number-3">10.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-10-4">
<ul class="org-ul">
<li>The most challenging tasks may be those in which predictions are used directly
for downstream model- ling or decision-making, especially in the clinic. As an
example, errors in sequence variant calling will be amplified if they are used
directly for genome-wide association studies.</li>

<li>In addition, the stochasticity and complexity of biological sys- tems imply
that for some problems, for instance, predicting gene regulation in disease,
perfect accuracy will be unattainable</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org6cbfe18" class="outline-2">
<h2 id="org6cbfe18"><span class="section-number-2">11</span> <span class="todo WAIT">WAIT</span> Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data</h2>
<div class="outline-text-2" id="text-11">
</div>
<div id="outline-container-org2b56346" class="outline-3">
<h3 id="org2b56346"><span class="section-number-3">11.1</span> Notes</h3>
<div class="outline-text-3" id="text-11-1">
<ul class="org-ul">
<li>Need to request code from authors
EMILYN: Ask Jody for the code for this paper. Perhaps this code is used inside
tb-profiler and if we could improve the ML model, this could be included in tb-profiler.</li>
<li>Good ML oriented paper</li>
</ul>
</div>
</div>
<div id="outline-container-org27d6392" class="outline-3">
<h3 id="org27d6392"><span class="section-number-3">11.2</span> FileName</h3>
<div class="outline-text-3" id="text-11-2">
<ul class="org-ul">
<li>Deelder2019<sub>MLPredictsAccurately.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org856c6ab" class="outline-3">
<h3 id="org856c6ab"><span class="section-number-3">11.3</span> Abstract</h3>
<div class="outline-text-3" id="text-11-3">
<ul class="org-ul">
<li>We applied machine learning approaches to 16,688 M. tuberculosis isolates that
have undergone WGS and laboratory drug-susceptibility testing (DST) across 14
antituberculosis drugs, with 22.5% of samples being multidrug resistant and
2.1% being extensively drug resistant.</li>

<li>We fitted separate models for each drug, with and without “co-occurrent
resistance” markers known to be causing resistance to drugs other than the one
of interest. Predictive performance was measured using sensitivity,
specificity, and the area under the receiver operating characteristic curve,
assuming DST results as the gold standard.</li>

<li>Overall, the gradient-boosted-tree models performed better than the
classification-tree models.</li>

<li>Our work demonstrates the utility of machine learning as a flexible approach
to drug resistance prediction that is able to accommodate a much larger number
of predictors and to summarize their predictive ability, thus assisting
clinical decision making and single nucleotide polymorphism detection in an
era of increasing WGS data generation.</li>
</ul>
</div>
</div>

<div id="outline-container-org31498b4" class="outline-3">
<h3 id="org31498b4"><span class="section-number-3">11.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-11-4">
<ul class="org-ul">
<li>The predictive performance was highest for resistance to first-line drugs,
and multidrug-resistant tuberculosis (area under the receiver operating characteristic curve above
96%), and lowest for third- line drugs (area under the curve below 85%).</li>
</ul>
</div>
</div>

<div id="outline-container-org4acfce8" class="outline-3">
<h3 id="org4acfce8"><span class="section-number-3">11.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-11-5">
</div>
<div id="outline-container-org23f4eb8" class="outline-4">
<h4 id="org23f4eb8"><span class="section-number-4">11.5.1</span> Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-orgd1b6c19" class="outline-2">
<h2 id="orgd1b6c19"><span class="section-number-2">12</span> <span class="todo ___">[?]</span> Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-12">
</div>
<div id="outline-container-org431a354" class="outline-3">
<h3 id="org431a354"><span class="section-number-3">12.1</span> Notes</h3>
<div class="outline-text-3" id="text-12-1">
<ul class="org-ul">
<li>This requires more knowledge of chemistry and perhaps also relies on lab based experiments</li>
</ul>
</div>
</div>
<div id="outline-container-orgc6417ce" class="outline-3">
<h3 id="orgc6417ce"><span class="section-number-3">12.2</span> FileName</h3>
<div class="outline-text-3" id="text-12-2">
<ul class="org-ul">
<li>Djout2016<sub>PredictiveModeling.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgd9ccb82" class="outline-3">
<h3 id="orgd9ccb82"><span class="section-number-3">12.3</span> Abstract</h3>
<div class="outline-text-3" id="text-12-3">
<ul class="org-ul">
<li>ThyX is an essential thymidylate synthase</li>
</ul>
<p>
that is mechanistically and structurally unrelated to the human enzyme. DNA
gyrase is a DNA topoisomerase present in bacteria and plants but not animals.
The current study set out to understand the structure-activity relationships of
these targets in Mtb using a combination of cheminformatics and in vitro
screening.
</p>
</div>
</div>
<div id="outline-container-org0062525" class="outline-3">
<h3 id="org0062525"><span class="section-number-3">12.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgf9684f6" class="outline-2">
<h2 id="orgf9684f6"><span class="section-number-2">13</span> <span class="todo STRT">STRT</span> Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports</h2>
<div class="outline-text-2" id="text-13">
</div>
<div id="outline-container-orga737675" class="outline-3">
<h3 id="orga737675"><span class="section-number-3">13.1</span> Notes</h3>
<div class="outline-text-3" id="text-13-1">
<ul class="org-ul">
<li>This is a survey report about NTM across a couple decades.</li>
<li>Could be used in building up the background context
EMILYN: This could be used in discussion.</li>
</ul>
</div>
</div>
<div id="outline-container-org6ec5335" class="outline-3">
<h3 id="org6ec5335"><span class="section-number-3">13.2</span> FileName</h3>
<div class="outline-text-3" id="text-13-2">
<ul class="org-ul">
<li>Donohue2018<sub>IncreasingNTMReportingRates.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org39f1227" class="outline-3">
<h3 id="org39f1227"><span class="section-number-3">13.3</span> Abstract</h3>
<div class="outline-text-3" id="text-13-3">
<ul class="org-ul">
<li>Changes in NTM diversity were observed in complex/groups known to be
clinically significant. Between 1994 and 2014 the rate implicating M.
abscesses-chelonae group and M. avium complex increased by 322 and 149%,
respectively.</li>
</ul>
</div>
</div>
<div id="outline-container-orgd9a67c6" class="outline-3">
<h3 id="orgd9a67c6"><span class="section-number-3">13.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-13-4">
<ul class="org-ul">
<li>Knowledge of the NTM species most frequently associ- ated with adverse human
health consequences will assist epidemiological investigators to identify
likely sources and modes of exposure associated with these types of in-
fections.</li>
<li>As the U.S. population ages, the public health burden from NTM associated
disorders is expected to increase.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org4293c3b" class="outline-2">
<h2 id="org4293c3b"><span class="section-number-2">14</span> <span class="todo ___">[-]</span> Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence</h2>
<div class="outline-text-2" id="text-14">
</div>
<div id="outline-container-orgc6e6bab" class="outline-3">
<h3 id="orgc6e6bab"><span class="section-number-3">14.1</span> Notes</h3>
<div class="outline-text-3" id="text-14-1">
<ul class="org-ul">
<li>Survey paper</li>
<li>Can be used for building background context</li>
</ul>
</div>
</div>
<div id="outline-container-org922047b" class="outline-3">
<h3 id="org922047b"><span class="section-number-3">14.2</span> FileName</h3>
<div class="outline-text-3" id="text-14-2">
<ul class="org-ul">
<li>Doshi2017<sub>TuberculosisControl.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org3ab3f5b" class="outline-3">
<h3 id="org3ab3f5b"><span class="section-number-3">14.3</span> Abstract</h3>
</div>
<div id="outline-container-org0f65e4f" class="outline-3">
<h3 id="org0f65e4f"><span class="section-number-3">14.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org6cc35d9" class="outline-2">
<h2 id="org6cc35d9"><span class="section-number-2">15</span> <span class="done _X_">[X]</span> Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models</h2>
<div class="outline-text-2" id="text-15">
</div>
<div id="outline-container-orgc34f6de" class="outline-3">
<h3 id="orgc34f6de"><span class="section-number-3">15.1</span> Notes</h3>
<div class="outline-text-3" id="text-15-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org46e54cb" class="outline-3">
<h3 id="org46e54cb"><span class="section-number-3">15.2</span> FileName</h3>
<div class="outline-text-3" id="text-15-2">
<ul class="org-ul">
<li>Ekins2013<sub>EnhancingHitIdentification.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org69459be" class="outline-3">
<h3 id="org69459be"><span class="section-number-3">15.3</span> Abstract</h3>
<div class="outline-text-3" id="text-15-3">
<ul class="org-ul">
<li>High-throughput screening (HTS) in whole cells is widely pursued to find
compounds active against Mycobacterium tuberculosis (Mtb) for further
development towards new tuberculosis (TB) drugs</li>
<li>We have recently built and validated Bayesian machine learning models for
predicting compounds with activity against Mtb based on publicly available
large-scale HTS data from the Tuberculosis Antimicrobial Acquisition
Coordinating Facility</li>
</ul>
</div>
</div>
<div id="outline-container-org1427865" class="outline-3">
<h3 id="org1427865"><span class="section-number-3">15.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgeec8a60" class="outline-2">
<h2 id="orgeec8a60"><span class="section-number-2">16</span> <span class="done _X_">[X]</span> Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation</h2>
<div class="outline-text-2" id="text-16">
</div>
<div id="outline-container-orgc086304" class="outline-3">
<h3 id="orgc086304"><span class="section-number-3">16.1</span> Notes</h3>
<div class="outline-text-3" id="text-16-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org5023c9c" class="outline-3">
<h3 id="org5023c9c"><span class="section-number-3">16.2</span> FileName</h3>
<div class="outline-text-3" id="text-16-2">
<ul class="org-ul">
<li>Ekins2013<sub>FusingDualEvent.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org75e3d14" class="outline-3">
<h3 id="org75e3d14"><span class="section-number-3">16.3</span> Abstract</h3>
<div class="outline-text-3" id="text-16-3">
<ul class="org-ul">
<li>A cheminformatics data-fusion</li>
</ul>
<p>
approach followed by Bayesian machine learning, Support Vector Machine, or Recursive
Partitioning model development (based on publicly available Mtb screening data) was used to
compare individual data sets and subsequent combined models.
</p>
<ul class="org-ul">
<li>We demonstrate that</li>
</ul>
<p>
combining three data sets incorporating antitubercular and cytotoxicity data in Vero cells
from our previous screens results in external validation receiver operator curve (ROC) of
0.83 (Bayesian or RP Forest)
</p>
</div>
</div>
<div id="outline-container-orga7b7477" class="outline-3">
<h3 id="orga7b7477"><span class="section-number-3">16.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org0fb76b4" class="outline-2">
<h2 id="org0fb76b4"><span class="section-number-2">17</span> <span class="done _X_">[X]</span> Ekins2014<sub>AreBiggerDataSets.pdf</sub></h2>
<div class="outline-text-2" id="text-17">
</div>
<div id="outline-container-org051540e" class="outline-3">
<h3 id="org051540e"><span class="section-number-3">17.1</span> Notes</h3>
<div class="outline-text-3" id="text-17-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-orgd01e027" class="outline-3">
<h3 id="orgd01e027"><span class="section-number-3">17.2</span> Title</h3>
</div>
<div id="outline-container-orgf84f2d8" class="outline-3">
<h3 id="orgf84f2d8"><span class="section-number-3">17.3</span> Abstract</h3>
</div>
<div id="outline-container-org07c827c" class="outline-3">
<h3 id="org07c827c"><span class="section-number-3">17.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org914fdf0" class="outline-2">
<h2 id="org914fdf0"><span class="section-number-2">18</span> <span class="done _X_">[X]</span> Ekins2014<sub>CombiningComputationalMethods.pdf</sub></h2>
<div class="outline-text-2" id="text-18">
</div>
<div id="outline-container-org0762453" class="outline-3">
<h3 id="org0762453"><span class="section-number-3">18.1</span> Notes</h3>
<div class="outline-text-3" id="text-18-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-orgc060b52" class="outline-3">
<h3 id="orgc060b52"><span class="section-number-3">18.2</span> Title</h3>
</div>
<div id="outline-container-org4c07fa7" class="outline-3">
<h3 id="org4c07fa7"><span class="section-number-3">18.3</span> Abstract</h3>
</div>
<div id="outline-container-org6ca8159" class="outline-3">
<h3 id="org6ca8159"><span class="section-number-3">18.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org03d038c" class="outline-2">
<h2 id="org03d038c"><span class="section-number-2">19</span> <span class="done _X_">[X]</span> Ekins2016<sub>MachineLearningModel.pdf</sub></h2>
<div class="outline-text-2" id="text-19">
</div>
<div id="outline-container-org6e8da0a" class="outline-3">
<h3 id="org6e8da0a"><span class="section-number-3">19.1</span> Notes</h3>
<div class="outline-text-3" id="text-19-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org2df7156" class="outline-3">
<h3 id="org2df7156"><span class="section-number-3">19.2</span> Title</h3>
</div>
<div id="outline-container-orgc6b6a47" class="outline-3">
<h3 id="orgc6b6a47"><span class="section-number-3">19.3</span> Abstract</h3>
</div>
<div id="outline-container-orgc708f08" class="outline-3">
<h3 id="orgc708f08"><span class="section-number-3">19.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org31dc89c" class="outline-2">
<h2 id="org31dc89c"><span class="section-number-2">20</span> <span class="done _X_">[X]</span> Ekins2017<sub>MachineLearning.pdf</sub></h2>
<div class="outline-text-2" id="text-20">
</div>
<div id="outline-container-org77271bc" class="outline-3">
<h3 id="org77271bc"><span class="section-number-3">20.1</span> Notes</h3>
<div class="outline-text-3" id="text-20-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org49a22de" class="outline-3">
<h3 id="org49a22de"><span class="section-number-3">20.2</span> Title</h3>
</div>
<div id="outline-container-org9219c8a" class="outline-3">
<h3 id="org9219c8a"><span class="section-number-3">20.3</span> Abstract</h3>
</div>
<div id="outline-container-org6d5d88c" class="outline-3">
<h3 id="org6d5d88c"><span class="section-number-3">20.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org0b14e0a" class="outline-2">
<h2 id="org0b14e0a"><span class="section-number-2">21</span> <span class="todo ___">[-]</span> Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis</h2>
<div class="outline-text-2" id="text-21">
</div>
<div id="outline-container-orgf45bb2e" class="outline-3">
<h3 id="orgf45bb2e"><span class="section-number-3">21.1</span> Notes</h3>
<div class="outline-text-3" id="text-21-1">
<ul class="org-ul">
<li>Image processing</li>
<li>Mentiones the database source, NIH</li>
<li>Has CSV as supplementary data</li>
</ul>
</div>
</div>
<div id="outline-container-orgbf3c921" class="outline-3">
<h3 id="orgbf3c921"><span class="section-number-3">21.2</span> FileName</h3>
<div class="outline-text-3" id="text-21-2">
<ul class="org-ul">
<li>Engle2019<sub>PerformanceOfQureAI.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org15feb0d" class="outline-3">
<h3 id="org15feb0d"><span class="section-number-3">21.3</span> Abstract</h3>
<div class="outline-text-3" id="text-21-3">
<ul class="org-ul">
<li>Availability of trained radiologists for fast processing of CXRs in regions
burdened with tuberculosis always has been a challenge, affecting both timely
diagnosis and patient monitoring.</li>

<li>The Qure.ai qXR arti- ficial intelligence automated CXR interpretation tool,
was blind-tested on the 346 radiologist- annotated CXRs from the TBPP
database.</li>

<li>We conclude that applying fully automated Qure.ai CXR analysis tool is useful
for fast, accurate, uniform, large-scale CXR annotation assistance, as it
performed well even for DR-TB cases that were not used for initial training.</li>
</ul>
</div>
</div>

<div id="outline-container-orgec53136" class="outline-3">
<h3 id="orgec53136"><span class="section-number-3">21.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-21-4">
<ul class="org-ul">
<li>We demonstrated that the Qure.ai qXR CXR annotations for cavity, nodule,
pleural effusion, and hilar lymphadenopathy, are in statistical agreement with
radiologist CXR annotations.</li>

<li>A corresponding analysis of patient case matched
CT image annotations recorded within 30 days of a CXR supports this result,
demonstrating statistical significance for cavity, pleural effusion, hilar
lymphadenopathy, and atelectasis.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org95a0312" class="outline-2">
<h2 id="org95a0312"><span class="section-number-2">22</span> <span class="todo ___">[-]</span> Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay</h2>
<div class="outline-text-2" id="text-22">
</div>
<div id="outline-container-org01eb91e" class="outline-3">
<h3 id="org01eb91e"><span class="section-number-3">22.1</span> Notes</h3>
<div class="outline-text-3" id="text-22-1">
<ul class="org-ul">
<li>Survey paper</li>
</ul>
</div>
</div>
<div id="outline-container-orgd52acda" class="outline-3">
<h3 id="orgd52acda"><span class="section-number-3">22.2</span> FileName</h3>
<div class="outline-text-3" id="text-22-2">
<ul class="org-ul">
<li>Greif2020<sub>FrequencyOfNTM.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org5dc068e" class="outline-3">
<h3 id="org5dc068e"><span class="section-number-3">22.3</span> Abstract</h3>
<div class="outline-text-3" id="text-22-3">
<ul class="org-ul">
<li>Nontuberculous mycobacteria (NTM) increasingly are recognized as
opportunistic pathogens of humans.</li>
<li>NTM species distribution is well documented in Europe and North America</li>
</ul>
</div>
</div>
<div id="outline-container-org841c69b" class="outline-3">
<h3 id="org841c69b"><span class="section-number-3">22.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-22-4">
<ul class="org-ul">
<li>Our findings indicate that rate of NTM have risen nearly 5-fold (4.79) in
Uruguay, from 0.33 cas- es/100,000 inhabitants in 2006 to 1.57 cases/100,000
inhabitants in 2018.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgb86575d" class="outline-2">
<h2 id="orgb86575d"><span class="section-number-2">23</span> <span class="todo ___">[-]</span> Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data</h2>
<div class="outline-text-2" id="text-23">
</div>
<div id="outline-container-orgd388bd6" class="outline-3">
<h3 id="orgd388bd6"><span class="section-number-3">23.1</span> Notes</h3>
<div class="outline-text-3" id="text-23-1">
<ul class="org-ul">
<li>Can perform the mentioned analysis on the MTB and NTM data</li>
<li>The paper focuses on the factors affecting the application and clinical application of ML models</li>
</ul>
</div>
</div>
<div id="outline-container-org8e6b88d" class="outline-3">
<h3 id="org8e6b88d"><span class="section-number-3">23.2</span> FileName</h3>
<div class="outline-text-3" id="text-23-2">
<ul class="org-ul">
<li>Hicks2019<sub>EvaluationOfParameters.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org1f6dc26" class="outline-3">
<h3 id="org1f6dc26"><span class="section-number-3">23.3</span> Abstract</h3>
<div class="outline-text-3" id="text-23-3">
<ul class="org-ul">
<li>There has been no systematic evaluation of factors that may influence
performance of ML models, how they might apply to and vary across clinical
populations, and what the implications might be in the clinical setting</li>

<li>We performed a meta-analysis of seven large Neisseria gonorrhoeae datasets, as
well as Klebsiella pneumoniae and Acinetobacter baumannii datasets, with whole
genome sequence data and antibiotic susceptibility phenotypes using set
covering machine classifi- cation, random forest classification, and random
forest regression models to predict resis- tance phenotypes from genotype.</li>

<li>We demonstrate how model performance varies by drug, dataset, resistance
metric, and species, reflecting the complexities of generating clinically
relevant conclusions from machine learning-derived models.</li>

<li>Our results highlight significant biological and technical challenges
facing the application of machine learning-based prediction of antibiotic
resistance as a diagnostic tool.</li>
</ul>
</div>
</div>

<div id="outline-container-org7f814aa" class="outline-3">
<h3 id="org7f814aa"><span class="section-number-3">23.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgcc12666" class="outline-2">
<h2 id="orgcc12666"><span class="section-number-2">24</span> <span class="done _X_">[X]</span> Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models</h2>
<div class="outline-text-2" id="text-24">
</div>
<div id="outline-container-org49f1e94" class="outline-3">
<h3 id="org49f1e94"><span class="section-number-3">24.1</span> Notes</h3>
<div class="outline-text-3" id="text-24-1">
<ul class="org-ul">
<li>Can use a different algorithm (XgBoost) on the dataset</li>
<li>Sample data and complete code is shared
<a href="https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics">https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics</a></li>
<li>Need to find the full dataset of 4000 surveyed patients
EMILYN: Please try to find the data</li>
</ul>
</div>
</div>
<div id="outline-container-org88b6ba4" class="outline-3">
<h3 id="org88b6ba4"><span class="section-number-3">24.2</span> FileName</h3>
<div class="outline-text-3" id="text-24-2">
<ul class="org-ul">
<li>Hussain2018<sub>PredictingTreatmentOutcomes.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org111d250" class="outline-3">
<h3 id="org111d250"><span class="section-number-3">24.3</span> Abstract</h3>
<div class="outline-text-3" id="text-24-3">
<ul class="org-ul">
<li>Tuberculosis (TB) is a deadly contagious disease and a serious global health
problem. It is curable but due to its lengthy treatment process, a patient is
likely to leave the treatment incomplete, leading to a more lethal, drug
resistant form of disease.</li>

<li>We aim to facilitate TB programs by predicting the outcome of the treatment of
a particular patient at the start of treatment so that their health workers
can be utilized in a targeted and cost-effective way.</li>

<li>The problem was modeled as a classification problem, and the outcome of
treatment was predicted using state-of-art implementations of 3 machine learning algorithms</li>
</ul>
</div>
</div>

<div id="outline-container-org911b57e" class="outline-3">
<h3 id="org911b57e"><span class="section-number-3">24.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-24-4">
<ul class="org-ul">
<li>Empirical results using data of more than 4000 patients reveal that we can
predict with more than 76% accuracy whether a TB patient would complete his (or her) treatment or not</li>
<li></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-org42f08c9" class="outline-2">
<h2 id="org42f08c9"><span class="section-number-2">25</span> <span class="todo WAIT">WAIT</span> Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-25">
<p>
EMILYN: This is interesting and will read this paper. Perhaps can find/propose
other projects and studies
</p>
</div>
<div id="outline-container-orge988dcc" class="outline-3">
<h3 id="orge988dcc"><span class="section-number-3">25.1</span> Notes</h3>
<div class="outline-text-3" id="text-25-1">
<ul class="org-ul">
<li>Used many ML algorithms, 85% accuracy</li>
<li>Could use XgBoost</li>
<li>Could conduct a similar study for NTM
EMILYN: This can&rsquo;t be used for NTM as drug database doesn&rsquo;t exist.</li>
<li>Mentions many online bio databases</li>
<li>Touches on BioChemistry</li>
</ul>
</div>
</div>
<div id="outline-container-orgef78b44" class="outline-3">
<h3 id="orgef78b44"><span class="section-number-3">25.2</span> FileName</h3>
<div class="outline-text-3" id="text-25-2">
<ul class="org-ul">
<li>Jamal2020<sub>AIAndMLBasedPredictive.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org3b7214d" class="outline-3">
<h3 id="org3b7214d"><span class="section-number-3">25.3</span> Abstract</h3>
<div class="outline-text-3" id="text-25-3">
<ul class="org-ul">
<li>The present work is a computational framework that uses artificial
intelligence (AI) based machine learning (ML) approaches for predicting
resistance in the genes rpoB, inhA, katG, pncA, gyrA and gyrB for the drugs
rifampicin, isoniazid, pyrazinamide and fluoroquinolones</li>

<li>The single nucleotide variations were represented by several sequence and
structural features that indicate the influence of mutations on the target
protein coded by each gene.</li>
</ul>
</div>
</div>

<div id="outline-container-orgcf3b80d" class="outline-3">
<h3 id="orgcf3b80d"><span class="section-number-3">25.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-25-4">
<ul class="org-ul">
<li>The present study proposes an AI/ML method to classify resistant and susceptible
mutations in TB and predict novel resistance conferring mutations.</li>

<li>The impact of the reported mutations was captured in the form of changes in the amino acid
residues, and the consequent change in properties vis-a-vis wild-type and mutant
proteins and represented as features used to train the models.</li>

<li>The classification model was generated for each gene and predictions were made for
SNVs linked with each gene for each drug</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgf8112f4" class="outline-2">
<h2 id="orgf8112f4"><span class="section-number-2">26</span> <span class="todo WAIT">WAIT</span> Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance</h2>
<div class="outline-text-2" id="text-26">
<p>
EMILYN: Please read this paper.
</p>
</div>
<div id="outline-container-orgf7aa7be" class="outline-3">
<h3 id="orgf7aa7be"><span class="section-number-3">26.1</span> Notes</h3>
<div class="outline-text-3" id="text-26-1">
<ul class="org-ul">
<li>They developed and online platform</li>

<li>Can be applied to NTM</li>

<li>Code is available
<a href="https://github.com/biosharp-dotnet/microbial_AMR_ML">https://github.com/biosharp-dotnet/microbial_AMR_ML</a></li>
</ul>
</div>
</div>
<div id="outline-container-orgd3857ee" class="outline-3">
<h3 id="orgd3857ee"><span class="section-number-3">26.2</span> FileName</h3>
<div class="outline-text-3" id="text-26-2">
<ul class="org-ul">
<li>Kavvas2018<sub>MachineLearning</sub><sub>.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org37851bc" class="outline-3">
<h3 id="org37851bc"><span class="section-number-3">26.3</span> Abstract</h3>
<div class="outline-text-3" id="text-26-3">
<ul class="org-ul">
<li>The many publicly available datasets describing its AMR characteristics demand
disparate data-type analyses.</li>

<li>Here, we develop a reference strain-agnostic computational platform that uses
machine learning approaches, complemented by both genetic interaction analysis
and 3D structural mutation-mapping, to identify signatures of AMR evolution to
13 antibiotics.</li>

<li>First, a pan-genome analysis shows that M. tuberculosis is highly conserved
with sequenced variation concentrated in PE/PPE/PGRS genes.</li>

<li>Second, the platform corroborates 33 genes known to confer resistance and identifies 24
new genetic signatures of AMR.</li>

<li>Third, 97 epistatic interactions across 10 resistance classes are revealed.</li>

<li>Fourth, detailed structural analysis of these
genes yields mechanistic bases for their selection.</li>

<li>The platform can be used to study other human pathogens</li>
</ul>
</div>
</div>

<div id="outline-container-orgcb5405a" class="outline-3">
<h3 id="orgcb5405a"><span class="section-number-3">26.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-26-4">
<ul class="org-ul">
<li>The platform was applied to 1595 sequenced strains to yield results in four
categories: pan-genome properties, identification of genes conferring
antibiotic resistance, their epistatic interactions, and protein structure
based mechanistic insights.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org55dfe62" class="outline-2">
<h2 id="org55dfe62"><span class="section-number-2">27</span> <span class="todo ___">[-]</span> Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes</h2>
<div class="outline-text-2" id="text-27">
</div>
<div id="outline-container-org95ca24c" class="outline-3">
<h3 id="org95ca24c"><span class="section-number-3">27.1</span> Notes</h3>
<div class="outline-text-3" id="text-27-1">
<ul class="org-ul">
<li>Uses multiple bio and ML packages in R</li>
<li>Doesn&rsquo;t use XgBoost or H20</li>
<li>Code hasn&rsquo;t been shared</li>
</ul>
</div>
</div>
<div id="outline-container-org7bcf5f5" class="outline-3">
<h3 id="org7bcf5f5"><span class="section-number-3">27.2</span> FileName</h3>
<div class="outline-text-3" id="text-27-2">
<ul class="org-ul">
<li>Khanna2018<sub>EnsembleTechnique.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org4ba203e" class="outline-3">
<h3 id="org4ba203e"><span class="section-number-3">27.3</span> Abstract</h3>
<div class="outline-text-3" id="text-27-3">
<ul class="org-ul">
<li>Development of an effective machine-learning model for T-cell Mycobacterium
tuberculosis (M. tuberculosis) epitopes is beneficial for saving biologist’s
time and effort for identifying epitope in a targeted antigen.</li>

<li>The proposed method is able to work with variable length epitopes having size
even greater than 9-mers. Identification of T-cell or B-cell epitopes in the
targeted antigen is the main goal in designing epitope-based vaccine,
immune-diagnostic tests and antibody production</li>

<li>The proposed ensemble model achieves 82.0% accuracy, 0.89 specificity, 0.77
sensitivity with repeated k-fold cross-validation having average accuracy of
80.61%.</li>
</ul>
</div>
</div>

<div id="outline-container-org9180cf6" class="outline-3">
<h3 id="org9180cf6"><span class="section-number-3">27.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-27-4">
<ul class="org-ul">
<li>Three models (decision tree, avNNet and RRF) have been used to design the
ensemble model which produces high accuracy, Gini, AUC, specificity and
sensitivity.</li>

<li>To validate and compare the proposed ensemble model, NetMHC 4.0, NetMHC 2.3
and CTLpred servers have been considered.</li>
</ul>
</div>
</div>
</div>


<div id="outline-container-org41a77f7" class="outline-2">
<h2 id="org41a77f7"><span class="section-number-2">28</span> <span class="todo ___">[-]</span> Killian2019 - Learning to Prescribe Interventions for Tuberculosis Patients Using Digital Adherence Data</h2>
<div class="outline-text-2" id="text-28">
</div>
<div id="outline-container-org62f9cb1" class="outline-3">
<h3 id="org62f9cb1"><span class="section-number-3">28.1</span> Notes</h3>
<div class="outline-text-3" id="text-28-1">
<ul class="org-ul">
<li>Wasn&rsquo;t able to find the data or the code in the paper</li>
</ul>
</div>
</div>
<div id="outline-container-org2d8d3a1" class="outline-3">
<h3 id="org2d8d3a1"><span class="section-number-3">28.2</span> FileName</h3>
<div class="outline-text-3" id="text-28-2">
<ul class="org-ul">
<li>Killian2019<sub>LearningToPrescribe.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgf7b752c" class="outline-3">
<h3 id="orgf7b752c"><span class="section-number-3">28.3</span> Abstract</h3>
<div class="outline-text-3" id="text-28-3">
<ul class="org-ul">
<li>We analyze data from one city served by 99DOTS, a phone-call- based DAT
deployed for Tuberculosis (TB) treatment in India where nearly 3 million
people are afflicted with the disease each year</li>

<li>We then construct a deep learning model, demonstrate its interpretability, and
show how it can be adapted and trained in three different clinical scenarios
to better target and improve patient care.</li>
</ul>
</div>
</div>

<div id="outline-container-orge1c8c9a" class="outline-3">
<h3 id="orge1c8c9a"><span class="section-number-3">28.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-28-4">
<ul class="org-ul">
<li>We finally show that tailoring our model for a specific intervention via
decision-focused learning can improve performance by a further 15%.</li>
</ul>
</div>
</div>
</div>
<div id="outline-container-org799097a" class="outline-2">
<h2 id="org799097a"><span class="section-number-2">29</span> <span class="todo WAIT">WAIT</span> Kouchaki2019 - Application of machine learning techniques to tuberculosis drug resistance analysis</h2>
<div class="outline-text-2" id="text-29">
</div>
<div id="outline-container-org106060c" class="outline-3">
<h3 id="org106060c"><span class="section-number-3">29.1</span> Notes</h3>
<div class="outline-text-3" id="text-29-1">
<ul class="org-ul">
<li>I&rsquo;ve asked the author to provide the source code via ResearchGate</li>
<li>They&rsquo;ve used 6 ML classifiers, not XgBoost or NeuralNetworks</li>
</ul>
</div>
</div>
<div id="outline-container-org94270d2" class="outline-3">
<h3 id="org94270d2"><span class="section-number-3">29.2</span> FileName</h3>
<div class="outline-text-3" id="text-29-2">
<ul class="org-ul">
<li>Kouchaki2019<sub>ApplicationOfML.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orga069eaa" class="outline-3">
<h3 id="orga069eaa"><span class="section-number-3">29.3</span> Abstract</h3>
<div class="outline-text-3" id="text-29-3">
<ul class="org-ul">
<li>Machine learning methods have been widely applied for timely predicting
resistance of MTB given a specific drug and identifying resistance markers.
However, they have been not validated on a large cohort of MTB samples from
multi-centers across the world in terms of resistance prediction and
resistance marker identification</li>

<li>Several machine learning classifiers and linear dimension re- duction
techniques were developed and compared for a cohort of 13 402 isolates
collected from 16 countries across 6 continents and tested 11 drugs.</li>
</ul>
</div>
</div>

<div id="outline-container-orge7c29f7" class="outline-3">
<h3 id="orge7c29f7"><span class="section-number-3">29.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-29-4">
<ul class="org-ul">
<li>Developing machine learning techniques is especially important to im- prove TB
resistance prediction, mainly sensitivity for less-studied drugs.</li>
<li></li>
</ul>
</div>
</div>
<div id="outline-container-org5eb2280" class="outline-3">
<h3 id="org5eb2280"><span class="section-number-3">29.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-29-5">
</div>
<div id="outline-container-org7c6a8fc" class="outline-4">
<h4 id="org7c6a8fc"><span class="section-number-4">29.5.1</span> Kouchaki2019<sub>ApplicationOfML</sub><sub>Supplementary.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-orga83158f" class="outline-2">
<h2 id="orga83158f"><span class="section-number-2">30</span> <span class="done DONE">DONE</span> Kouchaki2020 - Multi-Label Random Forest Model for Tuberculosis Drug Resistance Classification and Mutation Ranking</h2>
<div class="outline-text-2" id="text-30">
<p>
EMILYN: Please think of some projects. Maybe take hints from Notes
</p>
</div>
<div id="outline-container-org9be3362" class="outline-3">
<h3 id="org9be3362"><span class="section-number-3">30.1</span> Notes</h3>
<div class="outline-text-3" id="text-30-1">
<ul class="org-ul">
<li>Source code and data are both available</li>

<li>Maybe use XgBoost or H2O</li>
</ul>
</div>
</div>
<div id="outline-container-org7f3c9bb" class="outline-3">
<h3 id="org7f3c9bb"><span class="section-number-3">30.2</span> FileName</h3>
<div class="outline-text-3" id="text-30-2">
<ul class="org-ul">
<li>Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org0ba967d" class="outline-3">
<h3 id="org0ba967d"><span class="section-number-3">30.3</span> Abstract</h3>
<div class="outline-text-3" id="text-30-3">
<ul class="org-ul">
<li>Due to standard regimens for the use of first-line antibiotics, resistance
co-occurrence, in which samples are resistant to multiple drugs, is common.
Analysing all drugs simultaneously should therefore enable patterns reflecting
resistance co-occurrence to be exploited for resistance prediction</li>

<li>Here, multi-label random forest (MLRF) models are compared with single-label
random forest (SLRF) for both predicting phenotypic resistance from whole
genome sequences and identifying important mutations for better prediction of
four first-line drugs in a dataset of 13402 Mycobacterium tuberculosis
isolates.</li>
</ul>
</div>
</div>

<div id="outline-container-orgae759f4" class="outline-3">
<h3 id="orgae759f4"><span class="section-number-3">30.4</span> Conclusion</h3>
</div>
<div id="outline-container-org04124da" class="outline-3">
<h3 id="org04124da"><span class="section-number-3">30.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-30-5">
</div>
<div id="outline-container-org1008759" class="outline-4">
<h4 id="org1008759"><span class="section-number-4">30.5.1</span> Kouchaki2020<sub>MultiLabel.zip</sub></h4>
</div>
<div id="outline-container-orga9b14aa" class="outline-4">
<h4 id="orga9b14aa"><span class="section-number-4">30.5.2</span> Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub></h4>
</div>
<div id="outline-container-orgf4cd826" class="outline-4">
<h4 id="orgf4cd826"><span class="section-number-4">30.5.3</span> Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-org3123fae" class="outline-2">
<h2 id="org3123fae"><span class="section-number-2">31</span> <span class="todo ___">[-]</span> Kovalishyn2019 - Hybrid Design of Isonicotinic Acid Hydrazide Derivatives: Machine Learning Studies, Synthesis and Biological Evaluation of their Anti- tuberculosis Activity</h2>
<div class="outline-text-2" id="text-31">
</div>
<div id="outline-container-orgdc0d077" class="outline-3">
<h3 id="orgdc0d077"><span class="section-number-3">31.1</span> Notes</h3>
<div class="outline-text-3" id="text-31-1">
<ul class="org-ul">
<li>Is inclined more towards Chemistry (reagents) wrt TB</li>
</ul>
</div>
</div>
<div id="outline-container-org36706ac" class="outline-3">
<h3 id="org36706ac"><span class="section-number-3">31.2</span> FileName</h3>
<div class="outline-text-3" id="text-31-2">
<ul class="org-ul">
<li>Kovalishyn2019<sub>HybridDesign.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8d36144" class="outline-3">
<h3 id="org8d36144"><span class="section-number-3">31.3</span> Abstract</h3>
<div class="outline-text-3" id="text-31-3">
<ul class="org-ul">
<li>The goal of this work is to identify potent small molecule anti-TB agents by
machine learning, synthesis and biological evaluation.</li>

<li>The On-line Chemical Database and Modeling Environment (OCHEM) was used to
build pre- dictive machine learning models. Seven compounds were synthesized
and tested in vitro for their an- titubercular activity against H37Rv and
resistant Mtb strains.</li>

<li></li>
</ul>
</div>
</div>
<div id="outline-container-org7a94d65" class="outline-3">
<h3 id="org7a94d65"><span class="section-number-3">31.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-31-4">
<ul class="org-ul">
<li>The goal of this work was to investigate isoniazid- thiazole hybrid compounds
taking into consideration the key structural futures known to be important for
their antituber- cular activity.</li>

<li>The created binary classification models demonstrated good balanced accuracy.
The predictive power of the binary classifiers was tested by a 5-fold cross-
validation and resulted in balanced accuracies of 61–78 %. Seven new compounds
were synthesized and their activity against Mtb H37Rv and MDR strains was
evaluated at two concentrations (1.5 and 50 μМ) using rifampin and isoniazid
as controls.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org901ec89" class="outline-2">
<h2 id="org901ec89"><span class="section-number-2">32</span> <span class="todo ___">[-]</span> Kumari2019 - A genetic programming-based approach to identify potential inhibitors of serine protease of Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-32">
</div>
<div id="outline-container-org537b1b7" class="outline-3">
<h3 id="org537b1b7"><span class="section-number-3">32.1</span> Notes</h3>
<div class="outline-text-3" id="text-32-1">
<ul class="org-ul">
<li>Uses Genetic algorithms and ML algorithms as well</li>
<li>Data or code isn&rsquo;t available</li>
</ul>
</div>
</div>
<div id="outline-container-org3bffb78" class="outline-3">
<h3 id="org3bffb78"><span class="section-number-3">32.2</span> FileName</h3>
<div class="outline-text-3" id="text-32-2">
<ul class="org-ul">
<li>Kumari2019<sub>AGeneticProgramming.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org5dafb05" class="outline-3">
<h3 id="org5dafb05"><span class="section-number-3">32.3</span> Abstract</h3>
<div class="outline-text-3" id="text-32-3">
<ul class="org-ul">
<li>We applied genetic programming approaches to understand the impact of
descriptors on inhibitory effects of serine protease inhibitors of
Mycobacterium tuberculosis (Mtb) and the discovery of new in- hibitors as drug
candidates</li>

<li>The experimental dataset of serine protease inhibitors of Mtb descriptors was
optimized by genetic algorithm (GA) along with the correlation-based feature
selection (CFS) in order to develop predictive models using machine-learning
algorithms.</li>

<li>The use of CFS-GA with random forest-enhanced classification accuracy and
predicted new serine protease inhibitors of Mtb, which can be used for better
drug development against tuberculosis.</li>
</ul>
</div>
</div>

<div id="outline-container-org513c1c3" class="outline-3">
<h3 id="org513c1c3"><span class="section-number-3">32.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-32-4">
<ul class="org-ul">
<li>Comparative analysis of statistical measures for four different
machine-learning algorithms revealed that RF models with CFS-GA performed
better than the others.</li>

<li>The best RF models with CFS-GA predicted new serine protease inhibitors of
Mtb, which can be used for better drug development against tuberculosis</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgf3a0f5c" class="outline-2">
<h2 id="orgf3a0f5c"><span class="section-number-2">33</span> <span class="todo ___">[-]</span> Lai2019 - Comparison of the Predictive Outcomes for Anti-Tuberculosis Drug-Induced Hepatotoxicity by Different Machine Learning Techniques</h2>
<div class="outline-text-2" id="text-33">
</div>
<div id="outline-container-org775523b" class="outline-3">
<h3 id="org775523b"><span class="section-number-3">33.1</span> Notes</h3>
<div class="outline-text-3" id="text-33-1">
<ul class="org-ul">
<li>Good ML oriented paper</li>
<li>Can&rsquo;t find the data or code</li>
</ul>
</div>
</div>
<div id="outline-container-org48be139" class="outline-3">
<h3 id="org48be139"><span class="section-number-3">33.2</span> FileName</h3>
<div class="outline-text-3" id="text-33-2">
<ul class="org-ul">
<li>Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8788a2a" class="outline-3">
<h3 id="org8788a2a"><span class="section-number-3">33.3</span> Abstract</h3>
<div class="outline-text-3" id="text-33-3">
<ul class="org-ul">
<li>The study compared the performance of artificial neural network, support
vector machine and random forest on predicting anti-tuberculosis drugs induced
hepatotoxicity.</li>

<li>Combining genomic and clinical data can further increase the area under
receiver operating characteristic curve than using genomic or clinical data
alone.</li>

<li>Nine models were created with 7 clinical factors and 4 genotypes</li>
</ul>
</div>
</div>

<div id="outline-container-orgc2da543" class="outline-3">
<h3 id="orgc2da543"><span class="section-number-3">33.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orge57d55e" class="outline-2">
<h2 id="orge57d55e"><span class="section-number-2">34</span> <span class="todo ___">[-]</span> Lane2018 - Comparing and Validating Machine Learning Models for Mycobacterium tuberculosis Drug Discovery</h2>
<div class="outline-text-2" id="text-34">
</div>
<div id="outline-container-org9a3c98a" class="outline-3">
<h3 id="org9a3c98a"><span class="section-number-3">34.1</span> Notes</h3>
<div class="outline-text-3" id="text-34-1">
<ul class="org-ul">
<li>Involves chemistry</li>
</ul>
</div>
</div>
<div id="outline-container-org021950e" class="outline-3">
<h3 id="org021950e"><span class="section-number-3">34.2</span> FileName</h3>
<div class="outline-text-3" id="text-34-2">
<ul class="org-ul">
<li>Lane2018<sub>ComparingAndValidating.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orge96fb12" class="outline-3">
<h3 id="orge96fb12"><span class="section-number-3">34.3</span> Abstract</h3>
<div class="outline-text-3" id="text-34-3">
<ul class="org-ul">
<li>The need to develop new treatments for those infected with Mycobacterium
tuberculosis (Mtb) has led to many large-scale phenotypic screens and many
thousands of new active compounds identified in vitro</li>

<li>We have curated small molecule Mtb data and developed new models with a total
of 18,886 molecules with activity cutoffs of 10 μM, 1 μM, and 100 nM. These
data sets were used to evaluate different machine learning methods (including
deep learning) and metrics and to generate predictions for additional
molecules published in 2017</li>

<li>We have also compared these models with additional machine learning algorithms
showing Bayesian machine learning models con- structed with literature Mtb
data generated by different laboratories generally were equivalent to or
outperformed deep neural networks with external test sets.</li>
</ul>
</div>
</div>

<div id="outline-container-orgefed891" class="outline-3">
<h3 id="orgefed891"><span class="section-number-3">34.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgfe6b046" class="outline-2">
<h2 id="orgfe6b046"><span class="section-number-2">35</span> <span class="done _X_">[X]</span> Lin2001 - Conserved codon composition of ribosomal protein coding genes in Escherichia coli, Mycobacterium tuberculosis and Saccharomyces cerevisiae: lessons from supervised machine learning in functional genomics</h2>
<div class="outline-text-2" id="text-35">
</div>
<div id="outline-container-orgc2a806b" class="outline-3">
<h3 id="orgc2a806b"><span class="section-number-3">35.1</span> Notes</h3>
<div class="outline-text-3" id="text-35-1">
<ul class="org-ul">
<li>Deals with codons and homologs etc. No idea.</li>
</ul>
</div>
</div>
<div id="outline-container-orgee3a42b" class="outline-3">
<h3 id="orgee3a42b"><span class="section-number-3">35.2</span> FileName</h3>
<div class="outline-text-3" id="text-35-2">
<ul class="org-ul">
<li>Lin2001<sub>ConservedCodonComposition.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org0e1e902" class="outline-3">
<h3 id="org0e1e902"><span class="section-number-3">35.3</span> Abstract</h3>
</div>
<div id="outline-container-org20e73e0" class="outline-3">
<h3 id="org20e73e0"><span class="section-number-3">35.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org3320a29" class="outline-2">
<h2 id="org3320a29"><span class="section-number-2">36</span> <span class="todo ___">[?]</span> LokeshKumar2019 - Prediction of Multi Drug Resistant Tuberculosis using Machine Learning Techniques</h2>
<div class="outline-text-2" id="text-36">
</div>
<div id="outline-container-org7253372" class="outline-3">
<h3 id="org7253372"><span class="section-number-3">36.1</span> Notes</h3>
<div class="outline-text-3" id="text-36-1">
<ul class="org-ul">
<li>Survey paper</li>
<li>Offers quick overview of various algorithms with accuracy and other characteristics</li>
</ul>
</div>
</div>
<div id="outline-container-org10e1318" class="outline-3">
<h3 id="org10e1318"><span class="section-number-3">36.2</span> FileName</h3>
<div class="outline-text-3" id="text-36-2">
<ul class="org-ul">
<li>LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org59c4bd6" class="outline-3">
<h3 id="org59c4bd6"><span class="section-number-3">36.3</span> Abstract</h3>
<div class="outline-text-3" id="text-36-3">
<ul class="org-ul">
<li>We review different Machine Learning Approaches to predict MDR-TB. Different
feature estimation methods, execution of distinct machine learning models also
have been explored.</li>
<li>AdaBoost ensemble algorithm gave the highest accuracy of all</li>
</ul>
</div>
</div>
<div id="outline-container-org71e9d41" class="outline-3">
<h3 id="org71e9d41"><span class="section-number-3">36.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgd2df039" class="outline-2">
<h2 id="orgd2df039"><span class="section-number-2">37</span> <span class="done _X_">[X]</span> Mahe2019 - A large scale evaluation of TBProfiler and Mykrobe for antibiotic resistance prediction in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-37">
</div>
<div id="outline-container-org4d414dc" class="outline-3">
<h3 id="org4d414dc"><span class="section-number-3">37.1</span> Notes</h3>
<div class="outline-text-3" id="text-37-1">
<ul class="org-ul">
<li>Doesn&rsquo;t directly relate to the dissertation</li>
</ul>
</div>
</div>
<div id="outline-container-org7a953f6" class="outline-3">
<h3 id="org7a953f6"><span class="section-number-3">37.2</span> FileName</h3>
<div class="outline-text-3" id="text-37-2">
<ul class="org-ul">
<li>Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org3ed93c3" class="outline-3">
<h3 id="org3ed93c3"><span class="section-number-3">37.3</span> Abstract</h3>
</div>
<div id="outline-container-org5e89d65" class="outline-3">
<h3 id="org5e89d65"><span class="section-number-3">37.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org6663d3e" class="outline-2">
<h2 id="org6663d3e"><span class="section-number-2">38</span> <span class="todo ___">[-]</span> Marino2016 - Computational and Empirical Studies Predict Mycobacterium tuberculosis-Specific T Cells as a Biomarker for Infection Outcome</h2>
<div class="outline-text-2" id="text-38">
</div>
<div id="outline-container-org0e24d78" class="outline-3">
<h3 id="org0e24d78"><span class="section-number-3">38.1</span> Notes</h3>
<div class="outline-text-3" id="text-38-1">
<ul class="org-ul">
<li>They conducted their own experiments</li>
</ul>
</div>
</div>

<div id="outline-container-orgcd08c7b" class="outline-3">
<h3 id="orgcd08c7b"><span class="section-number-3">38.2</span> FileName</h3>
<div class="outline-text-3" id="text-38-2">
<ul class="org-ul">
<li>Marino2016<sub>ComputationalAndEmpirical.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org064cd4e" class="outline-3">
<h3 id="org064cd4e"><span class="section-number-3">38.3</span> Abstract</h3>
<div class="outline-text-3" id="text-38-3">
<ul class="org-ul">
<li>Identifying biomarkers for tuberculosis (TB) is an ongoing challenge in
developing immuno- logical correlates of infection outcome and protection.</li>

<li>To effectively predict biomarkers for infection progression in any disease,
including TB, large amounts of experi- mental data are required to reach
statistical power and make accurate predictions.</li>

<li>Machine learning techniques were used to interrogate the experimental NHP
datasets without identifying any potential TB biomarker. In parallel, we used
our extensive novel NHP datasets to build and calibrate a multi-organ
computational model that combines what is occurring at the site of infection
(e.g., lung) at a single granuloma scale with blood level readouts that can be
tracked in mon- keys and humans.</li>
</ul>
</div>
</div>

<div id="outline-container-org01fd79a" class="outline-3">
<h3 id="org01fd79a"><span class="section-number-3">38.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orge8d8ccf" class="outline-2">
<h2 id="orge8d8ccf"><span class="section-number-2">39</span> <span class="todo ___">[-]</span> Marques2019 - Pulmonary nontuberculous mycobacterial infections: presumptive diagnosis based on the international microbiological criteria adopted in the state of São Paulo, Brazil, 2011-2014</h2>
<div class="outline-text-2" id="text-39">
</div>
<div id="outline-container-orgd69853c" class="outline-3">
<h3 id="orgd69853c"><span class="section-number-3">39.1</span> Notes</h3>
<div class="outline-text-3" id="text-39-1">
<ul class="org-ul">
<li>Survey paper for NTM</li>
<li>Can be used for building up the background</li>
</ul>
</div>
</div>
<div id="outline-container-org1f2600f" class="outline-3">
<h3 id="org1f2600f"><span class="section-number-3">39.2</span> FileName</h3>
<div class="outline-text-3" id="text-39-2">
<ul class="org-ul">
<li>Marques2019<sub>PulmonaryNTMInfections.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org07117df" class="outline-3">
<h3 id="org07117df"><span class="section-number-3">39.3</span> Abstract</h3>
<div class="outline-text-3" id="text-39-3">
<ul class="org-ul">
<li>The objective of the present study was to determine the frequency at which the
various NTM species are identified among selected individuals and in potential
cases of pulmonary nontuberculous mycobacterial infection.</li>
</ul>
</div>
</div>
<div id="outline-container-org344e749" class="outline-3">
<h3 id="org344e749"><span class="section-number-3">39.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-39-4">
<ul class="org-ul">
<li>Although various NTM species were identified among the individuals studied,
those presumptively identified most frequently on the basis of the
microbiological criteria adopted in the state of São Paulo were the ones that
are most commonly associated with pulmonary nontuberculous mycobacterial
infection worldwide or in specific geographic regions.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org74089f1" class="outline-2">
<h2 id="org74089f1"><span class="section-number-2">40</span> <span class="todo ___">[-]</span> Meldenez2015 - On Combining Multiple-Instance Learning and Active Learning for Computer-Aided Detection of Tuberculosis</h2>
<div class="outline-text-2" id="text-40">
</div>
<div id="outline-container-orgd8008be" class="outline-3">
<h3 id="orgd8008be"><span class="section-number-3">40.1</span> Notes</h3>
<div class="outline-text-3" id="text-40-1">
<ul class="org-ul">
<li>Image processing, radiography</li>
</ul>
</div>
</div>
<div id="outline-container-org948de83" class="outline-3">
<h3 id="org948de83"><span class="section-number-3">40.2</span> FileName</h3>
<div class="outline-text-3" id="text-40-2">
<ul class="org-ul">
<li>Meldenez2015<sub>OnCombiningMultiple.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgf7329e5" class="outline-3">
<h3 id="orgf7329e5"><span class="section-number-3">40.3</span> Abstract</h3>
<div class="outline-text-3" id="text-40-3">
<ul class="org-ul">
<li>The major advantage of multiple-instance learning (MIL) applied to a
computer-aided detection (CAD) system is that it allows optimizing the latter
with case-level labels instead of accurate lesion outlines as traditionally
required for a supervised approach.</li>

<li>As shown in previous work, a MIL-based CAD system can perform comparably to
its supervised counterpart considering complex tasks such as chest radiograph
scoring in tuberculosis (TB) detection. However, despite this remarkable
achievement, the uncertainty inherent to MIL can lead to a less satisfactory
outcome if analysis at lower levels (e.g., regions or pixels) is needed.</li>

<li>To minimize the labeling effort, we develop a novel instance selection
mechanism that exploits the MIL problem definition through one-class
classification.</li>
</ul>
</div>
</div>

<div id="outline-container-org22427ec" class="outline-3">
<h3 id="org22427ec"><span class="section-number-3">40.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgf77043f" class="outline-2">
<h2 id="orgf77043f"><span class="section-number-2">41</span> <span class="todo ___">[-]</span> Mikusova2016 - Learning from the past for TB drug discovery in the future</h2>
<div class="outline-text-2" id="text-41">
</div>
<div id="outline-container-orge333a3c" class="outline-3">
<h3 id="orge333a3c"><span class="section-number-3">41.1</span> Notes</h3>
<div class="outline-text-3" id="text-41-1">
<ul class="org-ul">
<li>Involves chemistry</li>
<li>Survey paper</li>
<li>Can be used to build up the context for Drug-Target-Interaction</li>
</ul>
</div>
</div>
<div id="outline-container-org40a47f9" class="outline-3">
<h3 id="org40a47f9"><span class="section-number-3">41.2</span> FileName</h3>
<div class="outline-text-3" id="text-41-2">
<ul class="org-ul">
<li>Mikusova2016<sub>LearningFromThePast.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8a1b246" class="outline-3">
<h3 id="org8a1b246"><span class="section-number-3">41.3</span> Abstract</h3>
<div class="outline-text-3" id="text-41-3">
<ul class="org-ul">
<li>Tuberculosis drug discovery has shifted in recent years from a primarily
target-based approach to one that uses phenotypic high-throughput screens.</li>

<li>It is clear that we still have much to learn about the drug targets and the
complex effects of the drugs on Mycobacterium tuberculosis.</li>

<li>We propose a more integrated approach that learns from earlier drug discovery
efforts that could help to move drug discovery forward.</li>
</ul>
</div>
</div>

<div id="outline-container-orga5f8d17" class="outline-3">
<h3 id="orga5f8d17"><span class="section-number-3">41.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgb1e74c9" class="outline-2">
<h2 id="orgb1e74c9"><span class="section-number-2">42</span> <span class="todo ___">[-]</span> Ngo2019 - Genomic prediction of tuberculosis drug- resistance: benchmarking existing databases and prediction algorithms</h2>
<div class="outline-text-2" id="text-42">
</div>
<div id="outline-container-orgb78612c" class="outline-3">
<h3 id="orgb78612c"><span class="section-number-3">42.1</span> Notes</h3>
<div class="outline-text-3" id="text-42-1">
<ul class="org-ul">
<li>This paper seems important but I&rsquo;m unable to see how to use this in the dissertation.</li>
</ul>
</div>
</div>
<div id="outline-container-orgb4b8475" class="outline-3">
<h3 id="orgb4b8475"><span class="section-number-3">42.2</span> FileName</h3>
<div class="outline-text-3" id="text-42-2">
<ul class="org-ul">
<li>Ngo2019<sub>GenomicPrediction.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org5ec115e" class="outline-3">
<h3 id="org5ec115e"><span class="section-number-3">42.3</span> Abstract</h3>
<div class="outline-text-3" id="text-42-3">
<ul class="org-ul">
<li>It is possible to predict whether a tuberculosis (TB) patient will fail to
respond to specific antibiotics by sequencing the genome of the infecting
Mycobacterium tuberculosis (Mtb) and observing whether the pathogen carries
specific mutations at drug-resistance sites.</li>

<li>Here, we evaluate the performance of four popular tools (TBProfiler, MyKrobe,
KvarQ, PhyResSE) with 6746 isolates compiled from publicly available
databases, and subsequently identify highly probable phenotyping errors in the
databases by genetically predicting the drug phenotypes using all four
software.</li>

<li>Our results show that these bioinformatics tools generally perform well in
predicting the resistance status for two key first-line agents (isoniazid,
rifampicin), but the accuracy is lower for second-line injectables and
fluoroquinolones.</li>
</ul>
</div>
</div>

<div id="outline-container-org1d32a25" class="outline-3">
<h3 id="org1d32a25"><span class="section-number-3">42.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-42-4">
<ul class="org-ul">
<li>The use of whole-genome sequencing to replace conventional laboratory-based
drug-susceptibility testing reduces the time taken to culture and test against
individual drugs, but requires sophisticated bioinformatics algorithms to be
designed and validated to translate the genetic information into drug
susceptibility phenotypes.</li>

<li>Using latent class modeling, we have identi- fied which isolates in the
existing databases are likely to present erroneous DST phenotypes and
recalibrated the performance of these bioinformatics software adjusting for
errors in the existing databases.</li>

<li>However, we still observe low sensitivity in detecting resistance to ETO,
PAS, PTO, and PZA.</li>

<li>Without a clear understanding to the genetic biomarkers of drug resistance or
to their pene- trance, or at least more powerful algorithms that can pick up
additional intricacies of resistance across multiple drugs, such as what have
been done for HIV and cancer [22–24], the ability to successfully predict the
presence of drug-resistance is reduced.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgd294a50" class="outline-2">
<h2 id="orgd294a50"><span class="section-number-2">43</span> <span class="todo ___">[-]</span> Periwal2011 - Predictive models for anti-tubercular molecules using machine learning on high-throughput biological screening datasets</h2>
<div class="outline-text-2" id="text-43">
</div>
<div id="outline-container-org622b584" class="outline-3">
<h3 id="org622b584"><span class="section-number-3">43.1</span> Notes</h3>
<div class="outline-text-3" id="text-43-1">
<ul class="org-ul">
<li>Could build upon this paper by using XgBoost and other classification
algorithms</li>
<li>They&rsquo;ve used Weka</li>
<li>Data is available</li>
<li>Code is not available</li>
<li>They&rsquo;ve also relied on some closed-source software</li>
</ul>
</div>
</div>
<div id="outline-container-orgb2a1106" class="outline-3">
<h3 id="orgb2a1106"><span class="section-number-3">43.2</span> FileName</h3>
<div class="outline-text-3" id="text-43-2">
<ul class="org-ul">
<li>Periwal2011<sub>PredictiveModels.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgee70c69" class="outline-3">
<h3 id="orgee70c69"><span class="section-number-3">43.3</span> Abstract</h3>
<div class="outline-text-3" id="text-43-3">
<ul class="org-ul">
<li>Recent reports suggest that Mtb has been developing resistance to the widely
used anti-tubercular drugs resulting in the emergence and spread of multi
drug-resistant (MDR) and extensively drug-resistant (XDR) strains throughout
the world.</li>

<li>Whole organism screens on the other hand are target-agnostic and have been now
widely employed as an alternative for lead identification but they are limited
by the time and cost involved in running the screens for large compound
libraries.</li>
</ul>
</div>
</div>

<div id="outline-container-org786a841" class="outline-3">
<h3 id="org786a841"><span class="section-number-3">43.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-43-4">
<ul class="org-ul">
<li>We have employed a systematic and comprehensive approach to build supervised
classification based predictive models for anti-tubercular agents from
publicly available bioassay datasets for in-vitro screens for Mtb inhibitors.</li>

<li>We have generated classification models for anti-tubercular agents with four
commonly used state-of-the-art classifiers i.e. Naïve Bayes, Random Forest,
J48 and SMO</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgeaf56b4" class="outline-2">
<h2 id="orgeaf56b4"><span class="section-number-2">44</span> <span class="todo ___">[ ]</span> Ratnatunga2020 - The Rise of Non-Tuberculosis Mycobacterial Lung Disease</h2>
<div class="outline-text-2" id="text-44">
</div>
<div id="outline-container-org44d632f" class="outline-3">
<h3 id="org44d632f"><span class="section-number-3">44.1</span> Notes</h3>
<div class="outline-text-3" id="text-44-1">
<ul class="org-ul">
<li>Survey paper for NTM</li>
<li>Can be used for background context</li>
</ul>
</div>
</div>
<div id="outline-container-org7e0c5ba" class="outline-3">
<h3 id="org7e0c5ba"><span class="section-number-3">44.2</span> FileName</h3>
<div class="outline-text-3" id="text-44-2">
<ul class="org-ul">
<li>Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orge9372ae" class="outline-3">
<h3 id="orge9372ae"><span class="section-number-3">44.3</span> Abstract</h3>
<div class="outline-text-3" id="text-44-3">
<ul class="org-ul">
<li>These lesser known “cousins” of Mycobacterium tuberculosis (TB) were once
thought to be harmless environmental saprophytics and only dangerous to
individuals with defective lung structure or the immunosuppressed.</li>

<li>NTM have become extremely antibiotic resistant, and now have been found to be
internationally dispersed through person-to-person contact.</li>
</ul>
</div>
</div>

<div id="outline-container-orgd8c691d" class="outline-3">
<h3 id="orgd8c691d"><span class="section-number-3">44.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orga687d85" class="outline-2">
<h2 id="orga687d85"><span class="section-number-2">45</span> <span class="todo WAIT">WAIT</span> Romero2020 - Decision tree machine learning applied to bovine tuberculosis risk factors to aid disease control decision making</h2>
<div class="outline-text-2" id="text-45">
</div>
<div id="outline-container-org81825b2" class="outline-3">
<h3 id="org81825b2"><span class="section-number-3">45.1</span> Notes</h3>
<div class="outline-text-3" id="text-45-1">
<ul class="org-ul">
<li>Can use other algorithms for improving the accuracy i.e. XgBoost</li>

<li>Uses Decision Tree Classifier to segregate the areas into low, mid, high risk</li>

<li>Code is not provided</li>

<li>Ask LJMU staff for help in this</li>
</ul>
</div>
</div>
<div id="outline-container-orgf4a62e1" class="outline-3">
<h3 id="orgf4a62e1"><span class="section-number-3">45.2</span> FileName</h3>
<div class="outline-text-3" id="text-45-2">
<ul class="org-ul">
<li>Romero2020<sub>DecisionTree.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgc58b5a1" class="outline-3">
<h3 id="orgc58b5a1"><span class="section-number-3">45.3</span> Abstract</h3>
<div class="outline-text-3" id="text-45-3">
<ul class="org-ul">
<li>Identifying and understanding the risk factors for endemic bovine tuberculosis
(TB) in cattle herds is critical for the control of this disease. Exploratory
machine learning techniques can uncover complex non-linear relation- ships and
interactions within disease causation webs, and enhance our knowledge of TB
risk factors and how they are interrelated.</li>

<li>The best classification tree models informed multivariable binomial logistic
regression models in each area, adding statistical inference outputs. These
two approaches showed similar predictive performance although there were some
disparities regarding what constituted high-risk predictors.</li>

<li>Decision tree machine learning approaches can identify risk factors from webs
of causation: information which may then be used to inform decision making for
disease control purposes.</li>
</ul>
</div>
</div>

<div id="outline-container-org32c76ad" class="outline-3">
<h3 id="org32c76ad"><span class="section-number-3">45.4</span> Conclusion</h3>
</div>
<div id="outline-container-org20cdb11" class="outline-3">
<h3 id="org20cdb11"><span class="section-number-3">45.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-45-5">
</div>
<div id="outline-container-org93362d7" class="outline-4">
<h4 id="org93362d7"><span class="section-number-4">45.5.1</span> Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-org9f971cb" class="outline-2">
<h2 id="org9f971cb"><span class="section-number-2">46</span> <span class="done DONE">DONE</span> Sauer2018 - Feature selection and prediction of treatment failure in tuberculosis</h2>
<div class="outline-text-2" id="text-46">
<p>
EMILYN: Please think of project(s)
</p>
</div>
<div id="outline-container-orgdd9ad5d" class="outline-3">
<h3 id="orgdd9ad5d"><span class="section-number-3">46.1</span> Notes</h3>
<div class="outline-text-3" id="text-46-1">
<ul class="org-ul">
<li>Code is available
<a href="https://github.com/biosharp-dotnet/niaid_TB">https://github.com/biosharp-dotnet/niaid_TB</a></li>

<li>The data is sourced from dedicated NIH website
<a href="https://tbportals.niaid.nih.gov/">https://tbportals.niaid.nih.gov/</a></li>

<li>Can use XgBoost and/or H20</li>

<li>Can add the data from other sources as well, perhaps Brazilian</li>
</ul>
</div>
</div>
<div id="outline-container-org5cdbcc3" class="outline-3">
<h3 id="org5cdbcc3"><span class="section-number-3">46.2</span> FileName</h3>
<div class="outline-text-3" id="text-46-2">
<ul class="org-ul">
<li>Sauer2018<sub>FeatureSelection.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org53a2297" class="outline-3">
<h3 id="org53a2297"><span class="section-number-3">46.3</span> Abstract</h3>
<div class="outline-text-3" id="text-46-3">
<ul class="org-ul">
<li>On a multi-country dataset managed by the National Institute of Allergy and
Infectious Diseases we applied various machine learning techniques to
identify factors statistically associated with treatment failure and to
predict treatment failure based on baseline demographic and clinical
characteristics alone.</li>
</ul>
</div>
</div>

<div id="outline-container-orga2b155c" class="outline-3">
<h3 id="orga2b155c"><span class="section-number-3">46.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org062b295" class="outline-2">
<h2 id="org062b295"><span class="section-number-2">47</span> <span class="todo ___">[-]</span> Schiff2018 - Clinical relevance of non-tuberculous mycobacteria isolated from respiratory specimens: seven year experience in a UK hospital</h2>
<div class="outline-text-2" id="text-47">
</div>
<div id="outline-container-org16aa2f8" class="outline-3">
<h3 id="org16aa2f8"><span class="section-number-3">47.1</span> Notes</h3>
<div class="outline-text-3" id="text-47-1">
<ul class="org-ul">
<li>Survey report</li>

<li>Can be used for background context</li>
</ul>
</div>
</div>
<div id="outline-container-orge8b7ca0" class="outline-3">
<h3 id="orge8b7ca0"><span class="section-number-3">47.2</span> FileName</h3>
<div class="outline-text-3" id="text-47-2">
<ul class="org-ul">
<li>Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org29ec187" class="outline-3">
<h3 id="org29ec187"><span class="section-number-3">47.3</span> Abstract</h3>
</div>
<div id="outline-container-org17178fd" class="outline-3">
<h3 id="org17178fd"><span class="section-number-3">47.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org2649eee" class="outline-2">
<h2 id="org2649eee"><span class="section-number-2">48</span> <span class="todo WAIT">WAIT</span> Sergeev2016 - Genome-wide Analysis of MDR and XDR Tuberculosis from Belarus: Machine-learning Approach</h2>
<div class="outline-text-2" id="text-48">
</div>
<div id="outline-container-orgce785ce" class="outline-3">
<h3 id="orgce785ce"><span class="section-number-3">48.1</span> Notes</h3>
<div class="outline-text-3" id="text-48-1">
<ul class="org-ul">
<li>Uses XgBoost XGB classifier</li>

<li>Can apply this approach for NTM</li>

<li>Involves mathematical equations</li>

<li>I&rsquo;ve contacted the authors for the code</li>
</ul>
</div>
</div>
<div id="outline-container-org6202544" class="outline-3">
<h3 id="org6202544"><span class="section-number-3">48.2</span> FileName</h3>
<div class="outline-text-3" id="text-48-2">
<ul class="org-ul">
<li>Sergeev2016<sub>GenomeWide.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8855cd8" class="outline-3">
<h3 id="org8855cd8"><span class="section-number-3">48.3</span> Abstract</h3>
<div class="outline-text-3" id="text-48-3">
<ul class="org-ul">
<li>Alterations in pathogen genomes are among the main mechanisms by which
microorganisms exhibit drug resistance.</li>

<li>Analysis of 144 M. tuberculosis strains of different phenotypes including drug
susceptible, MDR and XDR isolated in Belarus was fulfilled in this paper. A
wide range of machine learning methods that can discover SNPs related to
drug-resistance in the whole bacteria genomes was investigated</li>

<li>Besides single-SNP testing approaches, methods that allow detecting joint
effects from interacting SNPs were considered.</li>

<li>Analysis of whole-genome sequences often leads to situations where the number
of treated features exceeds the number of available observations. For this
reason, special attention is paid to fair evaluation of the model prediction
quality and minimizing the risk of overfitting while estimating the underlying
parameters.</li>
</ul>
</div>
</div>


<div id="outline-container-org1a3f55d" class="outline-3">
<h3 id="org1a3f55d"><span class="section-number-3">48.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org5508a5a" class="outline-2">
<h2 id="org5508a5a"><span class="section-number-2">49</span> <span class="todo ___">[-]</span> Vizcaino2010 - Computational Prediction and Experimental Assessment of Secreted/Surface Proteins from Mycobacterium tuberculosis H37Rv</h2>
<div class="outline-text-2" id="text-49">
</div>
<div id="outline-container-org3898256" class="outline-3">
<h3 id="org3898256"><span class="section-number-3">49.1</span> Notes</h3>
<div class="outline-text-3" id="text-49-1">
<ul class="org-ul">
<li>I suspect, this involves some work in the wetlab</li>

<li>Might be interesting for Emilyn as it involves quite a lot of biological
knowledge as well</li>

<li>Has many microscopic photos of the bacteria</li>
</ul>
</div>
</div>
<div id="outline-container-orgb77fa24" class="outline-3">
<h3 id="orgb77fa24"><span class="section-number-3">49.2</span> FileName</h3>
<div class="outline-text-3" id="text-49-2">
<ul class="org-ul">
<li>Vizcaino2010<sub>ComputatationalPrediction.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org81f28a2" class="outline-3">
<h3 id="org81f28a2"><span class="section-number-3">49.3</span> Abstract</h3>
<div class="outline-text-3" id="text-49-3">
<ul class="org-ul">
<li>The mycobacterial cell envelope has been implicated in the pathogenicity of
tuberculosis and therefore has been a prime target for the identification and
characterization of surface proteins with potential application in drug and
vaccine development.</li>

<li>In this study, the genome of Mycobacterium tuberculosis
H37Rv was screened using Machine Learning tools that included feature-based
predictors, general localizers and transmembrane topology predictors to
identify proteins that are potentially secreted to the surface of M.
tuberculosis, or to the extracellular milieu through different secretory
pathways.</li>

<li>Surface localization was also confirmed for the positive controls, whereas
negative controls were located on the cytoplasm. Based on statistical learning
methods, we obtained computational subcellular localization predictions that
were experimentally assessed and allowed us to construct a computational
protocol with experimental support that allowed us to identify a new set of
secreted/surface proteins as potential vaccine candidates.</li>
</ul>
</div>
</div>

<div id="outline-container-org227a48e" class="outline-3">
<h3 id="org227a48e"><span class="section-number-3">49.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org709bfe6" class="outline-2">
<h2 id="org709bfe6"><span class="section-number-2">50</span> <span class="todo ___">[-]</span> Waman2019 - Mycobacterial genomics and structural bioinformatics: opportunities and challenges in drug discovery</h2>
<div class="outline-text-2" id="text-50">
</div>
<div id="outline-container-org3694c31" class="outline-3">
<h3 id="org3694c31"><span class="section-number-3">50.1</span> Notes</h3>
<div class="outline-text-3" id="text-50-1">
<ul class="org-ul">
<li>Survey paper, can be used for background context</li>
</ul>
</div>
</div>
<div id="outline-container-orga2ce9f6" class="outline-3">
<h3 id="orga2ce9f6"><span class="section-number-3">50.2</span> FileName</h3>
<div class="outline-text-3" id="text-50-2">
<ul class="org-ul">
<li>Waman2019<sub>MycobacterialGenomics.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org00b7acf" class="outline-3">
<h3 id="org00b7acf"><span class="section-number-3">50.3</span> Abstract</h3>
<div class="outline-text-3" id="text-50-3">
<ul class="org-ul">
<li>Here we describe the impact of the rapid expansion of genome sequencing and
genome/pathway annotations that have greatly improved the progress of
structure-guided drug discovery.</li>

<li>We focus on the applications of comparative genomics, metabolomics,
evolutionary bioinformatics and structural proteomics to identify potential
drug targets. The opportunities and challenges for the design of drugs for M.
tuberculosis, M. leprae and M. abscessus to combat resistance are discussed.</li>
</ul>
</div>
</div>

<div id="outline-container-org757c3c5" class="outline-3">
<h3 id="org757c3c5"><span class="section-number-3">50.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org73e95c0" class="outline-2">
<h2 id="org73e95c0"><span class="section-number-2">51</span> <span class="todo ___">[?]</span> Wang2020 - A deep learning-based method for drug-target interaction prediction based on long short-term memory neural network</h2>
<div class="outline-text-2" id="text-51">
</div>
<div id="outline-container-orgda3edf0" class="outline-3">
<h3 id="orgda3edf0"><span class="section-number-3">51.1</span> Notes</h3>
<div class="outline-text-3" id="text-51-1">
<ul class="org-ul">
<li>Doesn&rsquo;t directly relate to MTB or NTM, but could be used as a reference in
future work</li>

<li>Builds up the mathematical reasoning wrt bio-chemistry</li>

<li>Explains the techniques well</li>

<li>Data and code is available</li>
</ul>
<p>
<a href="https://deepbiolab.coding.net/s/fbdb894d-2730-425b-bac6-18ba55396bab">https://deepbiolab.coding.net/s/fbdb894d-2730-425b-bac6-18ba55396bab</a>
</p>
</div>
</div>
<div id="outline-container-org63e13ba" class="outline-3">
<h3 id="org63e13ba"><span class="section-number-3">51.2</span> FileName</h3>
<div class="outline-text-3" id="text-51-2">
<ul class="org-ul">
<li>Wang2020<sub>ADeepLearningBasedMethod.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgff382d7" class="outline-3">
<h3 id="orgff382d7"><span class="section-number-3">51.3</span> Abstract</h3>
<div class="outline-text-3" id="text-51-3">
<ul class="org-ul">
<li>Due to the influence of throughput, precision and cost, traditional
experimental methods are difficult to be widely used to infer these potential
Drug-Target Interactions (DTIs). Therefore, it is urgent to develop effective
computational methods to validate the interaction between drugs and target.</li>
</ul>
</div>
</div>

<div id="outline-container-org2703fae" class="outline-3">
<h3 id="org2703fae"><span class="section-number-3">51.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-51-4">
<ul class="org-ul">
<li>We have developed a deep learning-based method to infer potential DTIs using
compounds and proteins sequence.</li>

<li>Comparing with other classifiers, we have provided initial evidence that
DeepLSTM outperforms traditional ma- chine learning system on the DTIs task.</li>
</ul>
</div>
</div>


<div id="outline-container-orgf3774ef" class="outline-3">
<h3 id="orgf3774ef"><span class="section-number-3">51.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-51-5">
<ul class="org-ul">
<li>Wang2020<sub>ADeepLearningBasedMethod</sub><sub>Code.py</sub></li>
<li>Wang2020<sub>ADeepLearningBasedMethod</sub><sub>Supplementary.pdf</sub></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-orge662795" class="outline-2">
<h2 id="orge662795"><span class="section-number-2">52</span> <span class="todo ___">[?]</span> Weiner2017 - High-throughput and computational approaches for diagnostic and prognostic host tuberculosis biomarkers</h2>
<div class="outline-text-2" id="text-52">
</div>
<div id="outline-container-org656acf5" class="outline-3">
<h3 id="org656acf5"><span class="section-number-3">52.1</span> Notes</h3>
<div class="outline-text-3" id="text-52-1">
<ul class="org-ul">
<li>Survey paper</li>
</ul>
</div>
</div>

<div id="outline-container-orgf2ebcd2" class="outline-3">
<h3 id="orgf2ebcd2"><span class="section-number-3">52.2</span> FileName</h3>
<div class="outline-text-3" id="text-52-2">
<ul class="org-ul">
<li>Weiner2017<sub>HighThroughput.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org98880f6" class="outline-3">
<h3 id="org98880f6"><span class="section-number-3">52.3</span> Abstract</h3>
<div class="outline-text-3" id="text-52-3">
<ul class="org-ul">
<li>High-throughput techniques strive to identify new biomarkers that will be
useful for the diagnosis, treatment, and prevention of tuberculosis (TB).
However, their analysis and interpretation pose considerable challenges.</li>

<li>Recent developments in the high-throughput detection of host biomarkers in TB
are reported in this review.</li>
</ul>
</div>
</div>

<div id="outline-container-org833402f" class="outline-3">
<h3 id="org833402f"><span class="section-number-3">52.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgefd6324" class="outline-2">
<h2 id="orgefd6324"><span class="section-number-2">53</span> <span class="todo WAIT">WAIT</span> Wilder2019 - Improving tuberculosis treatment by integrating optimization and learning</h2>
<div class="outline-text-2" id="text-53">
</div>
<div id="outline-container-org6973ad5" class="outline-3">
<h3 id="org6973ad5"><span class="section-number-3">53.1</span> FileName</h3>
<div class="outline-text-3" id="text-53-1">
<ul class="org-ul">
<li>Wilder2019<sub>ImprovingTuberculosis.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org8eadce3" class="outline-3">
<h3 id="org8eadce3"><span class="section-number-3">53.2</span> Notes</h3>
<div class="outline-text-3" id="text-53-2">
<ul class="org-ul">
<li>Mentions <a href="https://www.99dots.org/">https://www.99dots.org/</a> for strategic intervention</li>

<li>Emailed the 99dots team for some data access</li>
</ul>
</div>
</div>
<div id="outline-container-org9f7b559" class="outline-3">
<h3 id="org9f7b559"><span class="section-number-3">53.3</span> Abstract</h3>
<div class="outline-text-3" id="text-53-3">
<ul class="org-ul">
<li>The prevalence of TB is caused in part by non-adherence to medication, which
results in greater risk of death, reinfection and contraction of
multidrug-resistant TB</li>

<li>Using data on 17,000 Indian patients provided by the NGO Everwell, we consider
the problem of predicting which patients are likely to miss doses in the
near future and optimizing interventions by health workers to avert such
treatment failures.</li>
</ul>
</div>
</div>

<div id="outline-container-org7902c74" class="outline-3">
<h3 id="org7902c74"><span class="section-number-3">53.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org1556e3b" class="outline-2">
<h2 id="org1556e3b"><span class="section-number-2">54</span> <span class="done DONE">DONE</span> Yang2017 - Machine learning for classifying tuberculosis drug-resistance from DNA sequencing data</h2>
<div class="outline-text-2" id="text-54">
<p>
EMILYN: We could have a project proposal, which compares the various attempts in
the field and perhaps include new data and algorithms.
</p>
</div>
<div id="outline-container-orgefbcefe" class="outline-3">
<h3 id="orgefbcefe"><span class="section-number-3">54.1</span> Notes</h3>
<div class="outline-text-3" id="text-54-1">
<ul class="org-ul">
<li>Code is available and already downloaded</li>

<li>Builds upon a previous publication Walker2015 for narrowing the feature set.</li>

<li>The data is the same as used in Walker2015, 1839 samples</li>
</ul>
</div>
</div>

<div id="outline-container-org2fd7808" class="outline-3">
<h3 id="org2fd7808"><span class="section-number-3">54.2</span> FileName</h3>
<div class="outline-text-3" id="text-54-2">
<ul class="org-ul">
<li>Yang2017<sub>MachineLearning.pdf</sub></li>
</ul>
</div>
</div>

<div id="outline-container-org0fc008a" class="outline-3">
<h3 id="org0fc008a"><span class="section-number-3">54.3</span> Abstract</h3>
<div class="outline-text-3" id="text-54-3">
<ul class="org-ul">
<li>Given the availability of DNA sequencing data from MTB, we developed machine
learning models for a cohort of 1839 UK bacterial isolates to classify MTB
resistance against eight anti- TB drugs (isoniazid, rifampicin, ethambutol,
pyrazinamide, ciprofloxacin, moxifloxacin, ofloxacin, streptomycin) and to
classify multi-drug resistance.</li>
</ul>
</div>
</div>

<div id="outline-container-org6087ded" class="outline-3">
<h3 id="org6087ded"><span class="section-number-3">54.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-54-4">
<ul class="org-ul">
<li>We performed both principal component analysis (PCA) and a sparse logistic
version (SL-PCA) to explore the underlying structure of genetic variation
within our cohort of bacterial isolates.</li>

<li>Among the machine learning classifiers, PM and SVM-RBF rank as the top two
best-performing classifiers overall. The former integrates prior knowledge of
resistance determinant explicitly (though prior knowledge about susceptibility
determinants, lineage defining SNPs and compensatory mutations was not
included), and the latter suggests that the method’s nonlinear model is
suitable for unveiling nonlinear rela- tionships among mutations in terms of
resistance association for INH, CIP and SM.</li>

<li>This work showed great potentials of machine learning in improving resistance
classification given high-dimensional genetic data, especially when the
underlying biological resistance mecha- nism is poorly understood for many
drugs</li>
</ul>
</div>
</div>

<div id="outline-container-org511e792" class="outline-3">
<h3 id="org511e792"><span class="section-number-3">54.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-54-5">
<ul class="org-ul">
<li>Yang2017<sub>MachineLearning</sub><sub>Code.zip</sub></li>
<li>Yang2017<sub>MachineLearning</sub><sub>Supplement.pdf</sub></li>
<li>Yang2017<sub>MachineLearning</sub><sub>Walker2015</sub><sub>BasePaper.pdf</sub></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-orgd140b58" class="outline-2">
<h2 id="orgd140b58"><span class="section-number-2">55</span> <span class="done DONE">DONE</span> Yang2019 - DeepAMR for predicting co-occurrent resistance of Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-55">
</div>
<div id="outline-container-orgd323000" class="outline-3">
<h3 id="orgd323000"><span class="section-number-3">55.1</span> Notes</h3>
<div class="outline-text-3" id="text-55-1">
<ul class="org-ul">
<li>Code is available and already downloaded</li>

<li>Paper also references
<a href="https://github.com/biosharp-dotnet/Keras_Deep_Clustering">https://github.com/biosharp-dotnet/Keras_Deep_Clustering</a></li>

<li>Makes use of Multi-label K-nearest neighbour (MLKNN)</li>
</ul>
</div>
</div>
<div id="outline-container-org2c3345b" class="outline-3">
<h3 id="org2c3345b"><span class="section-number-3">55.2</span> FileName</h3>
<div class="outline-text-3" id="text-55-2">
<ul class="org-ul">
<li>Yang2019<sub>DeepAMRForPredicting</sub><sub>Code.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org734aa38" class="outline-3">
<h3 id="org734aa38"><span class="section-number-3">55.3</span> Abstract</h3>
<div class="outline-text-3" id="text-55-3">
<ul class="org-ul">
<li>Existing methods based on genetic data analysis of Mycobacterium tuberculosis
(MTB) have been able to predict resistance of MTB to individual drugs, but
have not considered the resistance co-occurrence and cannot capture latent
structure of genomic data that corresponds to lineages.</li>

<li>We then proposed an end-to-end multi-task model with deep denoising
auto-encoder (DeepAMR) for multiple drug classification and developed
DeepAMR<sub>cluster</sub>, a clustering variant based on DeepAMR, for learning clusters
in latent space of the data.</li>
</ul>
</div>
</div>

<div id="outline-container-orgdd3dd6e" class="outline-3">
<h3 id="orgdd3dd6e"><span class="section-number-3">55.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-55-4">
<ul class="org-ul">
<li>The resistance prediction for the MTB is essentially a multi-label learn- ing
problem, where each isolate is resistant to a subset of the examined drugs.
The input is mutations for individual isolates, such that the pres- ence of a
SNP variable is denoted as one and zero otherwise</li>

<li>We only considered cross-resistance between four first-line drugs but ignored
that of second-line drugs because: (i) the inaccurate phenotyping for
second-line drugs would introduce large error in the classification for
first-line drugs and (ii) a small number of resistant isolates would result in
over-fitting easily for such a complex model.</li>
</ul>
</div>
</div>

<div id="outline-container-org28f98bb" class="outline-3">
<h3 id="org28f98bb"><span class="section-number-3">55.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-55-5">
<ul class="org-ul">
<li>Yang2019<sub>DeepAMRForPredicting</sub><sub>Code.zip</sub></li>
<li>Yang2019<sub>DeepAMRForPredicting</sub><sub>Supplementary.zip</sub></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-orga442335" class="outline-2">
<h2 id="orga442335"><span class="section-number-2">56</span> <span class="todo ___">[?]</span> Zhang2020 - An integrated machine learning approach for predicting DosR-regulated genes in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-56">
</div>
<div id="outline-container-org0adf093" class="outline-3">
<h3 id="org0adf093"><span class="section-number-3">56.1</span> Notes</h3>
<div class="outline-text-3" id="text-56-1">
<ul class="org-ul">
<li>Involves some background knowledge of Genomics</li>

<li>The papar has mathematical rigor</li>
</ul>
</div>
</div>
<div id="outline-container-orgff85635" class="outline-3">
<h3 id="orgff85635"><span class="section-number-3">56.2</span> FileName</h3>
<div class="outline-text-3" id="text-56-2">
<ul class="org-ul">
<li>Zhang2020<sub>AnIntegratedMLApproach.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgde621f2" class="outline-3">
<h3 id="orgde621f2"><span class="section-number-3">56.3</span> Abstract</h3>
<div class="outline-text-3" id="text-56-3">
<ul class="org-ul">
<li>DosR is an important regulator of the response to stress such as limited
oxygen availability in Mycobacterium tuberculosis</li>

<li>Most of the approaches capture only a first-order approximation to the complex
transcriptional processes by assuming linear gene responses and linear
dynamics in TFA, or ignore the temporal information in data from such systems.</li>
</ul>


<ul class="org-ul">
<li>Our reconstructed TFA is closer to the experimentally determined profile of
p53 concentration than that from the original study.</li>
</ul>
</div>
</div>


<div id="outline-container-org22e7dc7" class="outline-3">
<h3 id="org22e7dc7"><span class="section-number-3">56.4</span> Conclusion</h3>
</div>
</div>

<div id="outline-container-org174cdbc" class="outline-2">
<h2 id="org174cdbc"><span class="section-number-2">57</span> <span class="todo ___">[?]</span> Gabrilian2019 - TB DEPOT (Data Exploration Portal): A multi- domain tuberculosis data analysis resource</h2>
<div class="outline-text-2" id="text-57">
</div>
<div id="outline-container-orgbe83ed9" class="outline-3">
<h3 id="orgbe83ed9"><span class="section-number-3">57.1</span> Notes</h3>
</div>
<div id="outline-container-org93b29fa" class="outline-3">
<h3 id="org93b29fa"><span class="section-number-3">57.2</span> FileName</h3>
<div class="outline-text-3" id="text-57-2">
<ul class="org-ul">
<li>Gabrielian2019<sub>TbDepotDataAnalysisResource.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orge791a09" class="outline-3">
<h3 id="orge791a09"><span class="section-number-3">57.3</span> Abstract</h3>
</div>
<div id="outline-container-org61252e0" class="outline-3">
<h3 id="org61252e0"><span class="section-number-3">57.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org5e0241d" class="outline-2">
<h2 id="org5e0241d"><span class="section-number-2">58</span> <span class="done _X_">[X]</span> Garnier2019 - Automatic diagnostics of tuberculosis using convolutional neural networks analysis of MODS digital images</h2>
<div class="outline-text-2" id="text-58">
</div>
<div id="outline-container-org5003aad" class="outline-3">
<h3 id="org5003aad"><span class="section-number-3">58.1</span> Notes</h3>
</div>
<div id="outline-container-org6ceb1d6" class="outline-3">
<h3 id="org6ceb1d6"><span class="section-number-3">58.2</span> FileName</h3>
<div class="outline-text-3" id="text-58-2">
<ul class="org-ul">
<li>Garnier2019<sub>TbImagesDiagnostics.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org7aedc2a" class="outline-3">
<h3 id="org7aedc2a"><span class="section-number-3">58.3</span> Abstract</h3>
</div>
<div id="outline-container-org50f88c8" class="outline-3">
<h3 id="org50f88c8"><span class="section-number-3">58.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org7ace1ea" class="outline-2">
<h2 id="org7ace1ea"><span class="section-number-2">59</span> <span class="todo ___">[-]</span> Muzondiwa2020 - Resistance Sniffer: An online tool for prediction of drug resistance patterns of Mycobacterium tuberculosis isolates using next generation sequencing data</h2>
<div class="outline-text-2" id="text-59">
</div>
<div id="outline-container-org0858bb1" class="outline-3">
<h3 id="org0858bb1"><span class="section-number-3">59.1</span> Notes</h3>
<div class="outline-text-3" id="text-59-1">
<ul class="org-ul">
<li>In future, this paper could be the foundation of a proper prdiction tool and paper</li>
</ul>
</div>
</div>
<div id="outline-container-orgc9e78f0" class="outline-3">
<h3 id="orgc9e78f0"><span class="section-number-3">59.2</span> FileName</h3>
<div class="outline-text-3" id="text-59-2">
<ul class="org-ul">
<li>Muzondiwa2020<sub>ResistanceSniffer.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org62bef29" class="outline-3">
<h3 id="org62bef29"><span class="section-number-3">59.3</span> Abstract</h3>
</div>
<div id="outline-container-org1284d92" class="outline-3">
<h3 id="org1284d92"><span class="section-number-3">59.4</span> Conclusion</h3>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Abhinav Sharma</p>
<p class="date">Created: 2020-05-10 Sun 01:10</p>
</div>
</body>
</html>
