# Table of Contents

1.  [Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.](#org1b5ac65)
2.  [Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.](#orge876284)
3.  [Machine learning and tubercular drug target recognition.](#org31bad2b)
4.  [Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem](#orgb954c79)
    1.  [Notes](#org3af2075)
    2.  [FileName](#org40412d9)
    3.  [Abstract](#org0bbd057)
    4.  [FutureWork](#orgea30536)
5.  [Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm](#org3cabcbc)
    1.  [Notes](#org29168cf)
    2.  [FileName](#org0a0102a)
    3.  [Abstract](#orgd4b5fa7)
    4.  [Conclusion](#orge9822a8)
6.  [Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper](#orgcc48f60)
    1.  [Notes](#org19f4758)
    2.  [FileName](#org42711fd)
    3.  [Abstract](#org08f105d)
    4.  [Conclusion](#orgb610be2)
7.  [Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities](#org99c7d50)
    1.  [Notes](#org2bb03ea)
    2.  [FileName](#org0de4fef)
    3.  [Abstract](#orgfc429aa)
    4.  [Conclusion](#orgb221989)
8.  [Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data](#org5b59f6e)
    1.  [Notes](#org93782d0)
    2.  [FileName](#org5583f25)
    3.  [Abstract](#org43d726a)
    4.  [Conclusion](#org8ff37b6)
9.  [Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction](#orgb788691)
    1.  [Notes](#orgcc2d08c)
    2.  [FileName](#org4f50588)
    3.  [Abstract](#org4626bae)
    4.  [Conclusion](#org5636ab1)
10. [Ching2018 - Opportunities and obstacles for deep learning in biology and medicine](#org67edce3)
    1.  [Notes](#orgb1f040d)
    2.  [FileName](#org1317902)
    3.  [Abstract](#org0a6ad34)
    4.  [Conclusion](#org7683ee6)
11. [Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data](#org2166fc7)
    1.  [Notes](#orgb1b0a45)
    2.  [FileName](#org9f02c86)
    3.  [Abstract](#org06f5679)
    4.  [Conclusion](#orga1a1679)
    5.  [Supplementary](#org079f508)
        1.  [Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub>](#orgb12e761)
12. [Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis](#org458ec36)
    1.  [Notes](#org99f1375)
    2.  [FileName](#org8605625)
    3.  [Abstract](#org32a32ef)
    4.  [Conclusion](#org38d5d70)
13. [Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports](#orgc9f7a4d)
    1.  [Notes](#orgea645b2)
    2.  [FileName](#org96912b3)
    3.  [Abstract](#org6e12d69)
    4.  [Conclusion](#orgf5abeb8)
14. [Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence](#orgae365bb)
    1.  [Notes](#orgb6f1e8f)
    2.  [FileName](#org84190c4)
    3.  [Abstract](#orgd40ebae)
    4.  [Conclusion](#org02569fd)
15. [Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models](#org9235f30)
    1.  [Notes](#org8c00406)
    2.  [FileName](#org811debe)
    3.  [Abstract](#org415e17a)
    4.  [Conclusion](#org37b23d4)
16. [Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation](#org4c5973b)
    1.  [Notes](#org0dd0c9c)
    2.  [FileName](#orga1cbd7d)
    3.  [Abstract](#org50294b2)
    4.  [Conclusion](#orgfb3a7e6)
17. [Ekins2014<sub>AreBiggerDataSets.pdf</sub>](#org3f15d68)
    1.  [Notes](#org7273c44)
    2.  [Title](#org6fd457d)
    3.  [Abstract](#org0b492e2)
    4.  [Conclusion](#org6bf8e62)
18. [Ekins2014<sub>CombiningComputationalMethods.pdf</sub>](#org79dc8cf)
    1.  [Notes](#org5d05756)
    2.  [Title](#org1565661)
    3.  [Abstract](#org9d1af0f)
    4.  [Conclusion](#orged6a132)
19. [Ekins2016<sub>MachineLearningModel.pdf</sub>](#orge30f29e)
    1.  [Notes](#orge7ae988)
    2.  [Title](#org925101c)
    3.  [Abstract](#org52ffe48)
    4.  [Conclusion](#org41097fb)
20. [Ekins2017<sub>MachineLearning.pdf</sub>](#orgdcd4e99)
    1.  [Notes](#orgb27b3d7)
    2.  [Title](#orgdc686e2)
    3.  [Abstract](#orgc28d943)
    4.  [Conclusion](#org8936ad3)
21. [Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis](#org32d00af)
    1.  [Notes](#org9265909)
    2.  [FileName](#org095d404)
    3.  [Abstract](#org5b7be8a)
    4.  [Conclusion](#orga9f1e81)
22. [Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay](#org096a2eb)
    1.  [Notes](#orgd1f615e)
    2.  [FileName](#org6a4e792)
    3.  [Abstract](#org907b848)
    4.  [Conclusion](#orgc52d35e)
23. [Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data](#org9b9253c)
    1.  [Notes](#org3fb7600)
    2.  [FileName](#orgd5f6287)
    3.  [Abstract](#org9cb8098)
    4.  [Conclusion](#orgfb2bce0)
24. [Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models](#org7478609)
    1.  [Notes](#org8cbb11d)
    2.  [FileName](#org5cdf663)
    3.  [Abstract](#orgd1ca157)
    4.  [Conclusion](#org3f1b0ea)
25. [Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis](#org5ef0f47)
    1.  [Notes](#org4bf4c0b)
    2.  [FileName](#orgbc5fbfc)
    3.  [Abstract](#org231f0a8)
    4.  [Conclusion](#org2115d6d)
26. [Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance](#org4561305)
    1.  [Notes](#org06d59b6)
    2.  [FileName](#orgc1a4e40)
    3.  [Abstract](#orgfacbd67)
    4.  [Conclusion](#orgbff455b)
27. [Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes](#org31db702)
    1.  [Notes](#org13d8ba6)
    2.  [FileName](#org4bbe619)
    3.  [Abstract](#org52247f3)
    4.  [Conclusion](#orgb164426)
28. [Killian2019 - Learning to Prescribe Interventions for Tuberculosis Patients Using Digital Adherence Data](#org4eb7f15)
    1.  [Notes](#org1da2420)
    2.  [FileName](#orgc89a3a6)
    3.  [Abstract](#org64567a7)
    4.  [Conclusion](#org8e9092e)
29. [Kouchaki2019 - Application of machine learning techniques to tuberculosis drug resistance analysis](#orgb76b872)
    1.  [Notes](#org6311634)
    2.  [FileName](#org0c00ba1)
    3.  [Abstract](#org731afd1)
    4.  [Conclusion](#orgfc43954)
    5.  [Supplementary](#orgce81f81)
        1.  [Kouchaki2019<sub>ApplicationOfML</sub><sub>Supplementary.pdf</sub>](#org08bf183)
30. [Kouchaki2020 - Multi-Label Random Forest Model for Tuberculosis Drug Resistance Classification and Mutation Ranking](#org350bc02)
    1.  [Notes](#org73d98b4)
    2.  [FileName](#org07a1565)
    3.  [Abstract](#org2399359)
    4.  [Conclusion](#org61ce5d8)
    5.  [Supplementary](#orgf193555)
        1.  [Kouchaki2020<sub>MultiLabel.zip</sub>](#orge82f727)
        2.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>](#orga199105)
        3.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>](#orgb3e5224)
31. [Kovalishyn2019 - Hybrid Design of Isonicotinic Acid Hydrazide Derivatives: Machine Learning Studies, Synthesis and Biological Evaluation of their Anti- tuberculosis Activity](#orgb8f235a)
    1.  [Notes](#org7d40156)
    2.  [FileName](#orgfc72e57)
    3.  [Abstract](#org12c1664)
    4.  [Conclusion](#orga4eb91e)
32. [Kumari2019 - A genetic programming-based approach to identify potential inhibitors of serine protease of Mycobacterium tuberculosis](#orgbce092c)
    1.  [Notes](#org571efc8)
    2.  [FileName](#orgaa7dc2f)
    3.  [Abstract](#org0b1fb1c)
    4.  [Conclusion](#org84c254f)
33. [Lai2019 - Comparison of the Predictive Outcomes for Anti-Tuberculosis Drug-Induced Hepatotoxicity by Different Machine Learning Techniques](#org151e742)
    1.  [Notes](#orgc1fa3b5)
    2.  [FileName](#org8c2c872)
    3.  [Abstract](#org80d7806)
    4.  [Conclusion](#org0c2d590)
34. [Lane2018 - Comparing and Validating Machine Learning Models for Mycobacterium tuberculosis Drug Discovery](#org381822d)
    1.  [Notes](#org83d9892)
    2.  [FileName](#orgb21f75d)
    3.  [Abstract](#org51b0191)
    4.  [Conclusion](#org8a5fce9)
35. [Lin2001 - Conserved codon composition of ribosomal protein coding genes in Escherichia coli, Mycobacterium tuberculosis and Saccharomyces cerevisiae: lessons from supervised machine learning in functional genomics](#orgcbaa13d)
    1.  [Notes](#org374297e)
    2.  [FileName](#orga80e21d)
    3.  [Abstract](#org683b57b)
    4.  [Conclusion](#org3a1845a)
36. [LokeshKumar2019 - Prediction of Multi Drug Resistant Tuberculosis using Machine Learning Techniques](#orgfe08a7f)
    1.  [Notes](#org0064cb6)
    2.  [FileName](#org4eec524)
    3.  [Abstract](#org29d3779)
    4.  [Conclusion](#org6a267da)
37. [Mahe2019 - A large scale evaluation of TBProfiler and Mykrobe for antibiotic resistance prediction in Mycobacterium tuberculosis](#org39d7d66)
    1.  [Notes](#orgc2b1c06)
    2.  [FileName](#orge7b670d)
    3.  [Abstract](#orge778dfd)
    4.  [Conclusion](#org53444b8)
38. [Marino2016 - Computational and Empirical Studies Predict Mycobacterium tuberculosis-Specific T Cells as a Biomarker for Infection Outcome](#org6d6e189)
    1.  [Notes](#orgdfa4f2f)
    2.  [FileName](#org302ae1f)
    3.  [Abstract](#orgf4233b6)
    4.  [Conclusion](#org64737cd)
39. [Marques2019 - Pulmonary nontuberculous mycobacterial infections: presumptive diagnosis based on the international microbiological criteria adopted in the state of São Paulo, Brazil, 2011-2014](#org8049ffa)
    1.  [Notes](#org3c352c1)
    2.  [FileName](#org7729758)
    3.  [Abstract](#org618e04f)
    4.  [Conclusion](#org206108f)
40. [Meldenez2015 - On Combining Multiple-Instance Learning and Active Learning for Computer-Aided Detection of Tuberculosis](#org1653bf3)
    1.  [Notes](#org0b7cde2)
    2.  [FileName](#org32e9cdd)
    3.  [Abstract](#org72d2f54)
    4.  [Conclusion](#org2231265)
41. [Mikusova2016 - Learning from the past for TB drug discovery in the future](#orge7803ce)
    1.  [Notes](#org7f237e1)
    2.  [FileName](#orgc5d37f8)
    3.  [Abstract](#org0080b4f)
    4.  [Conclusion](#org529de8a)
42. [Ngo2019 - Genomic prediction of tuberculosis drug- resistance: benchmarking existing databases and prediction algorithms](#org0c488b6)
    1.  [Notes](#org95c9a78)
    2.  [FileName](#org7d5bbf2)
    3.  [Abstract](#org09f64ef)
    4.  [Conclusion](#org314fb32)
43. [Periwal2011 - Predictive models for anti-tubercular molecules using machine learning on high-throughput biological screening datasets](#orge4f19d6)
    1.  [Notes](#org2129bd0)
    2.  [FileName](#orgb5f9640)
    3.  [Abstract](#org5d0eae6)
    4.  [Conclusion](#org7e10bcc)
44. [Ratnatunga2020 - The Rise of Non-Tuberculosis Mycobacterial Lung Disease](#orgb645b0a)
    1.  [Notes](#org9726f1b)
    2.  [FileName](#orgb172b57)
    3.  [Abstract](#org0239e86)
    4.  [Conclusion](#org806a326)
45. [Romero2020 - Decision tree machine learning applied to bovine tuberculosis risk factors to aid disease control decision making](#orga1797fe)
    1.  [Notes](#org5ec8d01)
    2.  [FileName](#org5b5f4f8)
    3.  [Abstract](#org6b44ddf)
    4.  [Conclusion](#orge69eb5f)
    5.  [Supplementary](#org8d0a561)
        1.  [Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub>](#org0304e06)
46. [Sauer2018 - Feature selection and prediction of treatment failure in tuberculosis](#org835a97c)
    1.  [Notes](#orgfb0e836)
    2.  [FileName](#orgd2199bb)
    3.  [Abstract](#org5f929b4)
    4.  [Conclusion](#org91b1108)
47. [Schiff2018 - Clinical relevance of non-tuberculous mycobacteria isolated from respiratory specimens: seven year experience in a UK hospital](#org875c2bd)
    1.  [Notes](#org54256e4)
    2.  [FileName](#org513da23)
    3.  [Abstract](#orgd4e3a99)
    4.  [Conclusion](#orgb96f233)
48. [Genome-wide Analysis of MDR and XDR Tuberculosis from Belarus: Machine-learning Approach](#org1105648)
    1.  [FileName](#org156a5fc)
    2.  [Notes](#org3a35904)
    3.  [Abstract](#org021d03f)
    4.  [Conclusion](#org53797e6)
49. [Vizcaino2010<sub>ComputatationalPrediction.pdf</sub>](#org34e0186)
    1.  [FileName](#org976bd3a)
    2.  [Notes](#orgdb7d2df)
    3.  [Abstract](#org1d3732a)
    4.  [Conclusion](#org75e56d9)
50. [Waman2019<sub>MycobacterialGenomics.pdf</sub>](#org3800a5a)
    1.  [FileName](#orgaf5cc41)
    2.  [Notes](#org690861b)
    3.  [Abstract](#org5d80423)
    4.  [Conclusion](#orgca97a98)
51. [Wang2020<sub>ADeepLearningBasedMethod.pdf</sub>](#orgacde91a)
    1.  [FileName](#orgbd17c4b)
    2.  [Notes](#org2356d81)
    3.  [Abstract](#org95ca635)
    4.  [Conclusion](#orgb22e05f)
52. [Weiner2017<sub>HighThroughput.pdf</sub>](#orgaab12b9)
    1.  [FileName](#orgb69ddb6)
    2.  [Notes](#org776e57b)
    3.  [Abstract](#org042cad3)
    4.  [Conclusion](#orgb1ff1b2)
53. [Wilder2019<sub>ImprovingTuberculosis.pdf</sub>](#orgc69e8af)
    1.  [FileName](#org5b5f87b)
    2.  [Notes](#orgdc1aef2)
    3.  [Abstract](#orga35152d)
    4.  [Conclusion](#org944e6c3)
54. [Yang2017<sub>MachineLearning.pdf</sub>](#org43f8ceb)
    1.  [FileName](#orgb79a2a0)
    2.  [Notes](#org9be499f)
    3.  [Abstract](#org869a971)
    4.  [Conclusion](#org6d2656f)
55. [Yang2019<sub>DeepAMRForPredicting.pdf</sub>](#orgf9717df)
    1.  [FileName](#org76bb257)
    2.  [Notes](#orgb7cd4a0)
    3.  [Abstract](#orgf0be83d)
    4.  [Conclusion](#orgf5e4664)
56. [Zhang2020<sub>AnIntegratedMLApproach.pdf</sub>](#org2fabacc)
    1.  [FileName](#org2a21944)
    2.  [Notes](#orgf097571)
    3.  [Abstract](#orgbb9c5bd)
    4.  [Conclusion](#orgfe550c7)

<a id="org1b5ac65"></a>

# TODO Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.

<https://www.ncbi.nlm.nih.gov/pubmed/30084331>

<a id="orge876284"></a>

# TODO Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.

<https://www.ncbi.nlm.nih.gov/pubmed/30973110>

<a id="org31bad2b"></a>

# TODO Machine learning and tubercular drug target recognition.

<https://www.ncbi.nlm.nih.gov/pubmed/24245763>

<a id="orgb954c79"></a>

# WAIT Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem

<a id="org3af2075"></a>

## Notes

- Survey paper
- Could be used build up the background for the need for research in NTM

<a id="org40412d9"></a>

## FileName

- Ahmed2020<sub>NonTuberculosisMycoANeglectedAndEmergingProblem.pdf</sub>

<a id="org0bbd057"></a>

## Abstract

- The incidence of NTM infections is rising owing to a growing population of
  immunocompromised and vulnerable individuals, complex medical and surgical
  procedures, as well as increased awareness and diagnostic capabilities.

<a id="orgea30536"></a>

## FutureWork

- Simpler and more rapid tests would be ideal in making the diagnosis of NTM
  disease more quickly. However, the development of such tests is complicated
  by the large number of NTM species that can cause disease and by the criteria
  for disease, which require more than just the presence of the microorganism.
  As a result, NTM disease will require multiple diagnostic parameters, leading
  to delays and higher costs. More specific biomarkers to monitor the progress
  of treatment and test cure are lacking.

<a id="org3cabcbc"></a>

# STRT Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm

<a id="org29168cf"></a>

## Notes

- Can adopt a similar approach towards the classification of NTB lineages
- Can alter the algorithm to XGBoost to Neural Networks to improve accuracy

<a id="org0a0102a"></a>

## FileName

- Aze2015<sub>GenomicsAndMachine.pdf</sub>

<a id="orgd4b5fa7"></a>

## Abstract

- Mycobacterium tuberculosis complex taxonomy has rapidly evolved in the last 20
  years through intensive clinical isolation, advances in sequencing and in the
  description of fast-evolving loci (CRISPR and MIRU-VNTR)

- The objectives of this study were 1) to explicit the consensus that exists
  between the alternative taxonomies, and 2) to provide an on-line tool to ease
  classification of new isolates. Genotyping (24-VNTR, 43-spacers spoli-
  gotypes, IS6110-RFLP) was undertaken for 3,454 clinical isolates from the
  Netherlands (2004-2008). The resulting database was enlarged with African
  isolates to include most human tuberculosis diversity. Assignations were
  obtained using TB-Lineage, MIRU- VNTRPlus, SITVITWEB and an algorithm from
  Borile et al

- Original and consensus assignations of the all isolates from the database were
  subsequently implemented into an ensemble learning approach based on Machine
  Learning tool Weka to derive a classification scheme.

- When applied to independent datasets, it was able to suggest new sublineages
  such as pseudo-Beijing. This Lineage Prediction tool, efficient on 15-MIRU,
  24-VNTR and spoligotype data is available on the web interface “TBminer.”

- We successfully used Machine Learning on a large dataset to set up and make
  available the first consensual taxonomy for human Mycobacterium tuberculosis
  complex

<a id="orge9822a8"></a>

## Conclusion

- We developed an approach making use of standard typing data for human M.

tuberculosis isolates to infer a consensual taxonomy concordant with most
up-to-date data on Whole Genome Sequencing diversity.

- We hope that the same approach can benefit other human pathogens having
  alternative taxonomies according to Serotypes, CRISPRtypes, MLST-types such as
  Salmo- nella, Listeria, Brucella, and more broadly to other organisms such as
  bacterial plant- pathogens.

<a id="orgcc48f60"></a>

# [-] Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper

<a id="org19f4758"></a>

## Notes

- Not directly related to (N)TB
- Survey paper
- Analyzes various ML techniques
- Mentions various drug-target-interaction databases

<a id="org42711fd"></a>

## FileName

- Bagherian2019<sub>MachineLearning.pdf</sub>

<a id="org08f105d"></a>

## Abstract

- There is a need to develop novel and efficient prediction approaches in order
  to avoid costly and laborious yet not-always-deterministic experiments to
  determine drug–target interactions (DTIs) by experiments alone
- In this article, we describe the data required for the task of DTI prediction
  followed by a comprehensive catalog consisting of machine learning methods and
  databases, which have been proposed and utilized to predict DTIs
- The advantages and disadvantages of each set of methods are also briefly
  discussed

<a id="orgb610be2"></a>

## Conclusion

- Almost all the methods used in DTI prediction, particularly similarity-based
  methods, heavily rely on assertions concerning similar drugs and similar
  targets, the type of database used for the prediction plays a significant
  role.
- Future predictions should rely on more comprehensive inter- nal databases,
  which would require a significant effort to map and curate data across the
  sources that utilize different ways to define, name and identify the drugs and
  targets.

<a id="org99c7d50"></a>

# STRT Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities

<a id="org2bb03ea"></a>

## Notes

- Image processing algorithms, CNN
- They proposed an online database of X-ray images - by now this might have been developed

<a id="org0de4fef"></a>

## FileName

- Cao2016<sub>ImprovingDiagnosticsUsingDL.pdf</sub>

<a id="orgfc429aa"></a>

## Abstract

- The combination of machine learning and mobile computing techniques offers a
  unique opportunity to accelerate the TB diagnosis among these communities

- We first introduce major technique barriers and proposed system architecture

- The first activity aims to develop large-scale, real-world and well-annotated
  X-ray image database dedicated for automated TB screening.

- The second research activity focus on developing effective and efficient
  computational models (in particularly, deep convolutional neural networks
  (CNN)-based models) to classify the image into different category of TB
  manifestations

- Our future work includes: (1) to further improve the performance of the
  algorithms; and (2) to deploy our system in the city of Carabayllo in Perú, a
  densely occupied urban community and high-burden TB

<a id="orgb221989"></a>

## Conclusion

- The first progress is to build a large-scale, real-world, and well-annotated chest X- ray image
  database dedicated for TB screening.

- The second progress is to develop effective and efficient computational model for TB manifestation
  categorization.

- We will continue to develop the large scale, real-world X-ray TB database, as
  well as continuing to improve the performance of the computing algorithms

<a id="org5b59f6e"></a>

# STRT Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data

<a id="org93782d0"></a>

## Notes

- A good in-depth ML based paper
- They&rsquo;ve shared high-level implementation details briefly
- Hyperparameters are also in the supplementary material
- I was able to find the code on Github
  <https://github.com/biosharp-dotnet/wdnn>
- Could apply the same techniques to NTM
- Could change the algorithm and apply on the same data
- The author built upon the previous paper by improving the accuracy of the ML model - we could do the same with NTM

<a id="org5583f25"></a>

## FileName

- Chen2018<sub>DeepLearningPredictsTuberculosis.pdf</sub>

<a id="org43d726a"></a>

## Abstract

- Using targeted or whole genome sequencing and conventional drug resistance
  phenotyping data from 3,601 Mycobacterium tuberculosis strains, 1,228 of which
  were multidrug resistant, we implemented the first multitask deep learning
  framework to predict phenotypic drug resistance to 10 anti-tubercular drugs

- The proposed wide and deep neural network (WDNN) achieved improved predictive
  performance compared to regularized logistic regression and random forest

<a id="org8ff37b6"></a>

## Conclusion

- We demonstrate that L2 regression and MD-WDNN trained on a larger diverse
  dataset using a method of aggregating rare variants outperforms our previously
  reported random forest model. (Chen, 2018)

- A few prior studies have utilized algorithmic or machine learning methods
  using genomic data to account for the complex relationship between genotype
  and drug resistance in MTB

<a id="orgb788691"></a>

# STRT Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction

<a id="orgcc2d08c"></a>

## Notes

- A good in-depth ML based paper
- Could apply the same techniques to NTM
- Could change the algorithm and apply on the same data

<a id="org4f50588"></a>

## FileName

- Chen2019<sub>BeyondMultiDrugResistance.pdf</sub>

<a id="org4626bae"></a>

## Abstract

- Whole genome sequencing of clinical Mycobacterium tuberculosis isolates
  promises to circumvent the long wait times and limited scope of conventional
  phenotypic antimicrobial susceptibility, but gaps remain for predicting
  phenotype accurately from genotypic data especially for certain drugs.

- We collected targeted or whole genome sequencing and conventional drug
  resistance phenotyping data from 3601 Mycobacterium tuberculosis strains
  enriched for resistance to first- and second-line drugs, with 1228 multidrug
  resistant strains

- We investigated the utility of (1) rare variants and variants known to be
  determinants of resistance for at least one drug and (2) machine and
  statistical learning architectures in predicting phenotypic drug resistance to
  10 anti-tuberculosis drugs

<a id="org5636ab1"></a>

## Conclusion

- Our approach allowed us to gain new insights into the relative im- portance of
  specific genetic variants for resistance prediction

- In summary, we present an exploration of machine learning and traditional
  statistical models to identify the resistance of MTB isolates to 10
  anti-tuberculosis drugs from whole genome sequencing data.

- Our models trained on rare and common genetic variants achieved
  state-of-the-art performance on a large, aggregated TB dataset, with
  prediction times of less than a tenth of a second, demonstrating the efficacy
  of our model as a diagnostic tool for MTB drug resistance.

- The MD-WDNN represented the first multidrug model to our knowledge that

incorporated a high number of genotypic predictors known to be important to
determining resistance for one or more included drugs

<a id="org67edce3"></a>

# [-] Ching2018 - Opportunities and obstacles for deep learning in biology and medicine

<a id="orgb1f040d"></a>

## Notes

- Found the code <https://github.com/greenelab/deep-review>
- Can use this paper to build up the background for using DL in MTB and NTM analysis

<a id="org1317902"></a>

## FileName

- Ching2018<sub>OpportunitiesAndObstaclesForDLInBiology.pdf</sub>

<a id="org0a6ad34"></a>

## Abstract

- Following from an extensive literature review, we find that deep learning has
  yet to revolutionize biomedicine or definitively resolve any of the most
  pressing challenges in the field, but promising advances have been made on the
  prior state of the art
- The limited amount of labelled data for training presents problems in some
  domains, as do legal and privacy constraints on work with sensitive health
  records.

<a id="org7683ee6"></a>

## Conclusion

- The most challenging tasks may be those in which predictions are used directly
  for downstream model- ling or decision-making, especially in the clinic. As an
  example, errors in sequence variant calling will be amplified if they are used
  directly for genome-wide association studies.

- In addition, the stochasticity and complexity of biological sys- tems imply
  that for some problems, for instance, predicting gene regulation in disease,
  perfect accuracy will be unattainable

<a id="org2166fc7"></a>

# STRT Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data

<a id="orgb1b0a45"></a>

## Notes

- Need to request code from authors
- Good ML oriented paper

<a id="org9f02c86"></a>

## FileName

- Deelder2019<sub>MLPredictsAccurately.pdf</sub>

<a id="org06f5679"></a>

## Abstract

- We applied machine learning approaches to 16,688 M. tuberculosis isolates that
  have undergone WGS and laboratory drug-susceptibility testing (DST) across 14
  antituberculosis drugs, with 22.5% of samples being multidrug resistant and
  2.1% being extensively drug resistant.

- We fitted separate models for each drug, with and without “co-occurrent
  resistance” markers known to be causing resistance to drugs other than the one
  of interest. Predictive performance was measured using sensitivity,
  specificity, and the area under the receiver operating characteristic curve,
  assuming DST results as the gold standard.

- Overall, the gradient-boosted-tree models performed better than the
  classification-tree models.

- Our work demonstrates the utility of machine learning as a flexible approach
  to drug resistance prediction that is able to accommodate a much larger number
  of predictors and to summarize their predictive ability, thus assisting
  clinical decision making and single nucleotide polymorphism detection in an
  era of increasing WGS data generation.

<a id="orga1a1679"></a>

## Conclusion

- The predictive performance was highest for resistance to first-line drugs,
  and multidrug-resistant tuberculosis (area under the receiver operating characteristic curve above
  96%), and lowest for third- line drugs (area under the curve below 85%).

<a id="org079f508"></a>

## Supplementary

<a id="orgb12e761"></a>

### Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub>

<a id="org458ec36"></a>

# [?] Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis

<a id="org99f1375"></a>

## Notes

- This requires more knowledge of chemistry and perhaps also relies on lab based experiments

<a id="org8605625"></a>

## FileName

- Djout2016<sub>PredictiveModeling.pdf</sub>

<a id="org32a32ef"></a>

## Abstract

- ThyX is an essential thymidylate synthase

that is mechanistically and structurally unrelated to the human enzyme. DNA
gyrase is a DNA topoisomerase present in bacteria and plants but not animals.
The current study set out to understand the structure-activity relationships of
these targets in Mtb using a combination of cheminformatics and in vitro
screening.

<a id="org38d5d70"></a>

## Conclusion

<a id="orgc9f7a4d"></a>

# STRT Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports

<a id="orgea645b2"></a>

## Notes

- This is a survey report about NTM across a couple decades
- Could be used in building up the background context

<a id="org96912b3"></a>

## FileName

- Donohue2018<sub>IncreasingNTMReportingRates.pdf</sub>

<a id="org6e12d69"></a>

## Abstract

- Changes in NTM diversity were observed in complex/groups known to be
  clinically significant. Between 1994 and 2014 the rate implicating M.
  abscesses-chelonae group and M. avium complex increased by 322 and 149%,
  respectively.

<a id="orgf5abeb8"></a>

## Conclusion

- Knowledge of the NTM species most frequently associ- ated with adverse human
  health consequences will assist epidemiological investigators to identify
  likely sources and modes of exposure associated with these types of in-
  fections.
- As the U.S. population ages, the public health burden from NTM associated
  disorders is expected to increase.

<a id="orgae365bb"></a>

# [-] Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence

<a id="orgb6f1e8f"></a>

## Notes

- Survey paper
- Can be used for building background context

<a id="org84190c4"></a>

## FileName

- Doshi2017<sub>TuberculosisControl.pdf</sub>

<a id="orgd40ebae"></a>

## Abstract

<a id="org02569fd"></a>

## Conclusion

<a id="org9235f30"></a>

# [X] Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models

<a id="org8c00406"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org811debe"></a>

## FileName

- Ekins2013<sub>EnhancingHitIdentification.pdf</sub>

<a id="org415e17a"></a>

## Abstract

- High-throughput screening (HTS) in whole cells is widely pursued to find
  compounds active against Mycobacterium tuberculosis (Mtb) for further
  development towards new tuberculosis (TB) drugs
- We have recently built and validated Bayesian machine learning models for
  predicting compounds with activity against Mtb based on publicly available
  large-scale HTS data from the Tuberculosis Antimicrobial Acquisition
  Coordinating Facility

<a id="org37b23d4"></a>

## Conclusion

<a id="org4c5973b"></a>

# [X] Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation

<a id="org0dd0c9c"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orga1cbd7d"></a>

## FileName

- Ekins2013<sub>FusingDualEvent.pdf</sub>

<a id="org50294b2"></a>

## Abstract

- A cheminformatics data-fusion

approach followed by Bayesian machine learning, Support Vector Machine, or Recursive
Partitioning model development (based on publicly available Mtb screening data) was used to
compare individual data sets and subsequent combined models.

- We demonstrate that

combining three data sets incorporating antitubercular and cytotoxicity data in Vero cells
from our previous screens results in external validation receiver operator curve (ROC) of
0.83 (Bayesian or RP Forest)

<a id="orgfb3a7e6"></a>

## Conclusion

<a id="org3f15d68"></a>

# [X] Ekins2014<sub>AreBiggerDataSets.pdf</sub>

<a id="org7273c44"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org6fd457d"></a>

## Title

<a id="org0b492e2"></a>

## Abstract

<a id="org6bf8e62"></a>

## Conclusion

<a id="org79dc8cf"></a>

# [X] Ekins2014<sub>CombiningComputationalMethods.pdf</sub>

<a id="org5d05756"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org1565661"></a>

## Title

<a id="org9d1af0f"></a>

## Abstract

<a id="orged6a132"></a>

## Conclusion

<a id="orge30f29e"></a>

# [X] Ekins2016<sub>MachineLearningModel.pdf</sub>

<a id="orge7ae988"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org925101c"></a>

## Title

<a id="org52ffe48"></a>

## Abstract

<a id="org41097fb"></a>

## Conclusion

<a id="orgdcd4e99"></a>

# [X] Ekins2017<sub>MachineLearning.pdf</sub>

<a id="orgb27b3d7"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orgdc686e2"></a>

## Title

<a id="orgc28d943"></a>

## Abstract

<a id="org8936ad3"></a>

## Conclusion

<a id="org32d00af"></a>

# [-] Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis

<a id="org9265909"></a>

## Notes

- Image processing
- Mentiones the database source, NIH
- Has CSV as supplementary data

<a id="org095d404"></a>

## FileName

- Engle2019<sub>PerformanceOfQureAI.pdf</sub>

<a id="org5b7be8a"></a>

## Abstract

- Availability of trained radiologists for fast processing of CXRs in regions
  burdened with tuberculosis always has been a challenge, affecting both timely
  diagnosis and patient monitoring.

- The Qure.ai qXR arti- ficial intelligence automated CXR interpretation tool,
  was blind-tested on the 346 radiologist- annotated CXRs from the TBPP
  database.

- We conclude that applying fully automated Qure.ai CXR analysis tool is useful
  for fast, accurate, uniform, large-scale CXR annotation assistance, as it
  performed well even for DR-TB cases that were not used for initial training.

<a id="orga9f1e81"></a>

## Conclusion

- We demonstrated that the Qure.ai qXR CXR annotations for cavity, nodule,
  pleural effusion, and hilar lymphadenopathy, are in statistical agreement with
  radiologist CXR annotations.

- A corresponding analysis of patient case matched
  CT image annotations recorded within 30 days of a CXR supports this result,
  demonstrating statistical significance for cavity, pleural effusion, hilar
  lymphadenopathy, and atelectasis.

<a id="org096a2eb"></a>

# [-] Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay

<a id="orgd1f615e"></a>

## Notes

- Survey paper

<a id="org6a4e792"></a>

## FileName

- Greif2020<sub>FrequencyOfNTM.pdf</sub>

<a id="org907b848"></a>

## Abstract

- Nontuberculous mycobacteria (NTM) increasingly are recognized as
  opportunistic pathogens of humans.
- NTM species distribution is well documented in Europe and North America

<a id="orgc52d35e"></a>

## Conclusion

- Our findings indicate that rate of NTM have risen nearly 5-fold (4.79) in
  Uruguay, from 0.33 cas- es/100,000 inhabitants in 2006 to 1.57 cases/100,000
  inhabitants in 2018.

<a id="org9b9253c"></a>

# [-] Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data

<a id="org3fb7600"></a>

## Notes

- Can perform the mentioned analysis on the MTB and NTM data
- The paper focuses on the factors affecting the application and clinical application of ML models

<a id="orgd5f6287"></a>

## FileName

- Hicks2019<sub>EvaluationOfParameters.pdf</sub>

<a id="org9cb8098"></a>

## Abstract

- There has been no systematic evaluation of factors that may influence
  performance of ML models, how they might apply to and vary across clinical
  populations, and what the implications might be in the clinical setting

- We performed a meta-analysis of seven large Neisseria gonorrhoeae datasets, as
  well as Klebsiella pneumoniae and Acinetobacter baumannii datasets, with whole
  genome sequence data and antibiotic susceptibility phenotypes using set
  covering machine classifi- cation, random forest classification, and random
  forest regression models to predict resis- tance phenotypes from genotype.

- We demonstrate how model performance varies by drug, dataset, resistance
  metric, and species, reflecting the complexities of generating clinically
  relevant conclusions from machine learning-derived models.

- Our results highlight significant biological and technical challenges
  facing the application of machine learning-based prediction of antibiotic
  resistance as a diagnostic tool.

<a id="orgfb2bce0"></a>

## Conclusion

<a id="org7478609"></a>

# STRT Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models

<a id="org8cbb11d"></a>

## Notes

- Can use a different algorithm (XgBoost) on the dataset
- Sample data and complete code is shared
  <https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics>
- Need to find the full dataset of 4000 surveyed patients

<a id="org5cdf663"></a>

## FileName

- Hussain2018<sub>PredictingTreatmentOutcomes.pdf</sub>

<a id="orgd1ca157"></a>

## Abstract

- Tuberculosis (TB) is a deadly contagious disease and a serious global health
  problem. It is curable but due to its lengthy treatment process, a patient is
  likely to leave the treatment incomplete, leading to a more lethal, drug
  resistant form of disease.

- We aim to facilitate TB programs by predicting the outcome of the treatment of
  a particular patient at the start of treatment so that their health workers
  can be utilized in a targeted and cost-effective way.

- The problem was modeled as a classification problem, and the outcome of
  treatment was predicted using state-of-art implementations of 3 machine learning algorithms

<a id="org3f1b0ea"></a>

## Conclusion

- Empirical results using data of more than 4000 patients reveal that we can
  predict with more than 76% accuracy whether a TB patient would complete his (or her) treatment or not
-

<a id="org5ef0f47"></a>

# STRT Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis

<a id="org4bf4c0b"></a>

## Notes

- Used many ML algorithms, 85% accuracy
- Could use XgBoost
- Could conduct a similar study for NTM
- Mentions many online bio databases

<a id="orgbc5fbfc"></a>

## FileName

- Jamal2020<sub>AIAndMLBasedPredictive.pdf</sub>

<a id="org231f0a8"></a>

## Abstract

- The present work is a computational framework that uses artificial
  intelligence (AI) based machine learning (ML) approaches for predicting
  resistance in the genes rpoB, inhA, katG, pncA, gyrA and gyrB for the drugs
  rifampicin, isoniazid, pyrazinamide and fluoroquinolones

- The single nucleotide variations were represented by several sequence and
  structural features that indicate the influence of mutations on the target
  protein coded by each gene.

<a id="org2115d6d"></a>

## Conclusion

- The present study proposes an AI/ML method to classify resistant and susceptible
  mutations in TB and predict novel resistance conferring mutations.

- The impact of the reported mutations was captured in the form of changes in the amino acid
  residues, and the consequent change in properties vis-a-vis wild-type and mutant
  proteins and represented as features used to train the models.

- The classification model was generated for each gene and predictions were made for
  SNVs linked with each gene for each drug

<a id="org4561305"></a>

# STRT Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance

<a id="org06d59b6"></a>

## Notes

- They developed and online platform

- Can be applied to NTM

- Code is available
  <https://github.com/biosharp-dotnet/microbial_AMR_ML>

<a id="orgc1a4e40"></a>

## FileName

- Kavvas2018<sub>MachineLearning</sub><sub>.pdf</sub>

<a id="orgfacbd67"></a>

## Abstract

- The many publicly available datasets describing its AMR characteristics demand
  disparate data-type analyses.

- Here, we develop a reference strain-agnostic computational platform that uses
  machine learning approaches, complemented by both genetic interaction analysis
  and 3D structural mutation-mapping, to identify signatures of AMR evolution to
  13 antibiotics.

- First, a pan-genome analysis shows that M. tuberculosis is highly conserved
  with sequenced variation concentrated in PE/PPE/PGRS genes.

- Second, the platform corroborates 33 genes known to confer resistance and identifies 24
  new genetic signatures of AMR.

- Third, 97 epistatic interactions across 10 resistance classes are revealed.

- Fourth, detailed structural analysis of these
  genes yields mechanistic bases for their selection.

- The platform can be used to study other human pathogens

<a id="orgbff455b"></a>

## Conclusion

- The platform was applied to 1595 sequenced strains to yield results in four
  categories: pan-genome properties, identification of genes conferring
  antibiotic resistance, their epistatic interactions, and protein structure
  based mechanistic insights.

<a id="org31db702"></a>

# [-] Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes

<a id="org13d8ba6"></a>

## Notes

- Uses multiple bio and ML packages in R
- Doesn&rsquo;t use XgBoost or H20
- Code hasn&rsquo;t been shared

<a id="org4bbe619"></a>

## FileName

- Khanna2018<sub>EnsembleTechnique.pdf</sub>

<a id="org52247f3"></a>

## Abstract

- Development of an effective machine-learning model for T-cell Mycobacterium
  tuberculosis (M. tuberculosis) epitopes is beneficial for saving biologist’s
  time and effort for identifying epitope in a targeted antigen.

- The proposed method is able to work with variable length epitopes having size
  even greater than 9-mers. Identification of T-cell or B-cell epitopes in the
  targeted antigen is the main goal in designing epitope-based vaccine,
  immune-diagnostic tests and antibody production

- The proposed ensemble model achieves 82.0% accuracy, 0.89 specificity, 0.77
  sensitivity with repeated k-fold cross-validation having average accuracy of
  80.61%.

<a id="orgb164426"></a>

## Conclusion

- Three models (decision tree, avNNet and RRF) have been used to design the
  ensemble model which produces high accuracy, Gini, AUC, specificity and
  sensitivity.

- To validate and compare the proposed ensemble model, NetMHC 4.0, NetMHC 2.3
  and CTLpred servers have been considered.

<a id="org4eb7f15"></a>

# [-] Killian2019 - Learning to Prescribe Interventions for Tuberculosis Patients Using Digital Adherence Data

<a id="org1da2420"></a>

## Notes

- Wasn&rsquo;t able to find the data or the code in the paper

<a id="orgc89a3a6"></a>

## FileName

- Killian2019<sub>LearningToPrescribe.pdf</sub>

<a id="org64567a7"></a>

## Abstract

- We analyze data from one city served by 99DOTS, a phone-call- based DAT
  deployed for Tuberculosis (TB) treatment in India where nearly 3 million
  people are afflicted with the disease each year

- We then construct a deep learning model, demonstrate its interpretability, and
  show how it can be adapted and trained in three different clinical scenarios
  to better target and improve patient care.

<a id="org8e9092e"></a>

## Conclusion

- We finally show that tailoring our model for a specific intervention via
  decision-focused learning can improve performance by a further 15%.

<a id="orgb76b872"></a>

# WAIT Kouchaki2019 - Application of machine learning techniques to tuberculosis drug resistance analysis

<a id="org6311634"></a>

## Notes

- I&rsquo;ve asked the author to provide the source code via ResearchGate
- They&rsquo;ve used 6 ML classifiers, not XgBoost or NeuralNetworks

<a id="org0c00ba1"></a>

## FileName

- Kouchaki2019<sub>ApplicationOfML.pdf</sub>

<a id="org731afd1"></a>

## Abstract

- Machine learning methods have been widely applied for timely predicting
  resistance of MTB given a specific drug and identifying resistance markers.
  However, they have been not validated on a large cohort of MTB samples from
  multi-centers across the world in terms of resistance prediction and
  resistance marker identification

- Several machine learning classifiers and linear dimension re- duction
  techniques were developed and compared for a cohort of 13 402 isolates
  collected from 16 countries across 6 continents and tested 11 drugs.

<a id="orgfc43954"></a>

## Conclusion

- Developing machine learning techniques is especially important to im- prove TB
  resistance prediction, mainly sensitivity for less-studied drugs.
-

<a id="orgce81f81"></a>

## Supplementary

<a id="org08bf183"></a>

### Kouchaki2019<sub>ApplicationOfML</sub><sub>Supplementary.pdf</sub>

<a id="org350bc02"></a>

# STRT Kouchaki2020 - Multi-Label Random Forest Model for Tuberculosis Drug Resistance Classification and Mutation Ranking

<a id="org73d98b4"></a>

## Notes

- Source code and data are both available

<a id="org07a1565"></a>

## FileName

- Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub>

<a id="org2399359"></a>

## Abstract

- Due to standard regimens for the use of first-line antibiotics, resistance
  co-occurrence, in which samples are resistant to multiple drugs, is common.
  Analysing all drugs simultaneously should therefore enable patterns reflecting
  resistance co-occurrence to be exploited for resistance prediction

- Here, multi-label random forest (MLRF) models are compared with single-label
  random forest (SLRF) for both predicting phenotypic resistance from whole
  genome sequences and identifying important mutations for better prediction of
  four first-line drugs in a dataset of 13402 Mycobacterium tuberculosis
  isolates.

<a id="org61ce5d8"></a>

## Conclusion

<a id="orgf193555"></a>

## Supplementary

<a id="orge82f727"></a>

### Kouchaki2020<sub>MultiLabel.zip</sub>

<a id="orga199105"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>

<a id="orgb3e5224"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>

<a id="orgb8f235a"></a>

# [-] Kovalishyn2019 - Hybrid Design of Isonicotinic Acid Hydrazide Derivatives: Machine Learning Studies, Synthesis and Biological Evaluation of their Anti- tuberculosis Activity

<a id="org7d40156"></a>

## Notes

- Is inclined more towards Chemistry (reagents) wrt TB

<a id="orgfc72e57"></a>

## FileName

- Kovalishyn2019<sub>HybridDesign.pdf</sub>

<a id="org12c1664"></a>

## Abstract

- The goal of this work is to identify potent small molecule anti-TB agents by
  machine learning, synthesis and biological evaluation.

- The On-line Chemical Database and Modeling Environment (OCHEM) was used to
  build pre- dictive machine learning models. Seven compounds were synthesized
  and tested in vitro for their an- titubercular activity against H37Rv and
  resistant Mtb strains.

-

<a id="orga4eb91e"></a>

## Conclusion

- The goal of this work was to investigate isoniazid- thiazole hybrid compounds
  taking into consideration the key structural futures known to be important for
  their antituber- cular activity.

- The created binary classification models demonstrated good balanced accuracy.
  The predictive power of the binary classifiers was tested by a 5-fold cross-
  validation and resulted in balanced accuracies of 61–78 %. Seven new compounds
  were synthesized and their activity against Mtb H37Rv and MDR strains was
  evaluated at two concentrations (1.5 and 50 μМ) using rifampin and isoniazid
  as controls.

<a id="orgbce092c"></a>

# [-] Kumari2019 - A genetic programming-based approach to identify potential inhibitors of serine protease of Mycobacterium tuberculosis

<a id="org571efc8"></a>

## Notes

- Uses Genetic algorithms and ML algorithms as well
- Data or code isn&rsquo;t available

<a id="orgaa7dc2f"></a>

## FileName

- Kumari2019<sub>AGeneticProgramming.pdf</sub>

<a id="org0b1fb1c"></a>

## Abstract

- We applied genetic programming approaches to understand the impact of
  descriptors on inhibitory effects of serine protease inhibitors of
  Mycobacterium tuberculosis (Mtb) and the discovery of new in- hibitors as drug
  candidates

- The experimental dataset of serine protease inhibitors of Mtb descriptors was
  optimized by genetic algorithm (GA) along with the correlation-based feature
  selection (CFS) in order to develop predictive models using machine-learning
  algorithms.

- The use of CFS-GA with random forest-enhanced classification accuracy and
  predicted new serine protease inhibitors of Mtb, which can be used for better
  drug development against tuberculosis.

<a id="org84c254f"></a>

## Conclusion

- Comparative analysis of statistical measures for four different
  machine-learning algorithms revealed that RF models with CFS-GA performed
  better than the others.

- The best RF models with CFS-GA predicted new serine protease inhibitors of
  Mtb, which can be used for better drug development against tuberculosis

<a id="org151e742"></a>

# [-] Lai2019 - Comparison of the Predictive Outcomes for Anti-Tuberculosis Drug-Induced Hepatotoxicity by Different Machine Learning Techniques

<a id="orgc1fa3b5"></a>

## Notes

- Good ML oriented paper
- Can&rsquo;t find the data or code

<a id="org8c2c872"></a>

## FileName

- Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub>

<a id="org80d7806"></a>

## Abstract

- The study compared the performance of artificial neural network, support
  vector machine and random forest on predicting anti-tuberculosis drugs induced
  hepatotoxicity.

- Combining genomic and clinical data can further increase the area under
  receiver operating characteristic curve than using genomic or clinical data
  alone.

- Nine models were created with 7 clinical factors and 4 genotypes

<a id="org0c2d590"></a>

## Conclusion

<a id="org381822d"></a>

# [-] Lane2018 - Comparing and Validating Machine Learning Models for Mycobacterium tuberculosis Drug Discovery

<a id="org83d9892"></a>

## Notes

- Involves chemistry

<a id="orgb21f75d"></a>

## FileName

- Lane2018<sub>ComparingAndValidating.pdf</sub>

<a id="org51b0191"></a>

## Abstract

- The need to develop new treatments for those infected with Mycobacterium
  tuberculosis (Mtb) has led to many large-scale phenotypic screens and many
  thousands of new active compounds identified in vitro

- We have curated small molecule Mtb data and developed new models with a total
  of 18,886 molecules with activity cutoffs of 10 μM, 1 μM, and 100 nM. These
  data sets were used to evaluate different machine learning methods (including
  deep learning) and metrics and to generate predictions for additional
  molecules published in 2017

- We have also compared these models with additional machine learning algorithms
  showing Bayesian machine learning models con- structed with literature Mtb
  data generated by different laboratories generally were equivalent to or
  outperformed deep neural networks with external test sets.

<a id="org8a5fce9"></a>

## Conclusion

<a id="orgcbaa13d"></a>

# [X] Lin2001 - Conserved codon composition of ribosomal protein coding genes in Escherichia coli, Mycobacterium tuberculosis and Saccharomyces cerevisiae: lessons from supervised machine learning in functional genomics

<a id="org374297e"></a>

## Notes

- Deals with codons and homologs etc. No idea.

<a id="orga80e21d"></a>

## FileName

- Lin2001<sub>ConservedCodonComposition.pdf</sub>

<a id="org683b57b"></a>

## Abstract

<a id="org3a1845a"></a>

## Conclusion

<a id="orgfe08a7f"></a>

# [?] LokeshKumar2019 - Prediction of Multi Drug Resistant Tuberculosis using Machine Learning Techniques

<a id="org0064cb6"></a>

## Notes

- Survey paper
- Offers quick overview of various algorithms with accuracy and other characteristics

<a id="org4eec524"></a>

## FileName

- LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub>

<a id="org29d3779"></a>

## Abstract

- We review different Machine Learning Approaches to predict MDR-TB. Different
  feature estimation methods, execution of distinct machine learning models also
  have been explored.
- AdaBoost ensemble algorithm gave the highest accuracy of all

<a id="org6a267da"></a>

## Conclusion

<a id="org39d7d66"></a>

# [X] Mahe2019 - A large scale evaluation of TBProfiler and Mykrobe for antibiotic resistance prediction in Mycobacterium tuberculosis

<a id="orgc2b1c06"></a>

## Notes

- Doesn&rsquo;t directly relate to the dissertation

<a id="orge7b670d"></a>

## FileName

- Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub>

<a id="orge778dfd"></a>

## Abstract

<a id="org53444b8"></a>

## Conclusion

<a id="org6d6e189"></a>

# [-] Marino2016 - Computational and Empirical Studies Predict Mycobacterium tuberculosis-Specific T Cells as a Biomarker for Infection Outcome

<a id="orgdfa4f2f"></a>

## Notes

- They conducted their own experiments

<a id="org302ae1f"></a>

## FileName

- Marino2016<sub>ComputationalAndEmpirical.pdf</sub>

<a id="orgf4233b6"></a>

## Abstract

- Identifying biomarkers for tuberculosis (TB) is an ongoing challenge in
  developing immuno- logical correlates of infection outcome and protection.

- To effectively predict biomarkers for infection progression in any disease,
  including TB, large amounts of experi- mental data are required to reach
  statistical power and make accurate predictions.

- Machine learning techniques were used to interrogate the experimental NHP
  datasets without identifying any potential TB biomarker. In parallel, we used
  our extensive novel NHP datasets to build and calibrate a multi-organ
  computational model that combines what is occurring at the site of infection
  (e.g., lung) at a single granuloma scale with blood level readouts that can be
  tracked in mon- keys and humans.

<a id="org64737cd"></a>

## Conclusion

<a id="org8049ffa"></a>

# [-] Marques2019 - Pulmonary nontuberculous mycobacterial infections: presumptive diagnosis based on the international microbiological criteria adopted in the state of São Paulo, Brazil, 2011-2014

<a id="org3c352c1"></a>

## Notes

- Survey paper for NTM
- Can be used for building up the background

<a id="org7729758"></a>

## FileName

- Marques2019<sub>PulmonaryNTMInfections.pdf</sub>

<a id="org618e04f"></a>

## Abstract

- The objective of the present study was to determine the frequency at which the
  various NTM species are identified among selected individuals and in potential
  cases of pulmonary nontuberculous mycobacterial infection.

<a id="org206108f"></a>

## Conclusion

- Although various NTM species were identified among the individuals studied,
  those presumptively identified most frequently on the basis of the
  microbiological criteria adopted in the state of São Paulo were the ones that
  are most commonly associated with pulmonary nontuberculous mycobacterial
  infection worldwide or in specific geographic regions.

<a id="org1653bf3"></a>

# [-] Meldenez2015 - On Combining Multiple-Instance Learning and Active Learning for Computer-Aided Detection of Tuberculosis

<a id="org0b7cde2"></a>

## Notes

- Image processing, radiography

<a id="org32e9cdd"></a>

## FileName

- Meldenez2015<sub>OnCombiningMultiple.pdf</sub>

<a id="org72d2f54"></a>

## Abstract

- The major advantage of multiple-instance learning (MIL) applied to a
  computer-aided detection (CAD) system is that it allows optimizing the latter
  with case-level labels instead of accurate lesion outlines as traditionally
  required for a supervised approach.

- As shown in previous work, a MIL-based CAD system can perform comparably to
  its supervised counterpart considering complex tasks such as chest radiograph
  scoring in tuberculosis (TB) detection. However, despite this remarkable
  achievement, the uncertainty inherent to MIL can lead to a less satisfactory
  outcome if analysis at lower levels (e.g., regions or pixels) is needed.

- To minimize the labeling effort, we develop a novel instance selection
  mechanism that exploits the MIL problem definition through one-class
  classification.

<a id="org2231265"></a>

## Conclusion

<a id="orge7803ce"></a>

# [-] Mikusova2016 - Learning from the past for TB drug discovery in the future

<a id="org7f237e1"></a>

## Notes

- Involves chemistry
- Survey paper
- Can be used to build up the context for Drug-Target-Interaction

<a id="orgc5d37f8"></a>

## FileName

- Mikusova2016<sub>LearningFromThePast.pdf</sub>

<a id="org0080b4f"></a>

## Abstract

- Tuberculosis drug discovery has shifted in recent years from a primarily
  target-based approach to one that uses phenotypic high-throughput screens.

- It is clear that we still have much to learn about the drug targets and the
  complex effects of the drugs on Mycobacterium tuberculosis.

- We propose a more integrated approach that learns from earlier drug discovery
  efforts that could help to move drug discovery forward.

<a id="org529de8a"></a>

## Conclusion

<a id="org0c488b6"></a>

# [-] Ngo2019 - Genomic prediction of tuberculosis drug- resistance: benchmarking existing databases and prediction algorithms

<a id="org95c9a78"></a>

## Notes

- This paper seems important but I&rsquo;m unable to see how to use this in the dissertation.

<a id="org7d5bbf2"></a>

## FileName

- Ngo2019<sub>GenomicPrediction.pdf</sub>

<a id="org09f64ef"></a>

## Abstract

- It is possible to predict whether a tuberculosis (TB) patient will fail to
  respond to specific antibiotics by sequencing the genome of the infecting
  Mycobacterium tuberculosis (Mtb) and observing whether the pathogen carries
  specific mutations at drug-resistance sites.

- Here, we evaluate the performance of four popular tools (TBProfiler, MyKrobe,
  KvarQ, PhyResSE) with 6746 isolates compiled from publicly available
  databases, and subsequently identify highly probable phenotyping errors in the
  databases by genetically predicting the drug phenotypes using all four
  software.

- Our results show that these bioinformatics tools generally perform well in
  predicting the resistance status for two key first-line agents (isoniazid,
  rifampicin), but the accuracy is lower for second-line injectables and
  fluoroquinolones.

<a id="org314fb32"></a>

## Conclusion

- The use of whole-genome sequencing to replace conventional laboratory-based
  drug-susceptibility testing reduces the time taken to culture and test against
  individual drugs, but requires sophisticated bioinformatics algorithms to be
  designed and validated to translate the genetic information into drug
  susceptibility phenotypes.

- Using latent class modeling, we have identi- fied which isolates in the
  existing databases are likely to present erroneous DST phenotypes and
  recalibrated the performance of these bioinformatics software adjusting for
  errors in the existing databases.

- However, we still observe low sensitivity in detecting resistance to ETO,
  PAS, PTO, and PZA.

- Without a clear understanding to the genetic biomarkers of drug resistance or
  to their pene- trance, or at least more powerful algorithms that can pick up
  additional intricacies of resistance across multiple drugs, such as what have
  been done for HIV and cancer [22–24], the ability to successfully predict the
  presence of drug-resistance is reduced.

<a id="orge4f19d6"></a>

# [-] Periwal2011 - Predictive models for anti-tubercular molecules using machine learning on high-throughput biological screening datasets

<a id="org2129bd0"></a>

## Notes

- Could build upon this paper by using XgBoost and other classification
  algorithms
- They&rsquo;ve used Weka
- Data is available
- Code is not available
- They&rsquo;ve also relied on some closed-source software

<a id="orgb5f9640"></a>

## FileName

- Periwal2011<sub>PredictiveModels.pdf</sub>

<a id="org5d0eae6"></a>

## Abstract

- Recent reports suggest that Mtb has been developing resistance to the widely
  used anti-tubercular drugs resulting in the emergence and spread of multi
  drug-resistant (MDR) and extensively drug-resistant (XDR) strains throughout
  the world.

- Whole organism screens on the other hand are target-agnostic and have been now
  widely employed as an alternative for lead identification but they are limited
  by the time and cost involved in running the screens for large compound
  libraries.

<a id="org7e10bcc"></a>

## Conclusion

- We have employed a systematic and comprehensive approach to build supervised
  classification based predictive models for anti-tubercular agents from
  publicly available bioassay datasets for in-vitro screens for Mtb inhibitors.

- We have generated classification models for anti-tubercular agents with four
  commonly used state-of-the-art classifiers i.e. Naïve Bayes, Random Forest,
  J48 and SMO

<a id="orgb645b0a"></a>

# [ ] Ratnatunga2020 - The Rise of Non-Tuberculosis Mycobacterial Lung Disease

<a id="org9726f1b"></a>

## Notes

- Survey paper for NTM
- Can be used for background context

<a id="orgb172b57"></a>

## FileName

- Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub>

<a id="org0239e86"></a>

## Abstract

- These lesser known “cousins” of Mycobacterium tuberculosis (TB) were once
  thought to be harmless environmental saprophytics and only dangerous to
  individuals with defective lung structure or the immunosuppressed.

- NTM have become extremely antibiotic resistant, and now have been found to be
  internationally dispersed through person-to-person contact.

<a id="org806a326"></a>

## Conclusion

<a id="orga1797fe"></a>

# WAIT Romero2020 - Decision tree machine learning applied to bovine tuberculosis risk factors to aid disease control decision making

<a id="org5ec8d01"></a>

## Notes

- Can use other algorithms for improving the accuracy i.e. XgBoost

- Uses Decision Tree Classifier to segregate the areas into low, mid, high risk

- Code is not provided

- Ask LJMU staff for help in this

<a id="org5b5f4f8"></a>

## FileName

- Romero2020<sub>DecisionTree.pdf</sub>

<a id="org6b44ddf"></a>

## Abstract

- Identifying and understanding the risk factors for endemic bovine tuberculosis
  (TB) in cattle herds is critical for the control of this disease. Exploratory
  machine learning techniques can uncover complex non-linear relation- ships and
  interactions within disease causation webs, and enhance our knowledge of TB
  risk factors and how they are interrelated.

- The best classification tree models informed multivariable binomial logistic
  regression models in each area, adding statistical inference outputs. These
  two approaches showed similar predictive performance although there were some
  disparities regarding what constituted high-risk predictors.

- Decision tree machine learning approaches can identify risk factors from webs
  of causation: information which may then be used to inform decision making for
  disease control purposes.

<a id="orge69eb5f"></a>

## Conclusion

<a id="org8d0a561"></a>

## Supplementary

<a id="org0304e06"></a>

### Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub>

<a id="org835a97c"></a>

# STRT Sauer2018 - Feature selection and prediction of treatment failure in tuberculosis

<a id="orgfb0e836"></a>

## Notes

- Code is available
  <https://github.com/biosharp-dotnet/niaid_TB>

- The data is sourced from dedicated NIH website
  <https://tbportals.niaid.nih.gov/>

- Can use XgBoost and/or H20

- Can add the data from other sources as well

<a id="orgd2199bb"></a>

## FileName

- Sauer2018<sub>FeatureSelection.pdf</sub>

<a id="org5f929b4"></a>

## Abstract

- On a multi-country dataset managed by the National Institute of Allergy and
  Infectious Diseases we applied various machine learning techniques to
  identify factors statistically associated with treatment failure and to
  predict treatment failure based on baseline demographic and clinical
  characteristics alone.

<a id="org91b1108"></a>

## Conclusion

<a id="org875c2bd"></a>

# [-] Schiff2018 - Clinical relevance of non-tuberculous mycobacteria isolated from respiratory specimens: seven year experience in a UK hospital

<a id="org54256e4"></a>

## Notes

- Survey report

- Can be used for background context

<a id="org513da23"></a>

## FileName

- Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub>

<a id="orgd4e3a99"></a>

## Abstract

<a id="orgb96f233"></a>

## Conclusion

<a id="org1105648"></a>

# Genome-wide Analysis of MDR and XDR Tuberculosis from Belarus: Machine-learning Approach

<a id="org156a5fc"></a>

## FileName

- Sergeev2016<sub>GenomeWide.pdf</sub>

<a id="org3a35904"></a>

## Notes

<a id="org021d03f"></a>

## Abstract

<a id="org53797e6"></a>

## Conclusion

<a id="org34e0186"></a>

# Vizcaino2010<sub>ComputatationalPrediction.pdf</sub>

<a id="org976bd3a"></a>

## FileName

<a id="orgdb7d2df"></a>

## Notes

<a id="org1d3732a"></a>

## Abstract

<a id="org75e56d9"></a>

## Conclusion

<a id="org3800a5a"></a>

# Waman2019<sub>MycobacterialGenomics.pdf</sub>

<a id="orgaf5cc41"></a>

## FileName

<a id="org690861b"></a>

## Notes

<a id="org5d80423"></a>

## Abstract

<a id="orgca97a98"></a>

## Conclusion

<a id="orgacde91a"></a>

# Wang2020<sub>ADeepLearningBasedMethod.pdf</sub>

<a id="orgbd17c4b"></a>

## FileName

<a id="org2356d81"></a>

## Notes

<a id="org95ca635"></a>

## Abstract

<a id="orgb22e05f"></a>

## Conclusion

<a id="orgaab12b9"></a>

# Weiner2017<sub>HighThroughput.pdf</sub>

<a id="orgb69ddb6"></a>

## FileName

<a id="org776e57b"></a>

## Notes

<a id="org042cad3"></a>

## Abstract

<a id="orgb1ff1b2"></a>

## Conclusion

<a id="orgc69e8af"></a>

# Wilder2019<sub>ImprovingTuberculosis.pdf</sub>

<a id="org5b5f87b"></a>

## FileName

<a id="orgdc1aef2"></a>

## Notes

<a id="orga35152d"></a>

## Abstract

<a id="org944e6c3"></a>

## Conclusion

<a id="org43f8ceb"></a>

# Yang2017<sub>MachineLearning.pdf</sub>

<a id="orgb79a2a0"></a>

## FileName

<a id="org9be499f"></a>

## Notes

<a id="org869a971"></a>

## Abstract

<a id="org6d2656f"></a>

## Conclusion

<a id="orgf9717df"></a>

# Yang2019<sub>DeepAMRForPredicting.pdf</sub>

<a id="org76bb257"></a>

## FileName

<a id="orgb7cd4a0"></a>

## Notes

<a id="orgf0be83d"></a>

## Abstract

<a id="orgf5e4664"></a>

## Conclusion

<a id="org2fabacc"></a>

# Zhang2020<sub>AnIntegratedMLApproach.pdf</sub>

<a id="org2a21944"></a>

## FileName

<a id="orgf097571"></a>

## Notes

<a id="orgbb9c5bd"></a>

## Abstract

<a id="orgfe550c7"></a>

## Conclusion
