# Table of Contents

1.  [Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.](#org7142be8)
2.  [Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.](#orgc9c3123)
3.  [Machine learning and tubercular drug target recognition.](#orga038885)
4.  [Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem](#org8ef21c1)
    1.  [Notes](#orgf805580)
    2.  [FileName](#orge7c4f33)
    3.  [Abstract](#org45c74f3)
    4.  [FutureWork](#org9f58e7c)
5.  [Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm](#orgc19c339)
    1.  [Notes](#org2a1e03e)
    2.  [FileName](#orgbc38e9c)
    3.  [Abstract](#org41a0187)
    4.  [Conclusion](#orgb98a91d)
6.  [Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper](#org10a4982)
    1.  [Notes](#orgbbe248e)
    2.  [FileName](#orgb19e6fc)
    3.  [Abstract](#orgf92ff0d)
    4.  [Conclusion](#orgd5f9eea)
7.  [Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities](#org3a27f28)
    1.  [Notes](#org7359d57)
    2.  [FileName](#org75f66b8)
    3.  [Abstract](#orga23a500)
    4.  [Conclusion](#org10812e5)
8.  [Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data](#orge450ef2)
    1.  [Notes](#org5223dfb)
    2.  [FileName](#org6a58e3a)
    3.  [Abstract](#org7252290)
    4.  [Conclusion](#orgd9dd880)
9.  [Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction](#org9692920)
    1.  [Notes](#org77eabd9)
    2.  [FileName](#org46e661a)
    3.  [Abstract](#org5af63fd)
    4.  [Conclusion](#org499a9c8)
10. [Ching2018 - Opportunities and obstacles for deep learning in biology and medicine](#orgb9d0526)
    1.  [Notes](#orgf30bd1d)
    2.  [FileName](#org7735f5a)
    3.  [Abstract](#org57eb8ba)
    4.  [Conclusion](#orgd171acf)
11. [Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data](#org6b9697d)
    1.  [Notes](#orgf74d435)
    2.  [FileName](#orgc91f057)
    3.  [Abstract](#org3b86480)
    4.  [Conclusion](#org4afd046)
    5.  [Supplementary](#orgba91abd)
        1.  [Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub>](#org6f30c05)
12. [Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis](#org9022849)
    1.  [Notes](#orga25081b)
    2.  [FileName](#orgccf6d75)
    3.  [Abstract](#org777718d)
    4.  [Conclusion](#org418a729)
13. [Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports](#orga72ef3e)
    1.  [Notes](#orgc6f7de8)
    2.  [FileName](#orgc5d9274)
    3.  [Abstract](#org7b72d17)
    4.  [Conclusion](#orgfeeac59)
14. [Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence](#org0d9e9c4)
    1.  [Notes](#org26e4d36)
    2.  [FileName](#orgda81a3b)
    3.  [Abstract](#orgfcffdbc)
    4.  [Conclusion](#orgd205e05)
15. [Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models](#org6d3cf2d)
    1.  [Notes](#org2335961)
    2.  [FileName](#org5f7c707)
    3.  [Abstract](#org1ad2176)
    4.  [Conclusion](#org181ab64)
16. [Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation](#org400a610)
    1.  [Notes](#org3fbe871)
    2.  [FileName](#org64f202a)
    3.  [Abstract](#org6041101)
    4.  [Conclusion](#org0997272)
17. [Ekins2014<sub>AreBiggerDataSets.pdf</sub>](#org4c4a8ba)
    1.  [Notes](#org54a0863)
    2.  [Title](#orgdacc97a)
    3.  [Abstract](#org533bc3a)
    4.  [Conclusion](#orgdb9307d)
18. [Ekins2014<sub>CombiningComputationalMethods.pdf</sub>](#org65cfc8b)
    1.  [Notes](#org1de6938)
    2.  [Title](#orgb2a874e)
    3.  [Abstract](#orgc6c058a)
    4.  [Conclusion](#orgd08a994)
19. [Ekins2016<sub>MachineLearningModel.pdf</sub>](#orgee42dc9)
    1.  [Notes](#org42118f4)
    2.  [Title](#orgd9e4b95)
    3.  [Abstract](#orgabc4adb)
    4.  [Conclusion](#org2dc4f93)
20. [Ekins2017<sub>MachineLearning.pdf</sub>](#orgca84a31)
    1.  [Notes](#org7028615)
    2.  [Title](#orgdaa4ca2)
    3.  [Abstract](#orgb003076)
    4.  [Conclusion](#org50f5cb2)
21. [Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis](#orgb32f093)
    1.  [Notes](#org5ae7a5d)
    2.  [FileName](#org2496da5)
    3.  [Abstract](#orge90371c)
    4.  [Conclusion](#org8e241c4)
22. [Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay](#org5cc3de3)
    1.  [Notes](#org2304bf0)
    2.  [FileName](#orgfb45142)
    3.  [Abstract](#org37f87fc)
    4.  [Conclusion](#org719cbd6)
23. [Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data](#org3a3bde9)
    1.  [Notes](#orgfe26e9a)
    2.  [FileName](#orga3ae48a)
    3.  [Abstract](#orga3cd090)
    4.  [Conclusion](#org07c830e)
24. [Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models](#org58395d0)
    1.  [Notes](#orgf92c28f)
    2.  [FileName](#org9ed527c)
    3.  [Abstract](#org3253e1c)
    4.  [Conclusion](#org5c4bec9)
25. [Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis](#org80be3e5)
    1.  [Notes](#orge6d0e1a)
    2.  [FileName](#org987fb1b)
    3.  [Abstract](#org2185daf)
    4.  [Conclusion](#orga69d642)
26. [Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance](#orgb876439)
    1.  [Notes](#org3ccd408)
    2.  [FileName](#org231d13a)
    3.  [Abstract](#org05a71f6)
    4.  [Conclusion](#org9613f01)
27. [Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes](#orgc999872)
    1.  [Notes](#orgff95528)
    2.  [FileName](#orge963c73)
    3.  [Abstract](#org037443a)
    4.  [Conclusion](#orga8ef46e)
28. [Killian2019<sub>LearningToPrescribe.pdf</sub>](#org2612283)
    1.  [Title](#org28df13d)
    2.  [Abstract](#orge927dd2)
    3.  [Conclusion](#org0cbe5e3)
29. [Kouchaki2019<sub>ApplicationOfML.pdf</sub>](#org7feaca2)
    1.  [Title](#org45de957)
    2.  [Abstract](#orgb64d483)
    3.  [Conclusion](#orgd5784e2)
30. [Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub>](#org5cfaf30)
    1.  [Title](#org86e5e63)
    2.  [Abstract](#orgb5d6158)
    3.  [Conclusion](#org69e1208)
    4.  [Supplementary](#org93c47bf)
        1.  [Kouchaki2020<sub>MultiLabel.zip</sub>](#orgbc3d479)
        2.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>](#orga97997e)
        3.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>](#org7c7303b)
        4.  [Code](#org3275bb7)
31. [Kovalishyn2019<sub>HybridDesign.pdf</sub>](#orgbbc68e3)
    1.  [Title](#orgefb4396)
    2.  [Abstract](#org6384cea)
    3.  [Conclusion](#org435f477)
32. [Kumari2019<sub>AGeneticProgramming.pdf</sub>](#org8e89d4d)
    1.  [Title](#orgc0efb08)
    2.  [Abstract](#orgd354ab6)
    3.  [Conclusion](#org72fd149)
33. [Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub>](#orgf42bd6e)
    1.  [Abstract](#org40fcf91)
    2.  [Conclusion](#org9edc5d3)
34. [Lane2018<sub>ComparingAndValidating.pdf</sub>](#org9697582)
    1.  [Abstract](#org1a0bd32)
    2.  [Conclusion](#org1c2e34f)
35. [Lin2001<sub>ConservedCodonComposition.pdf</sub>](#org16501c0)
    1.  [Abstract](#orgf639764)
    2.  [Conclusion](#org158ddfb)
36. [LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub>](#orgf9f3c60)
    1.  [Abstract](#orgf2e11b1)
    2.  [Conclusion](#org97c867a)
37. [Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub>](#org67ba775)
    1.  [Abstract](#org09fc577)
    2.  [Conclusion](#org3df29a5)
38. [Marino2016<sub>ComputationalAndEmpirical.pdf</sub>](#org09f78eb)
    1.  [Abstract](#org821ce7d)
    2.  [Conclusion](#org886e02d)
39. [Marques2019<sub>PulmonaryNTMInfections.pdf</sub>](#org355ab96)
    1.  [Abstract](#org11646a4)
    2.  [Conclusion](#orgf44ca00)
40. [Meldenez2015<sub>OnCombiningMultiple.pdf</sub>](#orgc6146c9)
    1.  [Abstract](#orgc81f993)
    2.  [Conclusion](#org85434fc)
41. [Mikusova2016<sub>LearningFromThePast.pdf</sub>](#orgdd1f848)
    1.  [Abstract](#org4799880)
    2.  [Conclusion](#orgebfb6a2)
42. [Ngo2019<sub>GenomicPrediction.pdf</sub>](#orgbdd1328)
    1.  [Abstract](#org1b1ed3f)
    2.  [Conclusion](#org0b4d47d)
43. [Periwal2011<sub>PredictiveModels.pdf</sub>](#org3afeda7)
    1.  [Abstract](#org6f0cd02)
    2.  [Conclusion](#orgbbde880)
44. [Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub>](#orgcc90d7f)
    1.  [Abstract](#org60e602f)
    2.  [Conclusion](#orge2123e7)
45. [Romero2020<sub>DecisionTree.pdf</sub>](#orga41eb5f)
    1.  [Abstract](#org4ebc7f7)
    2.  [Conclusion](#org71948bf)
    3.  [Supplementary](#org52f981e)
        1.  [Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub>](#org6955fa0)
46. [Sauer2018<sub>FeatureSelection.pdf</sub>](#org1da15d7)
    1.  [Abstract](#org211ae2c)
    2.  [Conclusion](#orgdafd10e)
47. [Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub>](#org87b1ac9)
    1.  [Abstract](#org51d58e2)
    2.  [Conclusion](#org4897763)
48. [Sergeev2016<sub>GenomeWide.pdf</sub>](#orge866f21)
    1.  [Abstract](#orge6e4341)
    2.  [Conclusion](#orgdc28c1c)
49. [Vizcaino2010<sub>ComputatationalPrediction.pdf</sub>](#org2991928)
    1.  [Abstract](#orgb964b31)
    2.  [Conclusion](#org8da6a79)
50. [Waman2019<sub>MycobacterialGenomics.pdf</sub>](#org689ca3f)
    1.  [Abstract](#orgbbb2826)
    2.  [Conclusion](#org2fbb7cc)
51. [Wang2020<sub>ADeepLearningBasedMethod.pdf</sub>](#orgb08ddc6)
    1.  [Abstract](#org2dbb5a1)
    2.  [Conclusion](#org1fda881)
52. [Weiner2017<sub>HighThroughput.pdf</sub>](#org732d905)
    1.  [Abstract](#orgd3743ef)
    2.  [Conclusion](#org6464031)
53. [Wilder2019<sub>ImprovingTuberculosis.pdf</sub>](#orgde02da3)
    1.  [Abstract](#orgb627175)
    2.  [Conclusion](#org93dfb7d)
54. [Yang2017<sub>MachineLearning.pdf</sub>](#org024a837)
    1.  [Abstract](#org1462cc6)
    2.  [Conclusion](#org1d0726a)
55. [Yang2019<sub>DeepAMRForPredicting.pdf</sub>](#orgcf200fb)
    1.  [Abstract](#org118b0f2)
    2.  [Conclusion](#org67d1fb7)
56. [Zhang2020<sub>AnIntegratedMLApproach.pdf</sub>](#org4f55a9d)
    1.  [Abstract](#org4bac13f)
    2.  [Conclusion](#org5ad1a7b)

<a id="org7142be8"></a>

# TODO Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.

<https://www.ncbi.nlm.nih.gov/pubmed/30084331>

<a id="orgc9c3123"></a>

# TODO Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.

<https://www.ncbi.nlm.nih.gov/pubmed/30973110>

<a id="orga038885"></a>

# TODO Machine learning and tubercular drug target recognition.

<https://www.ncbi.nlm.nih.gov/pubmed/24245763>

<a id="org8ef21c1"></a>

# WAIT Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem

<a id="orgf805580"></a>

## Notes

- Survey paper
- Could be used build up the background for the need for research in NTM

<a id="orge7c4f33"></a>

## FileName

- Ahmed2020<sub>NonTuberculosisMycoANeglectedAndEmergingProblem.pdf</sub>

<a id="org45c74f3"></a>

## Abstract

- The incidence of NTM infections is rising owing to a growing population of
  immunocompromised and vulnerable individuals, complex medical and surgical
  procedures, as well as increased awareness and diagnostic capabilities.

<a id="org9f58e7c"></a>

## FutureWork

- Simpler and more rapid tests would be ideal in making the diagnosis of NTM
  disease more quickly. However, the development of such tests is complicated
  by the large number of NTM species that can cause disease and by the criteria
  for disease, which require more than just the presence of the microorganism.
  As a result, NTM disease will require multiple diagnostic parameters, leading
  to delays and higher costs. More specific biomarkers to monitor the progress
  of treatment and test cure are lacking.

<a id="orgc19c339"></a>

# STRT Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm

<a id="org2a1e03e"></a>

## Notes

- Can adopt a similar approach towards the classification of NTB lineages
- Can alter the algorithm to XGBoost to Neural Networks to improve accuracy

<a id="orgbc38e9c"></a>

## FileName

- Aze2015<sub>GenomicsAndMachine.pdf</sub>

<a id="org41a0187"></a>

## Abstract

- Mycobacterium tuberculosis complex taxonomy has rapidly evolved in the last 20
  years through intensive clinical isolation, advances in sequencing and in the
  description of fast-evolving loci (CRISPR and MIRU-VNTR)

- The objectives of this study were 1) to explicit the consensus that exists
  between the alternative taxonomies, and 2) to provide an on-line tool to ease
  classification of new isolates. Genotyping (24-VNTR, 43-spacers spoli-
  gotypes, IS6110-RFLP) was undertaken for 3,454 clinical isolates from the
  Netherlands (2004-2008). The resulting database was enlarged with African
  isolates to include most human tuberculosis diversity. Assignations were
  obtained using TB-Lineage, MIRU- VNTRPlus, SITVITWEB and an algorithm from
  Borile et al

- Original and consensus assignations of the all isolates from the database were
  subsequently implemented into an ensemble learning approach based on Machine
  Learning tool Weka to derive a classification scheme.

- When applied to independent datasets, it was able to suggest new sublineages
  such as pseudo-Beijing. This Lineage Prediction tool, efficient on 15-MIRU,
  24-VNTR and spoligotype data is available on the web interface “TBminer.”

- We successfully used Machine Learning on a large dataset to set up and make
  available the first consensual taxonomy for human Mycobacterium tuberculosis
  complex

<a id="orgb98a91d"></a>

## Conclusion

- We developed an approach making use of standard typing data for human M.

tuberculosis isolates to infer a consensual taxonomy concordant with most
up-to-date data on Whole Genome Sequencing diversity.

- We hope that the same approach can benefit other human pathogens having
  alternative taxonomies according to Serotypes, CRISPRtypes, MLST-types such as
  Salmo- nella, Listeria, Brucella, and more broadly to other organisms such as
  bacterial plant- pathogens.

<a id="org10a4982"></a>

# [-] Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper

<a id="orgbbe248e"></a>

## Notes

- Not directly related to (N)TB
- Survey paper
- Analyzes various ML techniques
- Mentions various drug-target-interaction databases

<a id="orgb19e6fc"></a>

## FileName

- Bagherian2019<sub>MachineLearning.pdf</sub>

<a id="orgf92ff0d"></a>

## Abstract

- There is a need to develop novel and efficient prediction approaches in order
  to avoid costly and laborious yet not-always-deterministic experiments to
  determine drug–target interactions (DTIs) by experiments alone
- In this article, we describe the data required for the task of DTI prediction
  followed by a comprehensive catalog consisting of machine learning methods and
  databases, which have been proposed and utilized to predict DTIs
- The advantages and disadvantages of each set of methods are also briefly
  discussed

<a id="orgd5f9eea"></a>

## Conclusion

- Almost all the methods used in DTI prediction, particularly similarity-based
  methods, heavily rely on assertions concerning similar drugs and similar
  targets, the type of database used for the prediction plays a significant
  role.
- Future predictions should rely on more comprehensive inter- nal databases,
  which would require a significant effort to map and curate data across the
  sources that utilize different ways to define, name and identify the drugs and
  targets.

<a id="org3a27f28"></a>

# STRT Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities

<a id="org7359d57"></a>

## Notes

- Image processing algorithms, CNN
- They proposed an online database of X-ray images - by now this might have been developed

<a id="org75f66b8"></a>

## FileName

- Cao2016<sub>ImprovingDiagnosticsUsingDL.pdf</sub>

<a id="orga23a500"></a>

## Abstract

- The combination of machine learning and mobile computing techniques offers a
  unique opportunity to accelerate the TB diagnosis among these communities

- We first introduce major technique barriers and proposed system architecture

- The first activity aims to develop large-scale, real-world and well-annotated
  X-ray image database dedicated for automated TB screening.

- The second research activity focus on developing effective and efficient
  computational models (in particularly, deep convolutional neural networks
  (CNN)-based models) to classify the image into different category of TB
  manifestations

- Our future work includes: (1) to further improve the performance of the
  algorithms; and (2) to deploy our system in the city of Carabayllo in Perú, a
  densely occupied urban community and high-burden TB

<a id="org10812e5"></a>

## Conclusion

- The first progress is to build a large-scale, real-world, and well-annotated chest X- ray image
  database dedicated for TB screening.

- The second progress is to develop effective and efficient computational model for TB manifestation
  categorization.

- We will continue to develop the large scale, real-world X-ray TB database, as
  well as continuing to improve the performance of the computing algorithms

<a id="orge450ef2"></a>

# STRT Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data

<a id="org5223dfb"></a>

## Notes

- A good in-depth ML based paper
- They&rsquo;ve shared high-level implementation details briefly
- Hyperparameters are also in the supplementary material
- I was able to find the code on Github
  <https://github.com/biosharp-dotnet/wdnn>
- Could apply the same techniques to NTM
- Could change the algorithm and apply on the same data
- The author built upon the previous paper by improving the accuracy of the ML model - we could do the same with NTM

<a id="org6a58e3a"></a>

## FileName

- Chen2018<sub>DeepLearningPredictsTuberculosis.pdf</sub>

<a id="org7252290"></a>

## Abstract

- Using targeted or whole genome sequencing and conventional drug resistance
  phenotyping data from 3,601 Mycobacterium tuberculosis strains, 1,228 of which
  were multidrug resistant, we implemented the first multitask deep learning
  framework to predict phenotypic drug resistance to 10 anti-tubercular drugs

- The proposed wide and deep neural network (WDNN) achieved improved predictive
  performance compared to regularized logistic regression and random forest

<a id="orgd9dd880"></a>

## Conclusion

- We demonstrate that L2 regression and MD-WDNN trained on a larger diverse
  dataset using a method of aggregating rare variants outperforms our previously
  reported random forest model. (Chen, 2018)

- A few prior studies have utilized algorithmic or machine learning methods
  using genomic data to account for the complex relationship between genotype
  and drug resistance in MTB

<a id="org9692920"></a>

# STRT Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction

<a id="org77eabd9"></a>

## Notes

- A good in-depth ML based paper
- Could apply the same techniques to NTM
- Could change the algorithm and apply on the same data

<a id="org46e661a"></a>

## FileName

- Chen2019<sub>BeyondMultiDrugResistance.pdf</sub>

<a id="org5af63fd"></a>

## Abstract

- Whole genome sequencing of clinical Mycobacterium tuberculosis isolates
  promises to circumvent the long wait times and limited scope of conventional
  phenotypic antimicrobial susceptibility, but gaps remain for predicting
  phenotype accurately from genotypic data especially for certain drugs.

- We collected targeted or whole genome sequencing and conventional drug
  resistance phenotyping data from 3601 Mycobacterium tuberculosis strains
  enriched for resistance to first- and second-line drugs, with 1228 multidrug
  resistant strains

- We investigated the utility of (1) rare variants and variants known to be
  determinants of resistance for at least one drug and (2) machine and
  statistical learning architectures in predicting phenotypic drug resistance to
  10 anti-tuberculosis drugs

<a id="org499a9c8"></a>

## Conclusion

- Our approach allowed us to gain new insights into the relative im- portance of
  specific genetic variants for resistance prediction

- In summary, we present an exploration of machine learning and traditional
  statistical models to identify the resistance of MTB isolates to 10
  anti-tuberculosis drugs from whole genome sequencing data.

- Our models trained on rare and common genetic variants achieved
  state-of-the-art performance on a large, aggregated TB dataset, with
  prediction times of less than a tenth of a second, demonstrating the efficacy
  of our model as a diagnostic tool for MTB drug resistance.

- The MD-WDNN represented the first multidrug model to our knowledge that

incorporated a high number of genotypic predictors known to be important to
determining resistance for one or more included drugs

<a id="orgb9d0526"></a>

# [-] Ching2018 - Opportunities and obstacles for deep learning in biology and medicine

<a id="orgf30bd1d"></a>

## Notes

- Found the code <https://github.com/greenelab/deep-review>
- Can use this paper to build up the background for using DL in MTB and NTM analysis

<a id="org7735f5a"></a>

## FileName

- Ching2018<sub>OpportunitiesAndObstaclesForDLInBiology.pdf</sub>

<a id="org57eb8ba"></a>

## Abstract

- Following from an extensive literature review, we find that deep learning has
  yet to revolutionize biomedicine or definitively resolve any of the most
  pressing challenges in the field, but promising advances have been made on the
  prior state of the art
- The limited amount of labelled data for training presents problems in some
  domains, as do legal and privacy constraints on work with sensitive health
  records.

<a id="orgd171acf"></a>

## Conclusion

- The most challenging tasks may be those in which predictions are used directly
  for downstream model- ling or decision-making, especially in the clinic. As an
  example, errors in sequence variant calling will be amplified if they are used
  directly for genome-wide association studies.

- In addition, the stochasticity and complexity of biological sys- tems imply
  that for some problems, for instance, predicting gene regulation in disease,
  perfect accuracy will be unattainable

<a id="org6b9697d"></a>

# STRT Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data

<a id="orgf74d435"></a>

## Notes

- Need to request code from authors
- Good ML oriented paper

<a id="orgc91f057"></a>

## FileName

- Deelder2019<sub>MLPredictsAccurately.pdf</sub>

<a id="org3b86480"></a>

## Abstract

- We applied machine learning approaches to 16,688 M. tuberculosis isolates that
  have undergone WGS and laboratory drug-susceptibility testing (DST) across 14
  antituberculosis drugs, with 22.5% of samples being multidrug resistant and
  2.1% being extensively drug resistant.

- We fitted separate models for each drug, with and without “co-occurrent
  resistance” markers known to be causing resistance to drugs other than the one
  of interest. Predictive performance was measured using sensitivity,
  specificity, and the area under the receiver operating characteristic curve,
  assuming DST results as the gold standard.

- Overall, the gradient-boosted-tree models performed better than the
  classification-tree models.

- Our work demonstrates the utility of machine learning as a flexible approach
  to drug resistance prediction that is able to accommodate a much larger number
  of predictors and to summarize their predictive ability, thus assisting
  clinical decision making and single nucleotide polymorphism detection in an
  era of increasing WGS data generation.

<a id="org4afd046"></a>

## Conclusion

- The predictive performance was highest for resistance to first-line drugs,
  and multidrug-resistant tuberculosis (area under the receiver operating characteristic curve above
  96%), and lowest for third- line drugs (area under the curve below 85%).

<a id="orgba91abd"></a>

## Supplementary

<a id="org6f30c05"></a>

### Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub>

<a id="org9022849"></a>

# [?] Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis

<a id="orga25081b"></a>

## Notes

- This requires more knowledge of chemistry and perhaps also relies on lab based experiments

<a id="orgccf6d75"></a>

## FileName

- Djout2016<sub>PredictiveModeling.pdf</sub>

<a id="org777718d"></a>

## Abstract

- ThyX is an essential thymidylate synthase

that is mechanistically and structurally unrelated to the human enzyme. DNA
gyrase is a DNA topoisomerase present in bacteria and plants but not animals.
The current study set out to understand the structure-activity relationships of
these targets in Mtb using a combination of cheminformatics and in vitro
screening.

<a id="org418a729"></a>

## Conclusion

<a id="orga72ef3e"></a>

# STRT Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports

<a id="orgc6f7de8"></a>

## Notes

- This is a survey report about NTM across a couple decades
- Could be used in building up the background context

<a id="orgc5d9274"></a>

## FileName

- Donohue2018<sub>IncreasingNTMReportingRates.pdf</sub>

<a id="org7b72d17"></a>

## Abstract

- Changes in NTM diversity were observed in complex/groups known to be
  clinically significant. Between 1994 and 2014 the rate implicating M.
  abscesses-chelonae group and M. avium complex increased by 322 and 149%,
  respectively.

<a id="orgfeeac59"></a>

## Conclusion

- Knowledge of the NTM species most frequently associ- ated with adverse human
  health consequences will assist epidemiological investigators to identify
  likely sources and modes of exposure associated with these types of in-
  fections.
- As the U.S. population ages, the public health burden from NTM associated
  disorders is expected to increase.

<a id="org0d9e9c4"></a>

# [-] Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence

<a id="org26e4d36"></a>

## Notes

- Survey paper
- Can be used for building background context

<a id="orgda81a3b"></a>

## FileName

- Doshi2017<sub>TuberculosisControl.pdf</sub>

<a id="orgfcffdbc"></a>

## Abstract

<a id="orgd205e05"></a>

## Conclusion

<a id="org6d3cf2d"></a>

# [X] Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models

<a id="org2335961"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org5f7c707"></a>

## FileName

- Ekins2013<sub>EnhancingHitIdentification.pdf</sub>

<a id="org1ad2176"></a>

## Abstract

- High-throughput screening (HTS) in whole cells is widely pursued to find
  compounds active against Mycobacterium tuberculosis (Mtb) for further
  development towards new tuberculosis (TB) drugs
- We have recently built and validated Bayesian machine learning models for
  predicting compounds with activity against Mtb based on publicly available
  large-scale HTS data from the Tuberculosis Antimicrobial Acquisition
  Coordinating Facility

<a id="org181ab64"></a>

## Conclusion

<a id="org400a610"></a>

# [X] Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation

<a id="org3fbe871"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="org64f202a"></a>

## FileName

- Ekins2013<sub>FusingDualEvent.pdf</sub>

<a id="org6041101"></a>

## Abstract

- A cheminformatics data-fusion

approach followed by Bayesian machine learning, Support Vector Machine, or Recursive
Partitioning model development (based on publicly available Mtb screening data) was used to
compare individual data sets and subsequent combined models.

- We demonstrate that

combining three data sets incorporating antitubercular and cytotoxicity data in Vero cells
from our previous screens results in external validation receiver operator curve (ROC) of
0.83 (Bayesian or RP Forest)

<a id="org0997272"></a>

## Conclusion

<a id="org4c4a8ba"></a>

# [X] Ekins2014<sub>AreBiggerDataSets.pdf</sub>

<a id="org54a0863"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orgdacc97a"></a>

## Title

<a id="org533bc3a"></a>

## Abstract

<a id="orgdb9307d"></a>

## Conclusion

<a id="org65cfc8b"></a>

# [X] Ekins2014<sub>CombiningComputationalMethods.pdf</sub>

<a id="org1de6938"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orgb2a874e"></a>

## Title

<a id="orgc6c058a"></a>

## Abstract

<a id="orgd08a994"></a>

## Conclusion

<a id="orgee42dc9"></a>

# [X] Ekins2016<sub>MachineLearningModel.pdf</sub>

<a id="org42118f4"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orgd9e4b95"></a>

## Title

<a id="orgabc4adb"></a>

## Abstract

<a id="org2dc4f93"></a>

## Conclusion

<a id="orgca84a31"></a>

# [X] Ekins2017<sub>MachineLearning.pdf</sub>

<a id="org7028615"></a>

## Notes

- Requires too much domain knowledge
- Perhaps uses an older technique i.e. HTS for analysis
- Could use other algorithms on the data used in this study

<a id="orgdaa4ca2"></a>

## Title

<a id="orgb003076"></a>

## Abstract

<a id="org50f5cb2"></a>

## Conclusion

<a id="orgb32f093"></a>

# [-] Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis

<a id="org5ae7a5d"></a>

## Notes

- Image processing
- Mentiones the database source, NIH
- Has CSV as supplementary data

<a id="org2496da5"></a>

## FileName

- Engle2019<sub>PerformanceOfQureAI.pdf</sub>

<a id="orge90371c"></a>

## Abstract

- Availability of trained radiologists for fast processing of CXRs in regions
  burdened with tuberculosis always has been a challenge, affecting both timely
  diagnosis and patient monitoring.

- The Qure.ai qXR arti- ficial intelligence automated CXR interpretation tool,
  was blind-tested on the 346 radiologist- annotated CXRs from the TBPP
  database.

- We conclude that applying fully automated Qure.ai CXR analysis tool is useful
  for fast, accurate, uniform, large-scale CXR annotation assistance, as it
  performed well even for DR-TB cases that were not used for initial training.

<a id="org8e241c4"></a>

## Conclusion

- We demonstrated that the Qure.ai qXR CXR annotations for cavity, nodule,
  pleural effusion, and hilar lymphadenopathy, are in statistical agreement with
  radiologist CXR annotations.

- A corresponding analysis of patient case matched
  CT image annotations recorded within 30 days of a CXR supports this result,
  demonstrating statistical significance for cavity, pleural effusion, hilar
  lymphadenopathy, and atelectasis.

<a id="org5cc3de3"></a>

# [-] Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay

<a id="org2304bf0"></a>

## Notes

- Survey paper

<a id="orgfb45142"></a>

## FileName

- Greif2020<sub>FrequencyOfNTM.pdf</sub>

<a id="org37f87fc"></a>

## Abstract

- Nontuberculous mycobacteria (NTM) increasingly are recognized as
  opportunistic pathogens of humans.
- NTM species distribution is well documented in Europe and North America

<a id="org719cbd6"></a>

## Conclusion

- Our findings indicate that rate of NTM have risen nearly 5-fold (4.79) in
  Uruguay, from 0.33 cas- es/100,000 inhabitants in 2006 to 1.57 cases/100,000
  inhabitants in 2018.

<a id="org3a3bde9"></a>

# [-] Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data

<a id="orgfe26e9a"></a>

## Notes

- Can perform the mentioned analysis on the MTB and NTM data
- The paper focuses on the factors affecting the application and clinical application of ML models

<a id="orga3ae48a"></a>

## FileName

- Hicks2019<sub>EvaluationOfParameters.pdf</sub>

<a id="orga3cd090"></a>

## Abstract

- There has been no systematic evaluation of factors that may influence
  performance of ML models, how they might apply to and vary across clinical
  populations, and what the implications might be in the clinical setting

- We performed a meta-analysis of seven large Neisseria gonorrhoeae datasets, as
  well as Klebsiella pneumoniae and Acinetobacter baumannii datasets, with whole
  genome sequence data and antibiotic susceptibility phenotypes using set
  covering machine classifi- cation, random forest classification, and random
  forest regression models to predict resis- tance phenotypes from genotype.

- We demonstrate how model performance varies by drug, dataset, resistance
  metric, and species, reflecting the complexities of generating clinically
  relevant conclusions from machine learning-derived models.

- Our results highlight significant biological and technical challenges
  facing the application of machine learning-based prediction of antibiotic
  resistance as a diagnostic tool.

<a id="org07c830e"></a>

## Conclusion

<a id="org58395d0"></a>

# STRT Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models

<a id="orgf92c28f"></a>

## Notes

- Can use a different algorithm (XgBoost) on the dataset
- Sample data and complete code is shared
  <https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics>
- Need to find the full dataset of 4000 surveyed patients

<a id="org9ed527c"></a>

## FileName

- Hussain2018<sub>PredictingTreatmentOutcomes.pdf</sub>

<a id="org3253e1c"></a>

## Abstract

- Tuberculosis (TB) is a deadly contagious disease and a serious global health
  problem. It is curable but due to its lengthy treatment process, a patient is
  likely to leave the treatment incomplete, leading to a more lethal, drug
  resistant form of disease.

- We aim to facilitate TB programs by predicting the outcome of the treatment of
  a particular patient at the start of treatment so that their health workers
  can be utilized in a targeted and cost-effective way.

- The problem was modeled as a classification problem, and the outcome of
  treatment was predicted using state-of-art implementations of 3 machine learning algorithms

<a id="org5c4bec9"></a>

## Conclusion

- Empirical results using data of more than 4000 patients reveal that we can
  predict with more than 76% accuracy whether a TB patient would complete his (or her) treatment or not
-

<a id="org80be3e5"></a>

# STRT Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis

<a id="orge6d0e1a"></a>

## Notes

- Used many ML algorithms, 85% accuracy
- Could use XgBoost
- Could conduct a similar study for NTM
- Mentions many online bio databases

<a id="org987fb1b"></a>

## FileName

- Jamal2020<sub>AIAndMLBasedPredictive.pdf</sub>

<a id="org2185daf"></a>

## Abstract

- The present work is a computational framework that uses artificial
  intelligence (AI) based machine learning (ML) approaches for predicting
  resistance in the genes rpoB, inhA, katG, pncA, gyrA and gyrB for the drugs
  rifampicin, isoniazid, pyrazinamide and fluoroquinolones

- The single nucleotide variations were represented by several sequence and
  structural features that indicate the influence of mutations on the target
  protein coded by each gene.

<a id="orga69d642"></a>

## Conclusion

- The present study proposes an AI/ML method to classify resistant and susceptible
  mutations in TB and predict novel resistance conferring mutations.

- The impact of the reported mutations was captured in the form of changes in the amino acid
  residues, and the consequent change in properties vis-a-vis wild-type and mutant
  proteins and represented as features used to train the models.

- The classification model was generated for each gene and predictions were made for
  SNVs linked with each gene for each drug

<a id="orgb876439"></a>

# STRT Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance

<a id="org3ccd408"></a>

## Notes

- They developed and online platform

- Can be applied to NTM

- Code is available
  <https://github.com/biosharp-dotnet/microbial_AMR_ML>

<a id="org231d13a"></a>

## FileName

- Kavvas2018<sub>MachineLearning</sub><sub>.pdf</sub>

<a id="org05a71f6"></a>

## Abstract

- The many publicly available datasets describing its AMR characteristics demand
  disparate data-type analyses.

- Here, we develop a reference strain-agnostic computational platform that uses
  machine learning approaches, complemented by both genetic interaction analysis
  and 3D structural mutation-mapping, to identify signatures of AMR evolution to
  13 antibiotics.

- First, a pan-genome analysis shows that M. tuberculosis is highly conserved
  with sequenced variation concentrated in PE/PPE/PGRS genes.

- Second, the platform corroborates 33 genes known to confer resistance and identifies 24
  new genetic signatures of AMR.

- Third, 97 epistatic interactions across 10 resistance classes are revealed.

- Fourth, detailed structural analysis of these
  genes yields mechanistic bases for their selection.

- The platform can be used to study other human pathogens

<a id="org9613f01"></a>

## Conclusion

- The platform was applied to 1595 sequenced strains to yield results in four
  categories: pan-genome properties, identification of genes conferring
  antibiotic resistance, their epistatic interactions, and protein structure
  based mechanistic insights.

<a id="orgc999872"></a>

# Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes

<a id="orgff95528"></a>

## Notes

-

<a id="orge963c73"></a>

## FileName

- Khanna2018<sub>EnsembleTechnique.pdf</sub>

<a id="org037443a"></a>

## Abstract

- Development of an effective machine-learning model for T-cell Mycobacterium
  tuberculosis (M. tuberculosis) epitopes is beneficial for saving biologist’s
  time and effort for identifying epitope in a targeted antigen.

- The proposed method is able to work with variable length epitopes having size

even greater than 9-mers. Identification of T-cell or B-cell epitopes in the
targeted antigen is the main goal in designing epitope-based vaccine,
immune-diagnostic tests and antibody production

<a id="orga8ef46e"></a>

## Conclusion

<a id="org2612283"></a>

# Killian2019<sub>LearningToPrescribe.pdf</sub>

<a id="org28df13d"></a>

## Title

<a id="orge927dd2"></a>

## Abstract

<a id="org0cbe5e3"></a>

## Conclusion

<a id="org7feaca2"></a>

# Kouchaki2019<sub>ApplicationOfML.pdf</sub>

<a id="org45de957"></a>

## Title

<a id="orgb64d483"></a>

## Abstract

<a id="orgd5784e2"></a>

## Conclusion

<a id="org5cfaf30"></a>

# Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub>

<a id="org86e5e63"></a>

## Title

<a id="orgb5d6158"></a>

## Abstract

<a id="org69e1208"></a>

## Conclusion

<a id="org93c47bf"></a>

## Supplementary

<a id="orgbc3d479"></a>

### Kouchaki2020<sub>MultiLabel.zip</sub>

<a id="orga97997e"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>

<a id="org7c7303b"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>

<a id="org3275bb7"></a>

### Code

Kouchaki2020<sub>MultiLabelSLRF</sub>/checkmutations.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/common<sub>functions.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/cvcodes.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/demo-mtbmlrf.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/ensemble<sub>learning</sub><sub>multilabel.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakefeat.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakepheno.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/feature<sub>extraction.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/load-dataset.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/mutation<sub>ranking.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/print<sub>stats.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/readme
Kouchaki2020<sub>MultiLabelSLRF</sub>/treeplot.py

<a id="orgbbc68e3"></a>

# Kovalishyn2019<sub>HybridDesign.pdf</sub>

<a id="orgefb4396"></a>

## Title

<a id="org6384cea"></a>

## Abstract

<a id="org435f477"></a>

## Conclusion

<a id="org8e89d4d"></a>

# Kumari2019<sub>AGeneticProgramming.pdf</sub>

<a id="orgc0efb08"></a>

## Title

<a id="orgd354ab6"></a>

## Abstract

<a id="org72fd149"></a>

## Conclusion

<a id="orgf42bd6e"></a>

# Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub>

<a id="org40fcf91"></a>

## Abstract

<a id="org9edc5d3"></a>

## Conclusion

<a id="org9697582"></a>

# Lane2018<sub>ComparingAndValidating.pdf</sub>

<a id="org1a0bd32"></a>

## Abstract

<a id="org1c2e34f"></a>

## Conclusion

<a id="org16501c0"></a>

# Lin2001<sub>ConservedCodonComposition.pdf</sub>

<a id="orgf639764"></a>

## Abstract

<a id="org158ddfb"></a>

## Conclusion

<a id="orgf9f3c60"></a>

# LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub>

<a id="orgf2e11b1"></a>

## Abstract

<a id="org97c867a"></a>

## Conclusion

<a id="org67ba775"></a>

# Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub>

<a id="org09fc577"></a>

## Abstract

<a id="org3df29a5"></a>

## Conclusion

<a id="org09f78eb"></a>

# Marino2016<sub>ComputationalAndEmpirical.pdf</sub>

<a id="org821ce7d"></a>

## Abstract

<a id="org886e02d"></a>

## Conclusion

<a id="org355ab96"></a>

# Marques2019<sub>PulmonaryNTMInfections.pdf</sub>

<a id="org11646a4"></a>

## Abstract

<a id="orgf44ca00"></a>

## Conclusion

<a id="orgc6146c9"></a>

# Meldenez2015<sub>OnCombiningMultiple.pdf</sub>

<a id="orgc81f993"></a>

## Abstract

<a id="org85434fc"></a>

## Conclusion

<a id="orgdd1f848"></a>

# Mikusova2016<sub>LearningFromThePast.pdf</sub>

<a id="org4799880"></a>

## Abstract

<a id="orgebfb6a2"></a>

## Conclusion

<a id="orgbdd1328"></a>

# Ngo2019<sub>GenomicPrediction.pdf</sub>

<a id="org1b1ed3f"></a>

## Abstract

<a id="org0b4d47d"></a>

## Conclusion

<a id="org3afeda7"></a>

# Periwal2011<sub>PredictiveModels.pdf</sub>

<a id="org6f0cd02"></a>

## Abstract

<a id="orgbbde880"></a>

## Conclusion

<a id="orgcc90d7f"></a>

# Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub>

<a id="org60e602f"></a>

## Abstract

<a id="orge2123e7"></a>

## Conclusion

<a id="orga41eb5f"></a>

# Romero2020<sub>DecisionTree.pdf</sub>

<a id="org4ebc7f7"></a>

## Abstract

<a id="org71948bf"></a>

## Conclusion

<a id="org52f981e"></a>

## Supplementary

<a id="org6955fa0"></a>

### Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub>

<a id="org1da15d7"></a>

# Sauer2018<sub>FeatureSelection.pdf</sub>

<a id="org211ae2c"></a>

## Abstract

<a id="orgdafd10e"></a>

## Conclusion

<a id="org87b1ac9"></a>

# Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub>

<a id="org51d58e2"></a>

## Abstract

<a id="org4897763"></a>

## Conclusion

<a id="orge866f21"></a>

# Sergeev2016<sub>GenomeWide.pdf</sub>

<a id="orge6e4341"></a>

## Abstract

<a id="orgdc28c1c"></a>

## Conclusion

<a id="org2991928"></a>

# Vizcaino2010<sub>ComputatationalPrediction.pdf</sub>

<a id="orgb964b31"></a>

## Abstract

<a id="org8da6a79"></a>

## Conclusion

<a id="org689ca3f"></a>

# Waman2019<sub>MycobacterialGenomics.pdf</sub>

<a id="orgbbb2826"></a>

## Abstract

<a id="org2fbb7cc"></a>

## Conclusion

<a id="orgb08ddc6"></a>

# Wang2020<sub>ADeepLearningBasedMethod.pdf</sub>

<a id="org2dbb5a1"></a>

## Abstract

<a id="org1fda881"></a>

## Conclusion

<a id="org732d905"></a>

# Weiner2017<sub>HighThroughput.pdf</sub>

<a id="orgd3743ef"></a>

## Abstract

<a id="org6464031"></a>

## Conclusion

<a id="orgde02da3"></a>

# Wilder2019<sub>ImprovingTuberculosis.pdf</sub>

<a id="orgb627175"></a>

## Abstract

<a id="org93dfb7d"></a>

## Conclusion

<a id="org024a837"></a>

# Yang2017<sub>MachineLearning.pdf</sub>

<a id="org1462cc6"></a>

## Abstract

<a id="org1d0726a"></a>

## Conclusion

<a id="orgcf200fb"></a>

# Yang2019<sub>DeepAMRForPredicting.pdf</sub>

<a id="org118b0f2"></a>

## Abstract

<a id="org67d1fb7"></a>

## Conclusion

<a id="org4f55a9d"></a>

# Zhang2020<sub>AnIntegratedMLApproach.pdf</sub>

<a id="org4bac13f"></a>

## Abstract

<a id="org5ad1a7b"></a>

## Conclusion
