# Table of Contents

1.  [Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis](#orgfc22aa6)
    1.  [Notes](#org87b41e5)
    2.  [FileName](#org967e3a4)
    3.  [Abstract](#org1854bea)
    4.  [Conclusion](#orgb77a334)
2.  [Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay](#org9b5ae92)
    1.  [Notes](#orgf16f3f7)
    2.  [FileName](#orgd8683a7)
    3.  [Abstract](#orgdec93ee)
    4.  [Conclusion](#org0d79393)
3.  [Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data](#orgdca14cc)
    1.  [Notes](#org54037b1)
    2.  [FileName](#org38efe04)
    3.  [Abstract](#orgce25468)
    4.  [Conclusion](#orgce157ff)
4.  [Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models](#orgf4474e6)
    1.  [Notes](#org3af5250)
    2.  [FileName](#org40073f5)
    3.  [Abstract](#org6e66215)
    4.  [Conclusion](#org7d263c9)
5.  [Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis](#orgce0224f)
    1.  [Notes](#orgde02c36)
    2.  [FileName](#org7bd21a7)
    3.  [Abstract](#org54ad103)
    4.  [Conclusion](#org54e153c)
6.  [Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance](#org4cec430)
    1.  [Notes](#orgb901800)
    2.  [FileName](#orgc74fdc6)
    3.  [Abstract](#org31a9e81)
    4.  [Conclusion](#org3306647)
7.  [Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes](#orgc6d2c30)
    1.  [Notes](#org5edbb30)
    2.  [FileName](#orgdbd552e)
    3.  [Abstract](#orgdd7cd35)
    4.  [Conclusion](#orgeb9f737)
8.  [Killian2019<sub>LearningToPrescribe.pdf</sub>](#orgc734c91)
    1.  [Title](#org9f3fa6e)
    2.  [Abstract](#org05f81ad)
    3.  [Conclusion](#org2f3e2e9)
9.  [Kouchaki2019<sub>ApplicationOfML.pdf</sub>](#orgec2e752)
    1.  [Title](#orgaa0d3ec)
    2.  [Abstract](#org78ab0e3)
    3.  [Conclusion](#org0517fa8)
10. [Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub>](#org885b0c3)
    1.  [Title](#orgc5728d4)
    2.  [Abstract](#org6451550)
    3.  [Conclusion](#org82fad1f)
    4.  [Supplementary](#orgaeb321a)
        1.  [Kouchaki2020<sub>MultiLabel.zip</sub>](#org64a43c4)
        2.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>](#orgcf3d89c)
        3.  [Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>](#orgd19630d)
        4.  [Code](#org386baa7)
11. [Kovalishyn2019<sub>HybridDesign.pdf</sub>](#org79a2a9b)
    1.  [Title](#orgfc8adf0)
    2.  [Abstract](#org6778224)
    3.  [Conclusion](#org391a1ce)
12. [Kumari2019<sub>AGeneticProgramming.pdf</sub>](#org1f73799)
    1.  [Title](#org78f7c8a)
    2.  [Abstract](#org712e9ff)
    3.  [Conclusion](#org30e272a)

<a id="orgfc22aa6"></a>

# [-] Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis

<a id="org87b41e5"></a>

## Notes

- Image processing
- Mentiones the database source, NIH
- Has CSV as supplementary data

<a id="org967e3a4"></a>

## FileName

- Engle2019<sub>PerformanceOfQureAI.pdf</sub>

<a id="org1854bea"></a>

## Abstract

- Availability of trained radiologists for fast processing of CXRs in regions
  burdened with tuberculosis always has been a challenge, affecting both timely
  diagnosis and patient monitoring.

- The Qure.ai qXR arti- ficial intelligence automated CXR interpretation tool,
  was blind-tested on the 346 radiologist- annotated CXRs from the TBPP
  database.

- We conclude that applying fully automated Qure.ai CXR analysis tool is useful
  for fast, accurate, uniform, large-scale CXR annotation assistance, as it
  performed well even for DR-TB cases that were not used for initial training.

<a id="orgb77a334"></a>

## Conclusion

- We demonstrated that the Qure.ai qXR CXR annotations for cavity, nodule,
  pleural effusion, and hilar lymphadenopathy, are in statistical agreement with
  radiologist CXR annotations.

- A corresponding analysis of patient case matched
  CT image annotations recorded within 30 days of a CXR supports this result,
  demonstrating statistical significance for cavity, pleural effusion, hilar
  lymphadenopathy, and atelectasis.

<a id="org9b5ae92"></a>

# [-] Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay

<a id="orgf16f3f7"></a>

## Notes

- Survey paper

<a id="orgd8683a7"></a>

## FileName

- Greif2020<sub>FrequencyOfNTM.pdf</sub>

<a id="orgdec93ee"></a>

## Abstract

- Nontuberculous mycobacteria (NTM) increasingly are recognized as
  opportunistic pathogens of humans.
- NTM species distribution is well documented in Europe and North America

<a id="org0d79393"></a>

## Conclusion

- Our findings indicate that rate of NTM have risen nearly 5-fold (4.79) in
  Uruguay, from 0.33 cas- es/100,000 inhabitants in 2006 to 1.57 cases/100,000
  inhabitants in 2018.

<a id="orgdca14cc"></a>

# [-] Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data

<a id="org54037b1"></a>

## Notes

- Can perform the mentioned analysis on the MTB and NTM data
- The paper focuses on the factors affecting the application and clinical application of ML models

<a id="org38efe04"></a>

## FileName

- Hicks2019<sub>EvaluationOfParameters.pdf</sub>

<a id="orgce25468"></a>

## Abstract

- There has been no systematic evaluation of factors that may influence
  performance of ML models, how they might apply to and vary across clinical
  populations, and what the implications might be in the clinical setting

- We performed a meta-analysis of seven large Neisseria gonorrhoeae datasets, as
  well as Klebsiella pneumoniae and Acinetobacter baumannii datasets, with whole
  genome sequence data and antibiotic susceptibility phenotypes using set
  covering machine classifi- cation, random forest classification, and random
  forest regression models to predict resis- tance phenotypes from genotype.

- We demonstrate how model performance varies by drug, dataset, resistance
  metric, and species, reflecting the complexities of generating clinically
  relevant conclusions from machine learning-derived models.

- Our results highlight significant biological and technical challenges
  facing the application of machine learning-based prediction of antibiotic
  resistance as a diagnostic tool.

<a id="orgce157ff"></a>

## Conclusion

<a id="orgf4474e6"></a>

# STRT Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models

<a id="org3af5250"></a>

## Notes

- Can use a different algorithm (XgBoost) on the dataset
- Sample data and complete code is shared
  <https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics>
- Need to find the full dataset of 4000 surveyed patients

<a id="org40073f5"></a>

## FileName

- Hussain2018<sub>PredictingTreatmentOutcomes.pdf</sub>

<a id="org6e66215"></a>

## Abstract

- Tuberculosis (TB) is a deadly contagious disease and a serious global health
  problem. It is curable but due to its lengthy treatment process, a patient is
  likely to leave the treatment incomplete, leading to a more lethal, drug
  resistant form of disease.

- We aim to facilitate TB programs by predicting the outcome of the treatment of
  a particular patient at the start of treatment so that their health workers
  can be utilized in a targeted and cost-effective way.

- The problem was modeled as a classification problem, and the outcome of
  treatment was predicted using state-of-art implementations of 3 machine learning algorithms

<a id="org7d263c9"></a>

## Conclusion

- Empirical results using data of more than 4000 patients reveal that we can
  predict with more than 76% accuracy whether a TB patient would complete his (or her) treatment or not
-

<a id="orgce0224f"></a>

# STRT Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis

<a id="orgde02c36"></a>

## Notes

- Used many ML algorithms, 85% accuracy
- Could use XgBoost
- Could conduct a similar study for NTM
- Mentions many online bio databases

<a id="org7bd21a7"></a>

## FileName

- Jamal2020<sub>AIAndMLBasedPredictive.pdf</sub>

<a id="org54ad103"></a>

## Abstract

- The present work is a computational framework that uses artificial
  intelligence (AI) based machine learning (ML) approaches for predicting
  resistance in the genes rpoB, inhA, katG, pncA, gyrA and gyrB for the drugs
  rifampicin, isoniazid, pyrazinamide and fluoroquinolones

- The single nucleotide variations were represented by several sequence and
  structural features that indicate the influence of mutations on the target
  protein coded by each gene.

<a id="org54e153c"></a>

## Conclusion

- The present study proposes an AI/ML method to classify resistant and susceptible
  mutations in TB and predict novel resistance conferring mutations.

- The impact of the reported mutations was captured in the form of changes in the amino acid
  residues, and the consequent change in properties vis-a-vis wild-type and mutant
  proteins and represented as features used to train the models.

- The classification model was generated for each gene and predictions were made for
  SNVs linked with each gene for each drug

<a id="org4cec430"></a>

# STRT Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance

<a id="orgb901800"></a>

## Notes

- They developed and online platform

- Can be applied to NTM

- Code is available
  <https://github.com/biosharp-dotnet/microbial_AMR_ML>

<a id="orgc74fdc6"></a>

## FileName

- Kavvas2018<sub>MachineLearning</sub><sub>.pdf</sub>

<a id="org31a9e81"></a>

## Abstract

- The many publicly available datasets describing its AMR characteristics demand
  disparate data-type analyses.

- Here, we develop a reference strain-agnostic computational platform that uses
  machine learning approaches, complemented by both genetic interaction analysis
  and 3D structural mutation-mapping, to identify signatures of AMR evolution to
  13 antibiotics.

- First, a pan-genome analysis shows that M. tuberculosis is highly conserved
  with sequenced variation concentrated in PE/PPE/PGRS genes.

- Second, the platform corroborates 33 genes known to confer resistance and identifies 24
  new genetic signatures of AMR.

- Third, 97 epistatic interactions across 10 resistance classes are revealed.

- Fourth, detailed structural analysis of these
  genes yields mechanistic bases for their selection.

- The platform can be used to study other human pathogens

<a id="org3306647"></a>

## Conclusion

- The platform was applied to 1595 sequenced strains to yield results in four
  categories: pan-genome properties, identification of genes conferring
  antibiotic resistance, their epistatic interactions, and protein structure
  based mechanistic insights.

<a id="orgc6d2c30"></a>

# Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes

<a id="org5edbb30"></a>

## Notes

-

<a id="orgdbd552e"></a>

## FileName

- Khanna2018<sub>EnsembleTechnique.pdf</sub>

<a id="orgdd7cd35"></a>

## Abstract

- Development of an effective machine-learning model for T-cell Mycobacterium
  tuberculosis (M. tuberculosis) epitopes is beneficial for saving biologistâ€™s
  time and effort for identifying epitope in a targeted antigen.

- The proposed method is able to work with variable length epitopes having size

even greater than 9-mers. Identification of T-cell or B-cell epitopes in the
targeted antigen is the main goal in designing epitope-based vaccine,
immune-diagnostic tests and antibody production

<a id="orgeb9f737"></a>

## Conclusion

<a id="orgc734c91"></a>

# Killian2019<sub>LearningToPrescribe.pdf</sub>

<a id="org9f3fa6e"></a>

## Title

<a id="org05f81ad"></a>

## Abstract

<a id="org2f3e2e9"></a>

## Conclusion

<a id="orgec2e752"></a>

# Kouchaki2019<sub>ApplicationOfML.pdf</sub>

<a id="orgaa0d3ec"></a>

## Title

<a id="org78ab0e3"></a>

## Abstract

<a id="org0517fa8"></a>

## Conclusion

<a id="org885b0c3"></a>

# Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub>

<a id="orgc5728d4"></a>

## Title

<a id="org6451550"></a>

## Abstract

<a id="org82fad1f"></a>

## Conclusion

<a id="orgaeb321a"></a>

## Supplementary

<a id="org64a43c4"></a>

### Kouchaki2020<sub>MultiLabel.zip</sub>

<a id="orgcf3d89c"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub>

<a id="orgd19630d"></a>

### Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub>

<a id="org386baa7"></a>

### Code

Kouchaki2020<sub>MultiLabelSLRF</sub>/checkmutations.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/common<sub>functions.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/cvcodes.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/demo-mtbmlrf.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/ensemble<sub>learning</sub><sub>multilabel.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakefeat.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakepheno.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/feature<sub>extraction.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/load-dataset.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/mutation<sub>ranking.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/print<sub>stats.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/readme
Kouchaki2020<sub>MultiLabelSLRF</sub>/treeplot.py

<a id="org79a2a9b"></a>

# Kovalishyn2019<sub>HybridDesign.pdf</sub>

<a id="orgfc8adf0"></a>

## Title

<a id="org6778224"></a>

## Abstract

<a id="org391a1ce"></a>

## Conclusion

<a id="org1f73799"></a>

# Kumari2019<sub>AGeneticProgramming.pdf</sub>

<a id="org78f7c8a"></a>

## Title

<a id="org712e9ff"></a>

## Abstract

<a id="org30e272a"></a>

## Conclusion
