<?xml version="1.0" encoding="utf-8"?>
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2020-05-07 Thu 23:23 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Other Papers</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Abhinav Sharma" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="https://fniessen.github.io/org-html-themes/styles/readtheorg/css/htmlize.css"/>
<link rel="stylesheet" type="text/css" href="https://fniessen.github.io/org-html-themes/styles/readtheorg/css/readtheorg.css"/>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script type="text/javascript" src="https://fniessen.github.io/org-html-themes/styles/lib/js/jquery.stickytableheaders.min.js"></script>
<script type="text/javascript" src="https://fniessen.github.io/org-html-themes/styles/readtheorg/js/readtheorg.js"></script>
<style> p{margin-bottom:5px;}</style>
<style> li{margin-bottom:10px}</style
<script type="text/javascript">
// @license magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&amp;dn=gpl-3.0.txt GPL-v3-or-Later
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.cacheClassElem = elem.className;
         elem.cacheClassTarget = target.className;
         target.className = "code-highlighted";
         elem.className   = "code-highlighted";
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(elem.cacheClassElem)
         elem.className = elem.cacheClassElem;
       if(elem.cacheClassTarget)
         target.className = elem.cacheClassTarget;
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="content">
<h1 class="title">Other Papers</h1>
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#orgeac68ff">1. <span class="todo TODO">TODO</span> Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.</a></li>
<li><a href="#org67b4839">2. <span class="todo TODO">TODO</span> Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.</a></li>
<li><a href="#org146b232">3. <span class="todo TODO">TODO</span> Machine learning and tubercular drug target recognition.</a></li>
<li><a href="#org67c0b60">4. <span class="todo WAIT">WAIT</span> Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem</a>
<ul>
<li><a href="#orge8c5b26">4.1. Notes</a></li>
<li><a href="#org08d0f5d">4.2. FileName</a></li>
<li><a href="#org3172342">4.3. Abstract</a></li>
<li><a href="#orgbb85b0f">4.4. FutureWork</a></li>
</ul>
</li>
<li><a href="#org0286b05">5. <span class="todo STRT">STRT</span> Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm</a>
<ul>
<li><a href="#org415be9e">5.1. Notes</a></li>
<li><a href="#org8a8a101">5.2. FileName</a></li>
<li><a href="#orgd3c11d0">5.3. Abstract</a></li>
<li><a href="#org5feafc5">5.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org2bfe471">6. <span class="todo ___">[-]</span> Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper</a>
<ul>
<li><a href="#orga2f3623">6.1. Notes</a></li>
<li><a href="#org4acb51e">6.2. FileName</a></li>
<li><a href="#orge202f06">6.3. Abstract</a></li>
<li><a href="#orgbe3e589">6.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd30053b">7. <span class="todo STRT">STRT</span> Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities</a>
<ul>
<li><a href="#orgd9163ed">7.1. Notes</a></li>
<li><a href="#orge4938e7">7.2. FileName</a></li>
<li><a href="#org17308bd">7.3. Abstract</a></li>
<li><a href="#orgfc3207b">7.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgfaebcad">8. <span class="todo STRT">STRT</span> Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data</a>
<ul>
<li><a href="#orgece2c0a">8.1. Notes</a></li>
<li><a href="#org999d68b">8.2. FileName</a></li>
<li><a href="#orgc31eb72">8.3. Abstract</a></li>
<li><a href="#orgf235def">8.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org1846218">9. <span class="todo STRT">STRT</span> Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction</a>
<ul>
<li><a href="#org471d5e2">9.1. Notes</a></li>
<li><a href="#org009a017">9.2. FileName</a></li>
<li><a href="#org9e4cd7f">9.3. Abstract</a></li>
<li><a href="#org3854dae">9.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgee9555a">10. <span class="todo ___">[-]</span> Ching2018 - Opportunities and obstacles for deep learning in biology and medicine</a>
<ul>
<li><a href="#orgae9b29c">10.1. Notes</a></li>
<li><a href="#org7b6eb2e">10.2. FileName</a></li>
<li><a href="#orgc66a5ac">10.3. Abstract</a></li>
<li><a href="#org8cbc3a1">10.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd6e47e1">11. <span class="todo STRT">STRT</span> Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data</a>
<ul>
<li><a href="#org0d024ea">11.1. Notes</a></li>
<li><a href="#orgb545f12">11.2. FileName</a></li>
<li><a href="#org663044e">11.3. Abstract</a></li>
<li><a href="#org2df82ce">11.4. Conclusion</a></li>
<li><a href="#org1705e89">11.5. Supplementary</a>
<ul>
<li><a href="#orgcde97fe">11.5.1. Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#orge6ab3fd">12. <span class="todo ___">[?]</span> Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org061793b">12.1. Notes</a></li>
<li><a href="#orgdcebba1">12.2. FileName</a></li>
<li><a href="#orgdbf7885">12.3. Abstract</a></li>
<li><a href="#org980d021">12.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org17328bd">13. <span class="todo STRT">STRT</span> Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports</a>
<ul>
<li><a href="#org4a4d45a">13.1. Notes</a></li>
<li><a href="#org679f37b">13.2. FileName</a></li>
<li><a href="#orge3414e1">13.3. Abstract</a></li>
<li><a href="#org7625801">13.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgbcef9f9">14. <span class="todo ___">[-]</span> Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence</a>
<ul>
<li><a href="#org86f3828">14.1. Notes</a></li>
<li><a href="#orgc9907ed">14.2. FileName</a></li>
<li><a href="#org96abdcd">14.3. Abstract</a></li>
<li><a href="#org8fc9219">14.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgb1a05fb">15. <span class="done _X_">[X]</span> Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models</a>
<ul>
<li><a href="#org883a1dd">15.1. Notes</a></li>
<li><a href="#orgbfb3e6b">15.2. FileName</a></li>
<li><a href="#orga9c6026">15.3. Abstract</a></li>
<li><a href="#org5b83b87">15.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org7717792">16. <span class="done _X_">[X]</span> Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation</a>
<ul>
<li><a href="#org65e674b">16.1. Notes</a></li>
<li><a href="#org6e97aba">16.2. FileName</a></li>
<li><a href="#orgb843ce5">16.3. Abstract</a></li>
<li><a href="#org16e989f">16.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org8d5b41f">17. <span class="done _X_">[X]</span> Ekins2014<sub>AreBiggerDataSets.pdf</sub></a>
<ul>
<li><a href="#org91d56d0">17.1. Notes</a></li>
<li><a href="#org8b14c08">17.2. Title</a></li>
<li><a href="#org177208c">17.3. Abstract</a></li>
<li><a href="#orgcfdae93">17.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org559393c">18. <span class="done _X_">[X]</span> Ekins2014<sub>CombiningComputationalMethods.pdf</sub></a>
<ul>
<li><a href="#org6145e2a">18.1. Notes</a></li>
<li><a href="#orga4118cd">18.2. Title</a></li>
<li><a href="#org5118973">18.3. Abstract</a></li>
<li><a href="#orgf04a27f">18.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgf4c9010">19. <span class="done _X_">[X]</span> Ekins2016<sub>MachineLearningModel.pdf</sub></a>
<ul>
<li><a href="#org2889971">19.1. Notes</a></li>
<li><a href="#orgfb532cd">19.2. Title</a></li>
<li><a href="#orgd1604d7">19.3. Abstract</a></li>
<li><a href="#orgf02f0e3">19.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org877fe2d">20. <span class="done _X_">[X]</span> Ekins2017<sub>MachineLearning.pdf</sub></a>
<ul>
<li><a href="#orgf04d498">20.1. Notes</a></li>
<li><a href="#org1858d12">20.2. Title</a></li>
<li><a href="#org4ec503f">20.3. Abstract</a></li>
<li><a href="#org3fcc27d">20.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd8c8050">21. <span class="todo ___">[-]</span> Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis</a>
<ul>
<li><a href="#orgae5efe7">21.1. Notes</a></li>
<li><a href="#org87fc61a">21.2. FileName</a></li>
<li><a href="#org54bfc94">21.3. Abstract</a></li>
<li><a href="#orgb6af2a9">21.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgfcc5de8">22. <span class="todo ___">[-]</span> Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay</a>
<ul>
<li><a href="#org4fa3c68">22.1. Notes</a></li>
<li><a href="#org166e779">22.2. FileName</a></li>
<li><a href="#org55b5d4f">22.3. Abstract</a></li>
<li><a href="#orgb60faf1">22.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgede7fa3">23. <span class="todo ___">[-]</span> Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data</a>
<ul>
<li><a href="#orgce3ce73">23.1. Notes</a></li>
<li><a href="#org5b46cc2">23.2. FileName</a></li>
<li><a href="#org0eb38d1">23.3. Abstract</a></li>
<li><a href="#org29c94dd">23.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgc44d895">24. <span class="todo STRT">STRT</span> Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models</a>
<ul>
<li><a href="#orgf971921">24.1. Notes</a></li>
<li><a href="#org87337eb">24.2. FileName</a></li>
<li><a href="#org554b93a">24.3. Abstract</a></li>
<li><a href="#orgcaa3ee5">24.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org3a22f80">25. <span class="todo STRT">STRT</span> Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis</a>
<ul>
<li><a href="#org42001db">25.1. Notes</a></li>
<li><a href="#org00d8e33">25.2. FileName</a></li>
<li><a href="#orgb44d2f2">25.3. Abstract</a></li>
<li><a href="#org977d047">25.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#org0a3d79b">26. <span class="todo STRT">STRT</span> Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance</a>
<ul>
<li><a href="#org3c3b766">26.1. Notes</a></li>
<li><a href="#org2d63eb5">26.2. FileName</a></li>
<li><a href="#orgbe3fb79">26.3. Abstract</a></li>
<li><a href="#org6732594">26.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgd865f61">27. Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes</a>
<ul>
<li><a href="#org124d613">27.1. Notes</a></li>
<li><a href="#org945d563">27.2. FileName</a></li>
<li><a href="#orgd651ccf">27.3. Abstract</a></li>
<li><a href="#org30646fd">27.4. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgc97b276">28. Killian2019<sub>LearningToPrescribe.pdf</sub></a>
<ul>
<li><a href="#org1d78b5e">28.1. Title</a></li>
<li><a href="#org7496bb9">28.2. Abstract</a></li>
<li><a href="#orgc6c0bba">28.3. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgb12fe76">29. Kouchaki2019<sub>ApplicationOfML.pdf</sub></a>
<ul>
<li><a href="#orgd23d957">29.1. Title</a></li>
<li><a href="#orgc546144">29.2. Abstract</a></li>
<li><a href="#org95ff3e8">29.3. Conclusion</a></li>
</ul>
</li>
<li><a href="#org2f53f70">30. Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub></a>
<ul>
<li><a href="#org90a5f65">30.1. Title</a></li>
<li><a href="#orgc0611c6">30.2. Abstract</a></li>
<li><a href="#org363e209">30.3. Conclusion</a></li>
<li><a href="#orgdd48189">30.4. Supplementary</a>
<ul>
<li><a href="#orgfb24d75">30.4.1. Kouchaki2020<sub>MultiLabel.zip</sub></a></li>
<li><a href="#org59cc48e">30.4.2. Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub></a></li>
<li><a href="#orgdd3806b">30.4.3. Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub></a></li>
<li><a href="#orgabe2b5a">30.4.4. Code</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org7b87ad4">31. Kovalishyn2019<sub>HybridDesign.pdf</sub></a>
<ul>
<li><a href="#org453a261">31.1. Title</a></li>
<li><a href="#orgf828b0a">31.2. Abstract</a></li>
<li><a href="#org08facb9">31.3. Conclusion</a></li>
</ul>
</li>
<li><a href="#org2162fbc">32. Kumari2019<sub>AGeneticProgramming.pdf</sub></a>
<ul>
<li><a href="#org200f695">32.1. Title</a></li>
<li><a href="#orgd9cd074">32.2. Abstract</a></li>
<li><a href="#org8ee76b4">32.3. Conclusion</a></li>
</ul>
</li>
<li><a href="#org1943dbf">33. Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub></a>
<ul>
<li><a href="#org8ebb60b">33.1. Abstract</a></li>
<li><a href="#org28329fb">33.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgc6cdfed">34. Lane2018<sub>ComparingAndValidating.pdf</sub></a>
<ul>
<li><a href="#orgae6002e">34.1. Abstract</a></li>
<li><a href="#org9a3501c">34.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgbe70cc1">35. Lin2001<sub>ConservedCodonComposition.pdf</sub></a>
<ul>
<li><a href="#orgd95c486">35.1. Abstract</a></li>
<li><a href="#orge116cb7">35.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org082ed14">36. LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub></a>
<ul>
<li><a href="#org9b8e95b">36.1. Abstract</a></li>
<li><a href="#orge1f03f7">36.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgdefaecb">37. Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub></a>
<ul>
<li><a href="#org81406fa">37.1. Abstract</a></li>
<li><a href="#org50ca4a9">37.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgba0ea42">38. Marino2016<sub>ComputationalAndEmpirical.pdf</sub></a>
<ul>
<li><a href="#org35d9381">38.1. Abstract</a></li>
<li><a href="#orga7f15c7">38.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org28ac74b">39. Marques2019<sub>PulmonaryNTMInfections.pdf</sub></a>
<ul>
<li><a href="#org6b562a9">39.1. Abstract</a></li>
<li><a href="#orgd7c434b">39.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org0446bbb">40. Meldenez2015<sub>OnCombiningMultiple.pdf</sub></a>
<ul>
<li><a href="#org46e872c">40.1. Abstract</a></li>
<li><a href="#org463c2b6">40.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org4ebc3d1">41. Mikusova2016<sub>LearningFromThePast.pdf</sub></a>
<ul>
<li><a href="#org5c0630c">41.1. Abstract</a></li>
<li><a href="#org9287af9">41.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgc2dbaf1">42. Ngo2019<sub>GenomicPrediction.pdf</sub></a>
<ul>
<li><a href="#org3fef9ca">42.1. Abstract</a></li>
<li><a href="#org3714ca4">42.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org8e5b7e6">43. Periwal2011<sub>PredictiveModels.pdf</sub></a>
<ul>
<li><a href="#orgbe53ccc">43.1. Abstract</a></li>
<li><a href="#org265a117">43.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org4fb0d36">44. Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub></a>
<ul>
<li><a href="#org891293c">44.1. Abstract</a></li>
<li><a href="#orgf8573f2">44.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org8039d2a">45. Romero2020<sub>DecisionTree.pdf</sub></a>
<ul>
<li><a href="#orga922e4a">45.1. Abstract</a></li>
<li><a href="#org7045fa2">45.2. Conclusion</a></li>
<li><a href="#orgdf19664">45.3. Supplementary</a>
<ul>
<li><a href="#org92ced8e">45.3.1. Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#org9b71e24">46. Sauer2018<sub>FeatureSelection.pdf</sub></a>
<ul>
<li><a href="#org469e39a">46.1. Abstract</a></li>
<li><a href="#org5c57fea">46.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org4b1a247">47. Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub></a>
<ul>
<li><a href="#org6989bbc">47.1. Abstract</a></li>
<li><a href="#org2dd0cbf">47.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org92ca51d">48. Sergeev2016<sub>GenomeWide.pdf</sub></a>
<ul>
<li><a href="#org38d8f40">48.1. Abstract</a></li>
<li><a href="#org720a485">48.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org2ad1e25">49. Vizcaino2010<sub>ComputatationalPrediction.pdf</sub></a>
<ul>
<li><a href="#orgfd3a860">49.1. Abstract</a></li>
<li><a href="#org5a0812b">49.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org007d856">50. Waman2019<sub>MycobacterialGenomics.pdf</sub></a>
<ul>
<li><a href="#org0be8aac">50.1. Abstract</a></li>
<li><a href="#org7f755fb">50.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgadcc047">51. Wang2020<sub>ADeepLearningBasedMethod.pdf</sub></a>
<ul>
<li><a href="#org175df56">51.1. Abstract</a></li>
<li><a href="#org2fa1db2">51.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org115a71a">52. Weiner2017<sub>HighThroughput.pdf</sub></a>
<ul>
<li><a href="#org6234e4d">52.1. Abstract</a></li>
<li><a href="#orgd1c84eb">52.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgb9fb9ef">53. Wilder2019<sub>ImprovingTuberculosis.pdf</sub></a>
<ul>
<li><a href="#org4963c28">53.1. Abstract</a></li>
<li><a href="#orgec277b9">53.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#orgc43a0dd">54. Yang2017<sub>MachineLearning.pdf</sub></a>
<ul>
<li><a href="#org47c9ebe">54.1. Abstract</a></li>
<li><a href="#org90664b9">54.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org0d45ec8">55. Yang2019<sub>DeepAMRForPredicting.pdf</sub></a>
<ul>
<li><a href="#orgfb3c3e7">55.1. Abstract</a></li>
<li><a href="#org57fb5d0">55.2. Conclusion</a></li>
</ul>
</li>
<li><a href="#org3b19f04">56. Zhang2020<sub>AnIntegratedMLApproach.pdf</sub></a>
<ul>
<li><a href="#org7a8dd9b">56.1. Abstract</a></li>
<li><a href="#orgf028465">56.2. Conclusion</a></li>
</ul>
</li>
</ul>
</div>
</div>

<div id="outline-container-orgeac68ff" class="outline-2">
<h2 id="orgeac68ff"><span class="section-number-2">1</span> <span class="todo TODO">TODO</span> Advances in Computational Studies of Potential Drug Targets in Mycobacterium tuberculosis.</h2>
<div class="outline-text-2" id="text-1">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30084331">https://www.ncbi.nlm.nih.gov/pubmed/30084331</a>
</p>
</div>
</div>
<div id="outline-container-org67b4839" class="outline-2">
<h2 id="org67b4839"><span class="section-number-2">2</span> <span class="todo TODO">TODO</span> Hybrid design of isonicotinic acid hydrazide derivatives: machine learning studies, synthesis and biological evaluation of their anti-tuberculosis activity.</h2>
<div class="outline-text-2" id="text-2">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30973110">https://www.ncbi.nlm.nih.gov/pubmed/30973110</a>
</p>
</div>
</div>
<div id="outline-container-org146b232" class="outline-2">
<h2 id="org146b232"><span class="section-number-2">3</span> <span class="todo TODO">TODO</span> Machine learning and tubercular drug target recognition.</h2>
<div class="outline-text-2" id="text-3">
<p>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/24245763">https://www.ncbi.nlm.nih.gov/pubmed/24245763</a>
</p>
</div>
</div>





<div id="outline-container-org67c0b60" class="outline-2">
<h2 id="org67c0b60"><span class="section-number-2">4</span> <span class="todo WAIT">WAIT</span> Ahmed2020 - Non-tuberculous mycobacterial infections—A neglected and emerging problem</h2>
<div class="outline-text-2" id="text-4">
</div>
<div id="outline-container-orge8c5b26" class="outline-3">
<h3 id="orge8c5b26"><span class="section-number-3">4.1</span> Notes</h3>
<div class="outline-text-3" id="text-4-1">
<ul class="org-ul">
<li>Survey paper</li>
<li>Could be used build up the background for the need for research in NTM</li>
</ul>
</div>
</div>
<div id="outline-container-org08d0f5d" class="outline-3">
<h3 id="org08d0f5d"><span class="section-number-3">4.2</span> FileName</h3>
<div class="outline-text-3" id="text-4-2">
<ul class="org-ul">
<li>Ahmed2020<sub>NonTuberculosisMycoANeglectedAndEmergingProblem.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org3172342" class="outline-3">
<h3 id="org3172342"><span class="section-number-3">4.3</span> Abstract</h3>
<div class="outline-text-3" id="text-4-3">
<ul class="org-ul">
<li>The incidence of NTM infections is rising owing to a growing population of
immunocompromised and vulnerable individuals, complex medical and surgical
procedures, as well as increased awareness and diagnostic capabilities.</li>
</ul>
</div>
</div>
<div id="outline-container-orgbb85b0f" class="outline-3">
<h3 id="orgbb85b0f"><span class="section-number-3">4.4</span> FutureWork</h3>
<div class="outline-text-3" id="text-4-4">
<ul class="org-ul">
<li>Simpler and more rapid tests would be ideal in making the diagnosis of NTM
disease more quickly. However, the development of such tests is complicated
by the large number of NTM species that can cause disease and by the criteria
for disease, which require more than just the presence of the microorganism.
As a result, NTM disease will require multiple diagnostic parameters, leading
to delays and higher costs. More specific biomarkers to monitor the progress
of treatment and test cure are lacking.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org0286b05" class="outline-2">
<h2 id="org0286b05"><span class="section-number-2">5</span> <span class="todo STRT">STRT</span> Aze2015 - Genomics and Machine Learning for Taxonomy Consensus: The Mycobacterium tuberculosis Complex Paradigm</h2>
<div class="outline-text-2" id="text-5">
</div>
<div id="outline-container-org415be9e" class="outline-3">
<h3 id="org415be9e"><span class="section-number-3">5.1</span> Notes</h3>
<div class="outline-text-3" id="text-5-1">
<ul class="org-ul">
<li>Can adopt a similar approach towards the classification of NTB lineages</li>
<li>Can alter the algorithm to XGBoost to Neural Networks to improve accuracy</li>
</ul>
</div>
</div>
<div id="outline-container-org8a8a101" class="outline-3">
<h3 id="org8a8a101"><span class="section-number-3">5.2</span> FileName</h3>
<div class="outline-text-3" id="text-5-2">
<ul class="org-ul">
<li>Aze2015<sub>GenomicsAndMachine.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgd3c11d0" class="outline-3">
<h3 id="orgd3c11d0"><span class="section-number-3">5.3</span> Abstract</h3>
<div class="outline-text-3" id="text-5-3">
<ul class="org-ul">
<li>Mycobacterium tuberculosis complex taxonomy has rapidly evolved in the last 20
years through intensive clinical isolation, advances in sequencing and in the
description of fast-evolving loci (CRISPR and MIRU-VNTR)</li>

<li>The objectives of this study were 1) to explicit the consensus that exists
between the alternative taxonomies, and 2) to provide an on-line tool to ease
classification of new isolates. Genotyping (24-VNTR, 43-spacers spoli-
gotypes, IS6110-RFLP) was undertaken for 3,454 clinical isolates from the
Netherlands (2004-2008). The resulting database was enlarged with African
isolates to include most human tuberculosis diversity. Assignations were
obtained using TB-Lineage, MIRU- VNTRPlus, SITVITWEB and an algorithm from
Borile et al</li>

<li>Original and consensus assignations of the all isolates from the database were
subsequently implemented into an ensemble learning approach based on Machine
Learning tool Weka to derive a classification scheme.</li>

<li>When applied to independent datasets, it was able to suggest new sublineages
such as pseudo-Beijing. This Lineage Prediction tool, efficient on 15-MIRU,
24-VNTR and spoligotype data is available on the web interface “TBminer.”</li>

<li>We successfully used Machine Learning on a large dataset to set up and make
available the first consensual taxonomy for human Mycobacterium tuberculosis
complex</li>
</ul>
</div>
</div>

<div id="outline-container-org5feafc5" class="outline-3">
<h3 id="org5feafc5"><span class="section-number-3">5.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-5-4">
<ul class="org-ul">
<li>We developed an approach making use of standard typing data for human M.</li>
</ul>
<p>
tuberculosis isolates to infer a consensual taxonomy concordant with most
up-to-date data on Whole Genome Sequencing diversity.
</p>

<ul class="org-ul">
<li>We hope that the same approach can benefit other human pathogens having
alternative taxonomies according to Serotypes, CRISPRtypes, MLST-types such as
Salmo- nella, Listeria, Brucella, and more broadly to other organisms such as
bacterial plant- pathogens.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org2bfe471" class="outline-2">
<h2 id="org2bfe471"><span class="section-number-2">6</span> <span class="todo ___">[-]</span> Bagherian2019 - Machine learning approaches and databases for prediction of drug–target interaction: a survey paper</h2>
<div class="outline-text-2" id="text-6">
</div>
<div id="outline-container-orga2f3623" class="outline-3">
<h3 id="orga2f3623"><span class="section-number-3">6.1</span> Notes</h3>
<div class="outline-text-3" id="text-6-1">
<ul class="org-ul">
<li>Not directly related to (N)TB</li>
<li>Survey paper</li>
<li>Analyzes various ML techniques</li>
<li>Mentions various drug-target-interaction databases</li>
</ul>
</div>
</div>
<div id="outline-container-org4acb51e" class="outline-3">
<h3 id="org4acb51e"><span class="section-number-3">6.2</span> FileName</h3>
<div class="outline-text-3" id="text-6-2">
<ul class="org-ul">
<li>Bagherian2019<sub>MachineLearning.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orge202f06" class="outline-3">
<h3 id="orge202f06"><span class="section-number-3">6.3</span> Abstract</h3>
<div class="outline-text-3" id="text-6-3">
<ul class="org-ul">
<li>There is a need to develop novel and efficient prediction approaches in order
to avoid costly and laborious yet not-always-deterministic experiments to
determine drug–target interactions (DTIs) by experiments alone</li>
<li>In this article, we describe the data required for the task of DTI prediction
followed by a comprehensive catalog consisting of machine learning methods and
databases, which have been proposed and utilized to predict DTIs</li>
<li>The advantages and disadvantages of each set of methods are also briefly
discussed</li>
</ul>
</div>
</div>
<div id="outline-container-orgbe3e589" class="outline-3">
<h3 id="orgbe3e589"><span class="section-number-3">6.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-6-4">
<ul class="org-ul">
<li>Almost all the methods used in DTI prediction, particularly similarity-based
methods, heavily rely on assertions concerning similar drugs and similar
targets, the type of database used for the prediction plays a significant
role.</li>
<li>Future predictions should rely on more comprehensive inter- nal databases,
which would require a significant effort to map and curate data across the
sources that utilize different ways to define, name and identify the drugs and
targets.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgd30053b" class="outline-2">
<h2 id="orgd30053b"><span class="section-number-2">7</span> <span class="todo STRT">STRT</span> Cao2016 - Improving Tuberculosis Diagnostics Using Deep Learning and Mobile Health Technologies among Resource-Poor and Marginalized Communities</h2>
<div class="outline-text-2" id="text-7">
</div>
<div id="outline-container-orgd9163ed" class="outline-3">
<h3 id="orgd9163ed"><span class="section-number-3">7.1</span> Notes</h3>
<div class="outline-text-3" id="text-7-1">
<ul class="org-ul">
<li>Image processing algorithms, CNN</li>
<li>They proposed an online database of X-ray images - by now this might have been developed</li>
</ul>
</div>
</div>
<div id="outline-container-orge4938e7" class="outline-3">
<h3 id="orge4938e7"><span class="section-number-3">7.2</span> FileName</h3>
<div class="outline-text-3" id="text-7-2">
<ul class="org-ul">
<li>Cao2016<sub>ImprovingDiagnosticsUsingDL.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org17308bd" class="outline-3">
<h3 id="org17308bd"><span class="section-number-3">7.3</span> Abstract</h3>
<div class="outline-text-3" id="text-7-3">
<ul class="org-ul">
<li>The combination of machine learning and mobile computing techniques offers a
unique opportunity to accelerate the TB diagnosis among these communities</li>

<li>We first introduce major technique barriers and proposed system architecture</li>

<li>The first activity aims to develop large-scale, real-world and well-annotated
X-ray image database dedicated for automated TB screening.</li>

<li>The second research activity focus on developing effective and efficient
computational models (in particularly, deep convolutional neural networks
(CNN)-based models) to classify the image into different category of TB
manifestations</li>

<li>Our future work includes: (1) to further improve the performance of the
algorithms; and (2) to deploy our system in the city of Carabayllo in Perú, a
densely occupied urban community and high-burden TB</li>
</ul>
</div>
</div>

<div id="outline-container-orgfc3207b" class="outline-3">
<h3 id="orgfc3207b"><span class="section-number-3">7.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-7-4">
<ul class="org-ul">
<li>The first progress is to build a large-scale, real-world, and well-annotated chest X- ray image
database dedicated for TB screening.</li>

<li>The second progress is to develop effective and efficient computational model for TB manifestation
categorization.</li>

<li>We will continue to develop the large scale, real-world X-ray TB database, as
well as continuing to improve the performance of the computing algorithms</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgfaebcad" class="outline-2">
<h2 id="orgfaebcad"><span class="section-number-2">8</span> <span class="todo STRT">STRT</span> Chen2018 - Deep Learning Predicts Tuberculosis Drug Resistance Status from Whole-Genome Sequencing Data</h2>
<div class="outline-text-2" id="text-8">
</div>
<div id="outline-container-orgece2c0a" class="outline-3">
<h3 id="orgece2c0a"><span class="section-number-3">8.1</span> Notes</h3>
<div class="outline-text-3" id="text-8-1">
<ul class="org-ul">
<li>A good in-depth ML based paper</li>
<li>They&rsquo;ve shared high-level implementation details briefly</li>
<li>Hyperparameters are also in the supplementary material</li>
<li>I was able to find the code on Github
<a href="https://github.com/biosharp-dotnet/wdnn">https://github.com/biosharp-dotnet/wdnn</a></li>
<li>Could apply the same techniques to NTM</li>
<li>Could change the algorithm and apply on the same data</li>
<li>The author built upon the previous paper by improving the accuracy of the ML model - we could do the same with NTM</li>
</ul>
</div>
</div>
<div id="outline-container-org999d68b" class="outline-3">
<h3 id="org999d68b"><span class="section-number-3">8.2</span> FileName</h3>
<div class="outline-text-3" id="text-8-2">
<ul class="org-ul">
<li>Chen2018<sub>DeepLearningPredictsTuberculosis.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgc31eb72" class="outline-3">
<h3 id="orgc31eb72"><span class="section-number-3">8.3</span> Abstract</h3>
<div class="outline-text-3" id="text-8-3">
<ul class="org-ul">
<li>Using targeted or whole genome sequencing and conventional drug resistance
phenotyping data from 3,601 Mycobacterium tuberculosis strains, 1,228 of which
were multidrug resistant, we implemented the first multitask deep learning
framework to predict phenotypic drug resistance to 10 anti-tubercular drugs</li>

<li>The proposed wide and deep neural network (WDNN) achieved improved predictive
performance compared to regularized logistic regression and random forest</li>
</ul>
</div>
</div>

<div id="outline-container-orgf235def" class="outline-3">
<h3 id="orgf235def"><span class="section-number-3">8.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-8-4">
<ul class="org-ul">
<li>We demonstrate that L2 regression and MD-WDNN trained on a larger diverse
dataset using a method of aggregating rare variants outperforms our previously
reported random forest model. (Chen, 2018)</li>

<li>A few prior studies have utilized algorithmic or machine learning methods
using genomic data to account for the complex relationship between genotype
and drug resistance in MTB</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org1846218" class="outline-2">
<h2 id="org1846218"><span class="section-number-2">9</span> <span class="todo STRT">STRT</span> Chen2019 - Beyond multidrug resistance: Leveraging rare variants with machine and statistical learning models in Mycobacterium tuberculosis resistance prediction</h2>
<div class="outline-text-2" id="text-9">
</div>
<div id="outline-container-org471d5e2" class="outline-3">
<h3 id="org471d5e2"><span class="section-number-3">9.1</span> Notes</h3>
<div class="outline-text-3" id="text-9-1">
<ul class="org-ul">
<li>A good in-depth ML based paper</li>
<li>Could apply the same techniques to NTM</li>
<li>Could change the algorithm and apply on the same data</li>
</ul>
</div>
</div>
<div id="outline-container-org009a017" class="outline-3">
<h3 id="org009a017"><span class="section-number-3">9.2</span> FileName</h3>
<div class="outline-text-3" id="text-9-2">
<ul class="org-ul">
<li>Chen2019<sub>BeyondMultiDrugResistance.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org9e4cd7f" class="outline-3">
<h3 id="org9e4cd7f"><span class="section-number-3">9.3</span> Abstract</h3>
<div class="outline-text-3" id="text-9-3">
<ul class="org-ul">
<li>Whole genome sequencing of clinical Mycobacterium tuberculosis isolates
promises to circumvent the long wait times and limited scope of conventional
phenotypic antimicrobial susceptibility, but gaps remain for predicting
phenotype accurately from genotypic data especially for certain drugs.</li>

<li>We collected targeted or whole genome sequencing and conventional drug
resistance phenotyping data from 3601 Mycobacterium tuberculosis strains
enriched for resistance to first- and second-line drugs, with 1228 multidrug
resistant strains</li>

<li>We investigated the utility of (1) rare variants and variants known to be
determinants of resistance for at least one drug and (2) machine and
statistical learning architectures in predicting phenotypic drug resistance to
10 anti-tuberculosis drugs</li>
</ul>
</div>
</div>

<div id="outline-container-org3854dae" class="outline-3">
<h3 id="org3854dae"><span class="section-number-3">9.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-9-4">
<ul class="org-ul">
<li>Our approach allowed us to gain new insights into the relative im- portance of
specific genetic variants for resistance prediction</li>

<li>In summary, we present an exploration of machine learning and traditional
statistical models to identify the resistance of MTB isolates to 10
anti-tuberculosis drugs from whole genome sequencing data.</li>

<li>Our models trained on rare and common genetic variants achieved
state-of-the-art performance on a large, aggregated TB dataset, with
prediction times of less than a tenth of a second, demonstrating the efficacy
of our model as a diagnostic tool for MTB drug resistance.</li>

<li>The MD-WDNN represented the first multidrug model to our knowledge that</li>
</ul>
<p>
incorporated a high number of genotypic predictors known to be important to
determining resistance for one or more included drugs
</p>
</div>
</div>
</div>


<div id="outline-container-orgee9555a" class="outline-2">
<h2 id="orgee9555a"><span class="section-number-2">10</span> <span class="todo ___">[-]</span> Ching2018 - Opportunities and obstacles for deep learning in biology and medicine</h2>
<div class="outline-text-2" id="text-10">
</div>
<div id="outline-container-orgae9b29c" class="outline-3">
<h3 id="orgae9b29c"><span class="section-number-3">10.1</span> Notes</h3>
<div class="outline-text-3" id="text-10-1">
<ul class="org-ul">
<li>Found the code <a href="https://github.com/greenelab/deep-review">https://github.com/greenelab/deep-review</a></li>
<li>Can use this paper to build up the background for using DL in MTB and NTM analysis</li>
</ul>
</div>
</div>
<div id="outline-container-org7b6eb2e" class="outline-3">
<h3 id="org7b6eb2e"><span class="section-number-3">10.2</span> FileName</h3>
<div class="outline-text-3" id="text-10-2">
<ul class="org-ul">
<li>Ching2018<sub>OpportunitiesAndObstaclesForDLInBiology.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgc66a5ac" class="outline-3">
<h3 id="orgc66a5ac"><span class="section-number-3">10.3</span> Abstract</h3>
<div class="outline-text-3" id="text-10-3">
<ul class="org-ul">
<li>Following from an extensive literature review, we find that deep learning has
yet to revolutionize biomedicine or definitively resolve any of the most
pressing challenges in the field, but promising advances have been made on the
prior state of the art</li>
<li>The limited amount of labelled data for training presents problems in some
domains, as do legal and privacy constraints on work with sensitive health
records.</li>
</ul>
</div>
</div>

<div id="outline-container-org8cbc3a1" class="outline-3">
<h3 id="org8cbc3a1"><span class="section-number-3">10.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-10-4">
<ul class="org-ul">
<li>The most challenging tasks may be those in which predictions are used directly
for downstream model- ling or decision-making, especially in the clinic. As an
example, errors in sequence variant calling will be amplified if they are used
directly for genome-wide association studies.</li>

<li>In addition, the stochasticity and complexity of biological sys- tems imply
that for some problems, for instance, predicting gene regulation in disease,
perfect accuracy will be unattainable</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgd6e47e1" class="outline-2">
<h2 id="orgd6e47e1"><span class="section-number-2">11</span> <span class="todo STRT">STRT</span> Deelder2019 - Machine Learning Predicts Accurately Mycobacterium tuberculosis Drug Resistance From Whole Genome Sequencing Data</h2>
<div class="outline-text-2" id="text-11">
</div>
<div id="outline-container-org0d024ea" class="outline-3">
<h3 id="org0d024ea"><span class="section-number-3">11.1</span> Notes</h3>
<div class="outline-text-3" id="text-11-1">
<ul class="org-ul">
<li>Need to request code from authors</li>
<li>Good ML oriented paper</li>
</ul>
</div>
</div>
<div id="outline-container-orgb545f12" class="outline-3">
<h3 id="orgb545f12"><span class="section-number-3">11.2</span> FileName</h3>
<div class="outline-text-3" id="text-11-2">
<ul class="org-ul">
<li>Deelder2019<sub>MLPredictsAccurately.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org663044e" class="outline-3">
<h3 id="org663044e"><span class="section-number-3">11.3</span> Abstract</h3>
<div class="outline-text-3" id="text-11-3">
<ul class="org-ul">
<li>We applied machine learning approaches to 16,688 M. tuberculosis isolates that
have undergone WGS and laboratory drug-susceptibility testing (DST) across 14
antituberculosis drugs, with 22.5% of samples being multidrug resistant and
2.1% being extensively drug resistant.</li>

<li>We fitted separate models for each drug, with and without “co-occurrent
resistance” markers known to be causing resistance to drugs other than the one
of interest. Predictive performance was measured using sensitivity,
specificity, and the area under the receiver operating characteristic curve,
assuming DST results as the gold standard.</li>

<li>Overall, the gradient-boosted-tree models performed better than the
classification-tree models.</li>

<li>Our work demonstrates the utility of machine learning as a flexible approach
to drug resistance prediction that is able to accommodate a much larger number
of predictors and to summarize their predictive ability, thus assisting
clinical decision making and single nucleotide polymorphism detection in an
era of increasing WGS data generation.</li>
</ul>
</div>
</div>

<div id="outline-container-org2df82ce" class="outline-3">
<h3 id="org2df82ce"><span class="section-number-3">11.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-11-4">
<ul class="org-ul">
<li>The predictive performance was highest for resistance to first-line drugs,
and multidrug-resistant tuberculosis (area under the receiver operating characteristic curve above
96%), and lowest for third- line drugs (area under the curve below 85%).</li>
</ul>
</div>
</div>

<div id="outline-container-org1705e89" class="outline-3">
<h3 id="org1705e89"><span class="section-number-3">11.5</span> Supplementary</h3>
<div class="outline-text-3" id="text-11-5">
</div>
<div id="outline-container-orgcde97fe" class="outline-4">
<h4 id="orgcde97fe"><span class="section-number-4">11.5.1</span> Deelder2019<sub>MLPredictsAccurately</sub><sub>Data</sub><sub>Sheet</sub><sub>1.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-orge6ab3fd" class="outline-2">
<h2 id="orge6ab3fd"><span class="section-number-2">12</span> <span class="todo ___">[?]</span> Djout2016 - Predictive modeling targets thymidylate synthase ThyX in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-12">
</div>
<div id="outline-container-org061793b" class="outline-3">
<h3 id="org061793b"><span class="section-number-3">12.1</span> Notes</h3>
<div class="outline-text-3" id="text-12-1">
<ul class="org-ul">
<li>This requires more knowledge of chemistry and perhaps also relies on lab based experiments</li>
</ul>
</div>
</div>
<div id="outline-container-orgdcebba1" class="outline-3">
<h3 id="orgdcebba1"><span class="section-number-3">12.2</span> FileName</h3>
<div class="outline-text-3" id="text-12-2">
<ul class="org-ul">
<li>Djout2016<sub>PredictiveModeling.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgdbf7885" class="outline-3">
<h3 id="orgdbf7885"><span class="section-number-3">12.3</span> Abstract</h3>
<div class="outline-text-3" id="text-12-3">
<ul class="org-ul">
<li>ThyX is an essential thymidylate synthase</li>
</ul>
<p>
that is mechanistically and structurally unrelated to the human enzyme. DNA
gyrase is a DNA topoisomerase present in bacteria and plants but not animals.
The current study set out to understand the structure-activity relationships of
these targets in Mtb using a combination of cheminformatics and in vitro
screening.
</p>
</div>
</div>
<div id="outline-container-org980d021" class="outline-3">
<h3 id="org980d021"><span class="section-number-3">12.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org17328bd" class="outline-2">
<h2 id="org17328bd"><span class="section-number-2">13</span> <span class="todo STRT">STRT</span> Donohue2018 - Increasing nontuberculous mycobacteria reporting rates and species diversity identified in clinical laboratory reports</h2>
<div class="outline-text-2" id="text-13">
</div>
<div id="outline-container-org4a4d45a" class="outline-3">
<h3 id="org4a4d45a"><span class="section-number-3">13.1</span> Notes</h3>
<div class="outline-text-3" id="text-13-1">
<ul class="org-ul">
<li>This is a survey report about NTM across a couple decades</li>
<li>Could be used in building up the background context</li>
</ul>
</div>
</div>
<div id="outline-container-org679f37b" class="outline-3">
<h3 id="org679f37b"><span class="section-number-3">13.2</span> FileName</h3>
<div class="outline-text-3" id="text-13-2">
<ul class="org-ul">
<li>Donohue2018<sub>IncreasingNTMReportingRates.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orge3414e1" class="outline-3">
<h3 id="orge3414e1"><span class="section-number-3">13.3</span> Abstract</h3>
<div class="outline-text-3" id="text-13-3">
<ul class="org-ul">
<li>Changes in NTM diversity were observed in complex/groups known to be
clinically significant. Between 1994 and 2014 the rate implicating M.
abscesses-chelonae group and M. avium complex increased by 322 and 149%,
respectively.</li>
</ul>
</div>
</div>
<div id="outline-container-org7625801" class="outline-3">
<h3 id="org7625801"><span class="section-number-3">13.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-13-4">
<ul class="org-ul">
<li>Knowledge of the NTM species most frequently associ- ated with adverse human
health consequences will assist epidemiological investigators to identify
likely sources and modes of exposure associated with these types of in-
fections.</li>
<li>As the U.S. population ages, the public health burden from NTM associated
disorders is expected to increase.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgbcef9f9" class="outline-2">
<h2 id="orgbcef9f9"><span class="section-number-2">14</span> <span class="todo ___">[-]</span> Doshi2017 - Tuberculosis control, and the where and why of artificial intelligence</h2>
<div class="outline-text-2" id="text-14">
</div>
<div id="outline-container-org86f3828" class="outline-3">
<h3 id="org86f3828"><span class="section-number-3">14.1</span> Notes</h3>
<div class="outline-text-3" id="text-14-1">
<ul class="org-ul">
<li>Survey paper</li>
<li>Can be used for building background context</li>
</ul>
</div>
</div>
<div id="outline-container-orgc9907ed" class="outline-3">
<h3 id="orgc9907ed"><span class="section-number-3">14.2</span> FileName</h3>
<div class="outline-text-3" id="text-14-2">
<ul class="org-ul">
<li>Doshi2017<sub>TuberculosisControl.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org96abdcd" class="outline-3">
<h3 id="org96abdcd"><span class="section-number-3">14.3</span> Abstract</h3>
</div>
<div id="outline-container-org8fc9219" class="outline-3">
<h3 id="org8fc9219"><span class="section-number-3">14.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgb1a05fb" class="outline-2">
<h2 id="orgb1a05fb"><span class="section-number-2">15</span> <span class="done _X_">[X]</span> Ekins2013 - Enhancing Hit Identification in Mycobacterium tuberculosis Drug Discovery Using Validated Dual-Event Bayesian Models</h2>
<div class="outline-text-2" id="text-15">
</div>
<div id="outline-container-org883a1dd" class="outline-3">
<h3 id="org883a1dd"><span class="section-number-3">15.1</span> Notes</h3>
<div class="outline-text-3" id="text-15-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-orgbfb3e6b" class="outline-3">
<h3 id="orgbfb3e6b"><span class="section-number-3">15.2</span> FileName</h3>
<div class="outline-text-3" id="text-15-2">
<ul class="org-ul">
<li>Ekins2013<sub>EnhancingHitIdentification.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orga9c6026" class="outline-3">
<h3 id="orga9c6026"><span class="section-number-3">15.3</span> Abstract</h3>
<div class="outline-text-3" id="text-15-3">
<ul class="org-ul">
<li>High-throughput screening (HTS) in whole cells is widely pursued to find
compounds active against Mycobacterium tuberculosis (Mtb) for further
development towards new tuberculosis (TB) drugs</li>
<li>We have recently built and validated Bayesian machine learning models for
predicting compounds with activity against Mtb based on publicly available
large-scale HTS data from the Tuberculosis Antimicrobial Acquisition
Coordinating Facility</li>
</ul>
</div>
</div>
<div id="outline-container-org5b83b87" class="outline-3">
<h3 id="org5b83b87"><span class="section-number-3">15.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org7717792" class="outline-2">
<h2 id="org7717792"><span class="section-number-2">16</span> <span class="done _X_">[X]</span> Ekins2013 - Fusing Dual-Event Data Sets for Mycobacterium tuberculosis Machine Learning Models and Their Evaluation</h2>
<div class="outline-text-2" id="text-16">
</div>
<div id="outline-container-org65e674b" class="outline-3">
<h3 id="org65e674b"><span class="section-number-3">16.1</span> Notes</h3>
<div class="outline-text-3" id="text-16-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org6e97aba" class="outline-3">
<h3 id="org6e97aba"><span class="section-number-3">16.2</span> FileName</h3>
<div class="outline-text-3" id="text-16-2">
<ul class="org-ul">
<li>Ekins2013<sub>FusingDualEvent.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgb843ce5" class="outline-3">
<h3 id="orgb843ce5"><span class="section-number-3">16.3</span> Abstract</h3>
<div class="outline-text-3" id="text-16-3">
<ul class="org-ul">
<li>A cheminformatics data-fusion</li>
</ul>
<p>
approach followed by Bayesian machine learning, Support Vector Machine, or Recursive
Partitioning model development (based on publicly available Mtb screening data) was used to
compare individual data sets and subsequent combined models.
</p>
<ul class="org-ul">
<li>We demonstrate that</li>
</ul>
<p>
combining three data sets incorporating antitubercular and cytotoxicity data in Vero cells
from our previous screens results in external validation receiver operator curve (ROC) of
0.83 (Bayesian or RP Forest)
</p>
</div>
</div>
<div id="outline-container-org16e989f" class="outline-3">
<h3 id="org16e989f"><span class="section-number-3">16.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org8d5b41f" class="outline-2">
<h2 id="org8d5b41f"><span class="section-number-2">17</span> <span class="done _X_">[X]</span> Ekins2014<sub>AreBiggerDataSets.pdf</sub></h2>
<div class="outline-text-2" id="text-17">
</div>
<div id="outline-container-org91d56d0" class="outline-3">
<h3 id="org91d56d0"><span class="section-number-3">17.1</span> Notes</h3>
<div class="outline-text-3" id="text-17-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org8b14c08" class="outline-3">
<h3 id="org8b14c08"><span class="section-number-3">17.2</span> Title</h3>
</div>
<div id="outline-container-org177208c" class="outline-3">
<h3 id="org177208c"><span class="section-number-3">17.3</span> Abstract</h3>
</div>
<div id="outline-container-orgcfdae93" class="outline-3">
<h3 id="orgcfdae93"><span class="section-number-3">17.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org559393c" class="outline-2">
<h2 id="org559393c"><span class="section-number-2">18</span> <span class="done _X_">[X]</span> Ekins2014<sub>CombiningComputationalMethods.pdf</sub></h2>
<div class="outline-text-2" id="text-18">
</div>
<div id="outline-container-org6145e2a" class="outline-3">
<h3 id="org6145e2a"><span class="section-number-3">18.1</span> Notes</h3>
<div class="outline-text-3" id="text-18-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-orga4118cd" class="outline-3">
<h3 id="orga4118cd"><span class="section-number-3">18.2</span> Title</h3>
</div>
<div id="outline-container-org5118973" class="outline-3">
<h3 id="org5118973"><span class="section-number-3">18.3</span> Abstract</h3>
</div>
<div id="outline-container-orgf04a27f" class="outline-3">
<h3 id="orgf04a27f"><span class="section-number-3">18.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgf4c9010" class="outline-2">
<h2 id="orgf4c9010"><span class="section-number-2">19</span> <span class="done _X_">[X]</span> Ekins2016<sub>MachineLearningModel.pdf</sub></h2>
<div class="outline-text-2" id="text-19">
</div>
<div id="outline-container-org2889971" class="outline-3">
<h3 id="org2889971"><span class="section-number-3">19.1</span> Notes</h3>
<div class="outline-text-3" id="text-19-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-orgfb532cd" class="outline-3">
<h3 id="orgfb532cd"><span class="section-number-3">19.2</span> Title</h3>
</div>
<div id="outline-container-orgd1604d7" class="outline-3">
<h3 id="orgd1604d7"><span class="section-number-3">19.3</span> Abstract</h3>
</div>
<div id="outline-container-orgf02f0e3" class="outline-3">
<h3 id="orgf02f0e3"><span class="section-number-3">19.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org877fe2d" class="outline-2">
<h2 id="org877fe2d"><span class="section-number-2">20</span> <span class="done _X_">[X]</span> Ekins2017<sub>MachineLearning.pdf</sub></h2>
<div class="outline-text-2" id="text-20">
</div>
<div id="outline-container-orgf04d498" class="outline-3">
<h3 id="orgf04d498"><span class="section-number-3">20.1</span> Notes</h3>
<div class="outline-text-3" id="text-20-1">
<ul class="org-ul">
<li>Requires too much domain knowledge</li>
<li>Perhaps uses an older technique i.e. HTS for analysis</li>
<li>Could use other algorithms on the data used in this study</li>
</ul>
</div>
</div>
<div id="outline-container-org1858d12" class="outline-3">
<h3 id="org1858d12"><span class="section-number-3">20.2</span> Title</h3>
</div>
<div id="outline-container-org4ec503f" class="outline-3">
<h3 id="org4ec503f"><span class="section-number-3">20.3</span> Abstract</h3>
</div>
<div id="outline-container-org3fcc27d" class="outline-3">
<h3 id="org3fcc27d"><span class="section-number-3">20.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgd8c8050" class="outline-2">
<h2 id="orgd8c8050"><span class="section-number-2">21</span> <span class="todo ___">[-]</span> Engle2019 - Performance of Qure.ai automatic classifiers against a large annotated database of patients with diverse forms of tuberculosis</h2>
<div class="outline-text-2" id="text-21">
</div>
<div id="outline-container-orgae5efe7" class="outline-3">
<h3 id="orgae5efe7"><span class="section-number-3">21.1</span> Notes</h3>
<div class="outline-text-3" id="text-21-1">
<ul class="org-ul">
<li>Image processing</li>
<li>Mentiones the database source, NIH</li>
<li>Has CSV as supplementary data</li>
</ul>
</div>
</div>
<div id="outline-container-org87fc61a" class="outline-3">
<h3 id="org87fc61a"><span class="section-number-3">21.2</span> FileName</h3>
<div class="outline-text-3" id="text-21-2">
<ul class="org-ul">
<li>Engle2019<sub>PerformanceOfQureAI.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org54bfc94" class="outline-3">
<h3 id="org54bfc94"><span class="section-number-3">21.3</span> Abstract</h3>
<div class="outline-text-3" id="text-21-3">
<ul class="org-ul">
<li>Availability of trained radiologists for fast processing of CXRs in regions
burdened with tuberculosis always has been a challenge, affecting both timely
diagnosis and patient monitoring.</li>

<li>The Qure.ai qXR arti- ficial intelligence automated CXR interpretation tool,
was blind-tested on the 346 radiologist- annotated CXRs from the TBPP
database.</li>

<li>We conclude that applying fully automated Qure.ai CXR analysis tool is useful
for fast, accurate, uniform, large-scale CXR annotation assistance, as it
performed well even for DR-TB cases that were not used for initial training.</li>
</ul>
</div>
</div>

<div id="outline-container-orgb6af2a9" class="outline-3">
<h3 id="orgb6af2a9"><span class="section-number-3">21.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-21-4">
<ul class="org-ul">
<li>We demonstrated that the Qure.ai qXR CXR annotations for cavity, nodule,
pleural effusion, and hilar lymphadenopathy, are in statistical agreement with
radiologist CXR annotations.</li>

<li>A corresponding analysis of patient case matched
CT image annotations recorded within 30 days of a CXR supports this result,
demonstrating statistical significance for cavity, pleural effusion, hilar
lymphadenopathy, and atelectasis.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgfcc5de8" class="outline-2">
<h2 id="orgfcc5de8"><span class="section-number-2">22</span> <span class="todo ___">[-]</span> Greif2020 - Species Distribution and Isolation Frequency of Nontuberculous Mycobacteria, Uruguay</h2>
<div class="outline-text-2" id="text-22">
</div>
<div id="outline-container-org4fa3c68" class="outline-3">
<h3 id="org4fa3c68"><span class="section-number-3">22.1</span> Notes</h3>
<div class="outline-text-3" id="text-22-1">
<ul class="org-ul">
<li>Survey paper</li>
</ul>
</div>
</div>
<div id="outline-container-org166e779" class="outline-3">
<h3 id="org166e779"><span class="section-number-3">22.2</span> FileName</h3>
<div class="outline-text-3" id="text-22-2">
<ul class="org-ul">
<li>Greif2020<sub>FrequencyOfNTM.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org55b5d4f" class="outline-3">
<h3 id="org55b5d4f"><span class="section-number-3">22.3</span> Abstract</h3>
<div class="outline-text-3" id="text-22-3">
<ul class="org-ul">
<li>Nontuberculous mycobacteria (NTM) increasingly are recognized as
opportunistic pathogens of humans.</li>
<li>NTM species distribution is well documented in Europe and North America</li>
</ul>
</div>
</div>
<div id="outline-container-orgb60faf1" class="outline-3">
<h3 id="orgb60faf1"><span class="section-number-3">22.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-22-4">
<ul class="org-ul">
<li>Our findings indicate that rate of NTM have risen nearly 5-fold (4.79) in
Uruguay, from 0.33 cas- es/100,000 inhabitants in 2006 to 1.57 cases/100,000
inhabitants in 2018.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgede7fa3" class="outline-2">
<h2 id="orgede7fa3"><span class="section-number-2">23</span> <span class="todo ___">[-]</span> Hicks2019 - Evaluation of parameters affecting performance and reliability of machine learning-based antibiotic susceptibility testing from whole genome sequencing data</h2>
<div class="outline-text-2" id="text-23">
</div>
<div id="outline-container-orgce3ce73" class="outline-3">
<h3 id="orgce3ce73"><span class="section-number-3">23.1</span> Notes</h3>
<div class="outline-text-3" id="text-23-1">
<ul class="org-ul">
<li>Can perform the mentioned analysis on the MTB and NTM data</li>
<li>The paper focuses on the factors affecting the application and clinical application of ML models</li>
</ul>
</div>
</div>
<div id="outline-container-org5b46cc2" class="outline-3">
<h3 id="org5b46cc2"><span class="section-number-3">23.2</span> FileName</h3>
<div class="outline-text-3" id="text-23-2">
<ul class="org-ul">
<li>Hicks2019<sub>EvaluationOfParameters.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org0eb38d1" class="outline-3">
<h3 id="org0eb38d1"><span class="section-number-3">23.3</span> Abstract</h3>
<div class="outline-text-3" id="text-23-3">
<ul class="org-ul">
<li>There has been no systematic evaluation of factors that may influence
performance of ML models, how they might apply to and vary across clinical
populations, and what the implications might be in the clinical setting</li>

<li>We performed a meta-analysis of seven large Neisseria gonorrhoeae datasets, as
well as Klebsiella pneumoniae and Acinetobacter baumannii datasets, with whole
genome sequence data and antibiotic susceptibility phenotypes using set
covering machine classifi- cation, random forest classification, and random
forest regression models to predict resis- tance phenotypes from genotype.</li>

<li>We demonstrate how model performance varies by drug, dataset, resistance
metric, and species, reflecting the complexities of generating clinically
relevant conclusions from machine learning-derived models.</li>

<li>Our results highlight significant biological and technical challenges
facing the application of machine learning-based prediction of antibiotic
resistance as a diagnostic tool.</li>
</ul>
</div>
</div>

<div id="outline-container-org29c94dd" class="outline-3">
<h3 id="org29c94dd"><span class="section-number-3">23.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgc44d895" class="outline-2">
<h2 id="orgc44d895"><span class="section-number-2">24</span> <span class="todo STRT">STRT</span> Hussain2018 - Predicting treatment outcome of drug-susceptible tuberculosis patients using machine-learning models</h2>
<div class="outline-text-2" id="text-24">
</div>
<div id="outline-container-orgf971921" class="outline-3">
<h3 id="orgf971921"><span class="section-number-3">24.1</span> Notes</h3>
<div class="outline-text-3" id="text-24-1">
<ul class="org-ul">
<li>Can use a different algorithm (XgBoost) on the dataset</li>
<li>Sample data and complete code is shared
<a href="https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics">https://github.com/biosharp-dotnet/tuberculosis-predictive-analytics</a></li>
<li>Need to find the full dataset of 4000 surveyed patients</li>
</ul>
</div>
</div>
<div id="outline-container-org87337eb" class="outline-3">
<h3 id="org87337eb"><span class="section-number-3">24.2</span> FileName</h3>
<div class="outline-text-3" id="text-24-2">
<ul class="org-ul">
<li>Hussain2018<sub>PredictingTreatmentOutcomes.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-org554b93a" class="outline-3">
<h3 id="org554b93a"><span class="section-number-3">24.3</span> Abstract</h3>
<div class="outline-text-3" id="text-24-3">
<ul class="org-ul">
<li>Tuberculosis (TB) is a deadly contagious disease and a serious global health
problem. It is curable but due to its lengthy treatment process, a patient is
likely to leave the treatment incomplete, leading to a more lethal, drug
resistant form of disease.</li>

<li>We aim to facilitate TB programs by predicting the outcome of the treatment of
a particular patient at the start of treatment so that their health workers
can be utilized in a targeted and cost-effective way.</li>

<li>The problem was modeled as a classification problem, and the outcome of
treatment was predicted using state-of-art implementations of 3 machine learning algorithms</li>
</ul>
</div>
</div>

<div id="outline-container-orgcaa3ee5" class="outline-3">
<h3 id="orgcaa3ee5"><span class="section-number-3">24.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-24-4">
<ul class="org-ul">
<li>Empirical results using data of more than 4000 patients reveal that we can
predict with more than 76% accuracy whether a TB patient would complete his (or her) treatment or not</li>
<li></li>
</ul>
</div>
</div>
</div>
<div id="outline-container-org3a22f80" class="outline-2">
<h2 id="org3a22f80"><span class="section-number-2">25</span> <span class="todo STRT">STRT</span> Jamal2020 - Artificial Intelligence and Machine learning based prediction of resistant and susceptible mutations in Mycobacterium tuberculosis</h2>
<div class="outline-text-2" id="text-25">
</div>
<div id="outline-container-org42001db" class="outline-3">
<h3 id="org42001db"><span class="section-number-3">25.1</span> Notes</h3>
<div class="outline-text-3" id="text-25-1">
<ul class="org-ul">
<li>Used many ML algorithms, 85% accuracy</li>
<li>Could use XgBoost</li>
<li>Could conduct a similar study for NTM</li>
<li>Mentions many online bio databases</li>
</ul>
</div>
</div>
<div id="outline-container-org00d8e33" class="outline-3">
<h3 id="org00d8e33"><span class="section-number-3">25.2</span> FileName</h3>
<div class="outline-text-3" id="text-25-2">
<ul class="org-ul">
<li>Jamal2020<sub>AIAndMLBasedPredictive.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgb44d2f2" class="outline-3">
<h3 id="orgb44d2f2"><span class="section-number-3">25.3</span> Abstract</h3>
<div class="outline-text-3" id="text-25-3">
<ul class="org-ul">
<li>The present work is a computational framework that uses artificial
intelligence (AI) based machine learning (ML) approaches for predicting
resistance in the genes rpoB, inhA, katG, pncA, gyrA and gyrB for the drugs
rifampicin, isoniazid, pyrazinamide and fluoroquinolones</li>

<li>The single nucleotide variations were represented by several sequence and
structural features that indicate the influence of mutations on the target
protein coded by each gene.</li>
</ul>
</div>
</div>

<div id="outline-container-org977d047" class="outline-3">
<h3 id="org977d047"><span class="section-number-3">25.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-25-4">
<ul class="org-ul">
<li>The present study proposes an AI/ML method to classify resistant and susceptible
mutations in TB and predict novel resistance conferring mutations.</li>

<li>The impact of the reported mutations was captured in the form of changes in the amino acid
residues, and the consequent change in properties vis-a-vis wild-type and mutant
proteins and represented as features used to train the models.</li>

<li>The classification model was generated for each gene and predictions were made for
SNVs linked with each gene for each drug</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-org0a3d79b" class="outline-2">
<h2 id="org0a3d79b"><span class="section-number-2">26</span> <span class="todo STRT">STRT</span> Kavvas2018 - Machine learning and structural analysis of Mycobacterium tuberculosis pan-genome identifies genetic signatures of antibiotic resistance</h2>
<div class="outline-text-2" id="text-26">
</div>
<div id="outline-container-org3c3b766" class="outline-3">
<h3 id="org3c3b766"><span class="section-number-3">26.1</span> Notes</h3>
<div class="outline-text-3" id="text-26-1">
<ul class="org-ul">
<li>They developed and online platform</li>

<li>Can be applied to NTM</li>

<li>Code is available
<a href="https://github.com/biosharp-dotnet/microbial_AMR_ML">https://github.com/biosharp-dotnet/microbial_AMR_ML</a></li>
</ul>
</div>
</div>
<div id="outline-container-org2d63eb5" class="outline-3">
<h3 id="org2d63eb5"><span class="section-number-3">26.2</span> FileName</h3>
<div class="outline-text-3" id="text-26-2">
<ul class="org-ul">
<li>Kavvas2018<sub>MachineLearning</sub><sub>.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgbe3fb79" class="outline-3">
<h3 id="orgbe3fb79"><span class="section-number-3">26.3</span> Abstract</h3>
<div class="outline-text-3" id="text-26-3">
<ul class="org-ul">
<li>The many publicly available datasets describing its AMR characteristics demand
disparate data-type analyses.</li>

<li>Here, we develop a reference strain-agnostic computational platform that uses
machine learning approaches, complemented by both genetic interaction analysis
and 3D structural mutation-mapping, to identify signatures of AMR evolution to
13 antibiotics.</li>

<li>First, a pan-genome analysis shows that M. tuberculosis is highly conserved
with sequenced variation concentrated in PE/PPE/PGRS genes.</li>

<li>Second, the platform corroborates 33 genes known to confer resistance and identifies 24
new genetic signatures of AMR.</li>

<li>Third, 97 epistatic interactions across 10 resistance classes are revealed.</li>

<li>Fourth, detailed structural analysis of these
genes yields mechanistic bases for their selection.</li>

<li>The platform can be used to study other human pathogens</li>
</ul>
</div>
</div>

<div id="outline-container-org6732594" class="outline-3">
<h3 id="org6732594"><span class="section-number-3">26.4</span> Conclusion</h3>
<div class="outline-text-3" id="text-26-4">
<ul class="org-ul">
<li>The platform was applied to 1595 sequenced strains to yield results in four
categories: pan-genome properties, identification of genes conferring
antibiotic resistance, their epistatic interactions, and protein structure
based mechanistic insights.</li>
</ul>
</div>
</div>
</div>

<div id="outline-container-orgd865f61" class="outline-2">
<h2 id="orgd865f61"><span class="section-number-2">27</span> Khanna2018 - Ensemble Technique for Prediction of T-cell Mycobacterium tuberculosis Epitopes</h2>
<div class="outline-text-2" id="text-27">
</div>
<div id="outline-container-org124d613" class="outline-3">
<h3 id="org124d613"><span class="section-number-3">27.1</span> Notes</h3>
<div class="outline-text-3" id="text-27-1">
<ul class="org-ul">
<li></li>
</ul>
</div>
</div>
<div id="outline-container-org945d563" class="outline-3">
<h3 id="org945d563"><span class="section-number-3">27.2</span> FileName</h3>
<div class="outline-text-3" id="text-27-2">
<ul class="org-ul">
<li>Khanna2018<sub>EnsembleTechnique.pdf</sub></li>
</ul>
</div>
</div>
<div id="outline-container-orgd651ccf" class="outline-3">
<h3 id="orgd651ccf"><span class="section-number-3">27.3</span> Abstract</h3>
<div class="outline-text-3" id="text-27-3">
<ul class="org-ul">
<li>Development of an effective machine-learning model for T-cell Mycobacterium
tuberculosis (M. tuberculosis) epitopes is beneficial for saving biologist’s
time and effort for identifying epitope in a targeted antigen.</li>

<li>The proposed method is able to work with variable length epitopes having size</li>
</ul>
<p>
even greater than 9-mers. Identification of T-cell or B-cell epitopes in the
targeted antigen is the main goal in designing epitope-based vaccine,
immune-diagnostic tests and antibody production
</p>
</div>
</div>

<div id="outline-container-org30646fd" class="outline-3">
<h3 id="org30646fd"><span class="section-number-3">27.4</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgc97b276" class="outline-2">
<h2 id="orgc97b276"><span class="section-number-2">28</span> Killian2019<sub>LearningToPrescribe.pdf</sub></h2>
<div class="outline-text-2" id="text-28">
</div>
<div id="outline-container-org1d78b5e" class="outline-3">
<h3 id="org1d78b5e"><span class="section-number-3">28.1</span> Title</h3>
</div>
<div id="outline-container-org7496bb9" class="outline-3">
<h3 id="org7496bb9"><span class="section-number-3">28.2</span> Abstract</h3>
</div>
<div id="outline-container-orgc6c0bba" class="outline-3">
<h3 id="orgc6c0bba"><span class="section-number-3">28.3</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgb12fe76" class="outline-2">
<h2 id="orgb12fe76"><span class="section-number-2">29</span> Kouchaki2019<sub>ApplicationOfML.pdf</sub></h2>
<div class="outline-text-2" id="text-29">
</div>
<div id="outline-container-orgd23d957" class="outline-3">
<h3 id="orgd23d957"><span class="section-number-3">29.1</span> Title</h3>
</div>
<div id="outline-container-orgc546144" class="outline-3">
<h3 id="orgc546144"><span class="section-number-3">29.2</span> Abstract</h3>
</div>
<div id="outline-container-org95ff3e8" class="outline-3">
<h3 id="org95ff3e8"><span class="section-number-3">29.3</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org2f53f70" class="outline-2">
<h2 id="org2f53f70"><span class="section-number-2">30</span> Kouchaki2020<sub>MultiLabelRandomForest.pdf</sub></h2>
<div class="outline-text-2" id="text-30">
</div>
<div id="outline-container-org90a5f65" class="outline-3">
<h3 id="org90a5f65"><span class="section-number-3">30.1</span> Title</h3>
</div>
<div id="outline-container-orgc0611c6" class="outline-3">
<h3 id="orgc0611c6"><span class="section-number-3">30.2</span> Abstract</h3>
</div>
<div id="outline-container-org363e209" class="outline-3">
<h3 id="org363e209"><span class="section-number-3">30.3</span> Conclusion</h3>
</div>
<div id="outline-container-orgdd48189" class="outline-3">
<h3 id="orgdd48189"><span class="section-number-3">30.4</span> Supplementary</h3>
<div class="outline-text-3" id="text-30-4">
</div>
<div id="outline-container-orgfb24d75" class="outline-4">
<h4 id="orgfb24d75"><span class="section-number-4">30.4.1</span> Kouchaki2020<sub>MultiLabel.zip</sub></h4>
</div>
<div id="outline-container-org59cc48e" class="outline-4">
<h4 id="org59cc48e"><span class="section-number-4">30.4.2</span> Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>1.pdf</sub></h4>
</div>
<div id="outline-container-orgdd3806b" class="outline-4">
<h4 id="orgdd3806b"><span class="section-number-4">30.4.3</span> Kouchaki2020<sub>MultiLabel</sub><sub>DataSheet</sub><sub>2.pdf</sub></h4>
</div>
<div id="outline-container-orgabe2b5a" class="outline-4">
<h4 id="orgabe2b5a"><span class="section-number-4">30.4.4</span> Code</h4>
<div class="outline-text-4" id="text-30-4-4">
<p>
Kouchaki2020<sub>MultiLabelSLRF</sub>/checkmutations.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/common<sub>functions.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/cvcodes.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/demo-mtbmlrf.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/ensemble<sub>learning</sub><sub>multilabel.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakefeat.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/fakepheno.csv
Kouchaki2020<sub>MultiLabelSLRF</sub>/feature<sub>extraction.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/load-dataset.py
Kouchaki2020<sub>MultiLabelSLRF</sub>/mutation<sub>ranking.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/print<sub>stats.py</sub>
Kouchaki2020<sub>MultiLabelSLRF</sub>/readme
Kouchaki2020<sub>MultiLabelSLRF</sub>/treeplot.py
</p>
</div>
</div>
</div>
</div>

<div id="outline-container-org7b87ad4" class="outline-2">
<h2 id="org7b87ad4"><span class="section-number-2">31</span> Kovalishyn2019<sub>HybridDesign.pdf</sub></h2>
<div class="outline-text-2" id="text-31">
</div>
<div id="outline-container-org453a261" class="outline-3">
<h3 id="org453a261"><span class="section-number-3">31.1</span> Title</h3>
</div>
<div id="outline-container-orgf828b0a" class="outline-3">
<h3 id="orgf828b0a"><span class="section-number-3">31.2</span> Abstract</h3>
</div>
<div id="outline-container-org08facb9" class="outline-3">
<h3 id="org08facb9"><span class="section-number-3">31.3</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org2162fbc" class="outline-2">
<h2 id="org2162fbc"><span class="section-number-2">32</span> Kumari2019<sub>AGeneticProgramming.pdf</sub></h2>
<div class="outline-text-2" id="text-32">
</div>
<div id="outline-container-org200f695" class="outline-3">
<h3 id="org200f695"><span class="section-number-3">32.1</span> Title</h3>
</div>
<div id="outline-container-orgd9cd074" class="outline-3">
<h3 id="orgd9cd074"><span class="section-number-3">32.2</span> Abstract</h3>
</div>
<div id="outline-container-org8ee76b4" class="outline-3">
<h3 id="org8ee76b4"><span class="section-number-3">32.3</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org1943dbf" class="outline-2">
<h2 id="org1943dbf"><span class="section-number-2">33</span> Lai2019<sub>ComparisonOfThePredictiveOutcomes.pdf</sub></h2>
<div class="outline-text-2" id="text-33">
</div>
<div id="outline-container-org8ebb60b" class="outline-3">
<h3 id="org8ebb60b"><span class="section-number-3">33.1</span> Abstract</h3>
</div>
<div id="outline-container-org28329fb" class="outline-3">
<h3 id="org28329fb"><span class="section-number-3">33.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgc6cdfed" class="outline-2">
<h2 id="orgc6cdfed"><span class="section-number-2">34</span> Lane2018<sub>ComparingAndValidating.pdf</sub></h2>
<div class="outline-text-2" id="text-34">
</div>
<div id="outline-container-orgae6002e" class="outline-3">
<h3 id="orgae6002e"><span class="section-number-3">34.1</span> Abstract</h3>
</div>
<div id="outline-container-org9a3501c" class="outline-3">
<h3 id="org9a3501c"><span class="section-number-3">34.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgbe70cc1" class="outline-2">
<h2 id="orgbe70cc1"><span class="section-number-2">35</span> Lin2001<sub>ConservedCodonComposition.pdf</sub></h2>
<div class="outline-text-2" id="text-35">
</div>
<div id="outline-container-orgd95c486" class="outline-3">
<h3 id="orgd95c486"><span class="section-number-3">35.1</span> Abstract</h3>
</div>
<div id="outline-container-orge116cb7" class="outline-3">
<h3 id="orge116cb7"><span class="section-number-3">35.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org082ed14" class="outline-2">
<h2 id="org082ed14"><span class="section-number-2">36</span> LokeshKumar2019<sub>PredictionOfMultiDrugResistant.pdf</sub></h2>
<div class="outline-text-2" id="text-36">
</div>
<div id="outline-container-org9b8e95b" class="outline-3">
<h3 id="org9b8e95b"><span class="section-number-3">36.1</span> Abstract</h3>
</div>
<div id="outline-container-orge1f03f7" class="outline-3">
<h3 id="orge1f03f7"><span class="section-number-3">36.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgdefaecb" class="outline-2">
<h2 id="orgdefaecb"><span class="section-number-2">37</span> Mahe2019<sub>ALargeScaleEvaluationTBProfiler.pdf</sub></h2>
<div class="outline-text-2" id="text-37">
</div>
<div id="outline-container-org81406fa" class="outline-3">
<h3 id="org81406fa"><span class="section-number-3">37.1</span> Abstract</h3>
</div>
<div id="outline-container-org50ca4a9" class="outline-3">
<h3 id="org50ca4a9"><span class="section-number-3">37.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgba0ea42" class="outline-2">
<h2 id="orgba0ea42"><span class="section-number-2">38</span> Marino2016<sub>ComputationalAndEmpirical.pdf</sub></h2>
<div class="outline-text-2" id="text-38">
</div>
<div id="outline-container-org35d9381" class="outline-3">
<h3 id="org35d9381"><span class="section-number-3">38.1</span> Abstract</h3>
</div>
<div id="outline-container-orga7f15c7" class="outline-3">
<h3 id="orga7f15c7"><span class="section-number-3">38.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org28ac74b" class="outline-2">
<h2 id="org28ac74b"><span class="section-number-2">39</span> Marques2019<sub>PulmonaryNTMInfections.pdf</sub></h2>
<div class="outline-text-2" id="text-39">
</div>
<div id="outline-container-org6b562a9" class="outline-3">
<h3 id="org6b562a9"><span class="section-number-3">39.1</span> Abstract</h3>
</div>
<div id="outline-container-orgd7c434b" class="outline-3">
<h3 id="orgd7c434b"><span class="section-number-3">39.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org0446bbb" class="outline-2">
<h2 id="org0446bbb"><span class="section-number-2">40</span> Meldenez2015<sub>OnCombiningMultiple.pdf</sub></h2>
<div class="outline-text-2" id="text-40">
</div>
<div id="outline-container-org46e872c" class="outline-3">
<h3 id="org46e872c"><span class="section-number-3">40.1</span> Abstract</h3>
</div>
<div id="outline-container-org463c2b6" class="outline-3">
<h3 id="org463c2b6"><span class="section-number-3">40.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org4ebc3d1" class="outline-2">
<h2 id="org4ebc3d1"><span class="section-number-2">41</span> Mikusova2016<sub>LearningFromThePast.pdf</sub></h2>
<div class="outline-text-2" id="text-41">
</div>
<div id="outline-container-org5c0630c" class="outline-3">
<h3 id="org5c0630c"><span class="section-number-3">41.1</span> Abstract</h3>
</div>
<div id="outline-container-org9287af9" class="outline-3">
<h3 id="org9287af9"><span class="section-number-3">41.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgc2dbaf1" class="outline-2">
<h2 id="orgc2dbaf1"><span class="section-number-2">42</span> Ngo2019<sub>GenomicPrediction.pdf</sub></h2>
<div class="outline-text-2" id="text-42">
</div>
<div id="outline-container-org3fef9ca" class="outline-3">
<h3 id="org3fef9ca"><span class="section-number-3">42.1</span> Abstract</h3>
</div>
<div id="outline-container-org3714ca4" class="outline-3">
<h3 id="org3714ca4"><span class="section-number-3">42.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org8e5b7e6" class="outline-2">
<h2 id="org8e5b7e6"><span class="section-number-2">43</span> Periwal2011<sub>PredictiveModels.pdf</sub></h2>
<div class="outline-text-2" id="text-43">
</div>
<div id="outline-container-orgbe53ccc" class="outline-3">
<h3 id="orgbe53ccc"><span class="section-number-3">43.1</span> Abstract</h3>
</div>
<div id="outline-container-org265a117" class="outline-3">
<h3 id="org265a117"><span class="section-number-3">43.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org4fb0d36" class="outline-2">
<h2 id="org4fb0d36"><span class="section-number-2">44</span> Ratnatunga2020<sub>RiseOfNTMLungDisease.pdf</sub></h2>
<div class="outline-text-2" id="text-44">
</div>
<div id="outline-container-org891293c" class="outline-3">
<h3 id="org891293c"><span class="section-number-3">44.1</span> Abstract</h3>
</div>
<div id="outline-container-orgf8573f2" class="outline-3">
<h3 id="orgf8573f2"><span class="section-number-3">44.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org8039d2a" class="outline-2">
<h2 id="org8039d2a"><span class="section-number-2">45</span> Romero2020<sub>DecisionTree.pdf</sub></h2>
<div class="outline-text-2" id="text-45">
</div>
<div id="outline-container-orga922e4a" class="outline-3">
<h3 id="orga922e4a"><span class="section-number-3">45.1</span> Abstract</h3>
</div>
<div id="outline-container-org7045fa2" class="outline-3">
<h3 id="org7045fa2"><span class="section-number-3">45.2</span> Conclusion</h3>
</div>
<div id="outline-container-orgdf19664" class="outline-3">
<h3 id="orgdf19664"><span class="section-number-3">45.3</span> Supplementary</h3>
<div class="outline-text-3" id="text-45-3">
</div>
<div id="outline-container-org92ced8e" class="outline-4">
<h4 id="org92ced8e"><span class="section-number-4">45.3.1</span> Romero2020<sub>DecisionTree</sub><sub>Corrigendum.pdf</sub></h4>
</div>
</div>
</div>
<div id="outline-container-org9b71e24" class="outline-2">
<h2 id="org9b71e24"><span class="section-number-2">46</span> Sauer2018<sub>FeatureSelection.pdf</sub></h2>
<div class="outline-text-2" id="text-46">
</div>
<div id="outline-container-org469e39a" class="outline-3">
<h3 id="org469e39a"><span class="section-number-3">46.1</span> Abstract</h3>
</div>
<div id="outline-container-org5c57fea" class="outline-3">
<h3 id="org5c57fea"><span class="section-number-3">46.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org4b1a247" class="outline-2">
<h2 id="org4b1a247"><span class="section-number-2">47</span> Schiff2018<sub>ClinicalRelevanceOfNTM.pdf</sub></h2>
<div class="outline-text-2" id="text-47">
</div>
<div id="outline-container-org6989bbc" class="outline-3">
<h3 id="org6989bbc"><span class="section-number-3">47.1</span> Abstract</h3>
</div>
<div id="outline-container-org2dd0cbf" class="outline-3">
<h3 id="org2dd0cbf"><span class="section-number-3">47.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org92ca51d" class="outline-2">
<h2 id="org92ca51d"><span class="section-number-2">48</span> Sergeev2016<sub>GenomeWide.pdf</sub></h2>
<div class="outline-text-2" id="text-48">
</div>
<div id="outline-container-org38d8f40" class="outline-3">
<h3 id="org38d8f40"><span class="section-number-3">48.1</span> Abstract</h3>
</div>
<div id="outline-container-org720a485" class="outline-3">
<h3 id="org720a485"><span class="section-number-3">48.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org2ad1e25" class="outline-2">
<h2 id="org2ad1e25"><span class="section-number-2">49</span> Vizcaino2010<sub>ComputatationalPrediction.pdf</sub></h2>
<div class="outline-text-2" id="text-49">
</div>
<div id="outline-container-orgfd3a860" class="outline-3">
<h3 id="orgfd3a860"><span class="section-number-3">49.1</span> Abstract</h3>
</div>
<div id="outline-container-org5a0812b" class="outline-3">
<h3 id="org5a0812b"><span class="section-number-3">49.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org007d856" class="outline-2">
<h2 id="org007d856"><span class="section-number-2">50</span> Waman2019<sub>MycobacterialGenomics.pdf</sub></h2>
<div class="outline-text-2" id="text-50">
</div>
<div id="outline-container-org0be8aac" class="outline-3">
<h3 id="org0be8aac"><span class="section-number-3">50.1</span> Abstract</h3>
</div>
<div id="outline-container-org7f755fb" class="outline-3">
<h3 id="org7f755fb"><span class="section-number-3">50.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgadcc047" class="outline-2">
<h2 id="orgadcc047"><span class="section-number-2">51</span> Wang2020<sub>ADeepLearningBasedMethod.pdf</sub></h2>
<div class="outline-text-2" id="text-51">
</div>
<div id="outline-container-org175df56" class="outline-3">
<h3 id="org175df56"><span class="section-number-3">51.1</span> Abstract</h3>
</div>
<div id="outline-container-org2fa1db2" class="outline-3">
<h3 id="org2fa1db2"><span class="section-number-3">51.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org115a71a" class="outline-2">
<h2 id="org115a71a"><span class="section-number-2">52</span> Weiner2017<sub>HighThroughput.pdf</sub></h2>
<div class="outline-text-2" id="text-52">
</div>
<div id="outline-container-org6234e4d" class="outline-3">
<h3 id="org6234e4d"><span class="section-number-3">52.1</span> Abstract</h3>
</div>
<div id="outline-container-orgd1c84eb" class="outline-3">
<h3 id="orgd1c84eb"><span class="section-number-3">52.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgb9fb9ef" class="outline-2">
<h2 id="orgb9fb9ef"><span class="section-number-2">53</span> Wilder2019<sub>ImprovingTuberculosis.pdf</sub></h2>
<div class="outline-text-2" id="text-53">
</div>
<div id="outline-container-org4963c28" class="outline-3">
<h3 id="org4963c28"><span class="section-number-3">53.1</span> Abstract</h3>
</div>
<div id="outline-container-orgec277b9" class="outline-3">
<h3 id="orgec277b9"><span class="section-number-3">53.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-orgc43a0dd" class="outline-2">
<h2 id="orgc43a0dd"><span class="section-number-2">54</span> Yang2017<sub>MachineLearning.pdf</sub></h2>
<div class="outline-text-2" id="text-54">
</div>
<div id="outline-container-org47c9ebe" class="outline-3">
<h3 id="org47c9ebe"><span class="section-number-3">54.1</span> Abstract</h3>
</div>
<div id="outline-container-org90664b9" class="outline-3">
<h3 id="org90664b9"><span class="section-number-3">54.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org0d45ec8" class="outline-2">
<h2 id="org0d45ec8"><span class="section-number-2">55</span> Yang2019<sub>DeepAMRForPredicting.pdf</sub></h2>
<div class="outline-text-2" id="text-55">
</div>
<div id="outline-container-orgfb3c3e7" class="outline-3">
<h3 id="orgfb3c3e7"><span class="section-number-3">55.1</span> Abstract</h3>
</div>
<div id="outline-container-org57fb5d0" class="outline-3">
<h3 id="org57fb5d0"><span class="section-number-3">55.2</span> Conclusion</h3>
</div>
</div>
<div id="outline-container-org3b19f04" class="outline-2">
<h2 id="org3b19f04"><span class="section-number-2">56</span> Zhang2020<sub>AnIntegratedMLApproach.pdf</sub></h2>
<div class="outline-text-2" id="text-56">
</div>
<div id="outline-container-org7a8dd9b" class="outline-3">
<h3 id="org7a8dd9b"><span class="section-number-3">56.1</span> Abstract</h3>
</div>
<div id="outline-container-orgf028465" class="outline-3">
<h3 id="orgf028465"><span class="section-number-3">56.2</span> Conclusion</h3>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Abhinav Sharma</p>
<p class="date">Created: 2020-05-07 Thu 23:23</p>
</div>
</body>
</html>
